<html><body><pre style='margin:2rem auto;width:fit-content'>ABSTRACT
When two colonies of the tunicate <span style='background-color:#cfc' title='Botryllus schlosseri : Exact'>Botryllus schlosseri</span> encounter one another,
they can either undergo vascular fusion or trigger a rejection reaction that blocks fusion.
This interaction is a form of allorecognition, defined as the ability to recognize self from
non-self, and in <span style='background-color:#fcc' title=' : NoMatch'>B. schlosseri,</span> the outcome of the allorecognition process is determined
by a single, highly-polymorphic fusion-histocompatibility (FuHC) locus. Another highly
polymorphic gene that encodes a Dnal-related protein was found recently in the FuHC
locus. The function of this gene is unknown, but the gene is of particular interest because
it shows a high level of polymorphism, which may indicate a function in allorecognition.
In this study, Dnal was characterized genetically in order to look for patterns of
polymorphism and population structure. The Dnal gene was sequenced from three
California populations and two Woods Hole populations. The sequences showed a high
level of polymorphism, with an average of 82.6 polymorphic sites and 33.35 pairwise
differences within populations. However, the gene showed no population structure
(FsT=O) and 100% of the variation was attributable to within population variation. A
locus by locus AMOVA showed divergence particularly at nucleotides at the 5' end of
the gene and that most of the variation was attributable to within population variation.
Amino acid variability was concentrated at the C-terminal end of DnaJ. These results
suggest either incomplete lineage sorting, linkage disequilibrium and gene hitchhiking or
balancing selection for genetic diversity. The high amount of variation in this gene is
unprecedented and may indicate a particular role in its function.
INTRODUCTION
Allorecognition is the ability to recognize self from non-self and has been found
in nearly all metazoans (Medawar, 1957; Burnet, 1971; Buss, 1987). This system of
recognition controls different biological functions, such as the immunological response
against pathogens, the prevention of inbreeding in angiosperms (Nasrallah, 2002), and the
ability to differentiate between closely and distantly related individuals (Maynard-Smith
and Szathmary, 1995). Allorecognition must depend on the presence of a highly
polymorphic gene which is required for the specificity of this process. In vertebrates, it is
controlled by the Major Histocompatibility Complex (MHC), while in angiosperms, it is
controlled by the self-incompatibility locus (SI). While allorecognition is pervasive
amongst the animal kingdom, the molecular basis underlying this specificity for most
taxa remains unknown.
The colonial ascidian Botryllus schlosseri is an excellent model organism for the
study of allorecognition for several reasons. First, B. schlosseri is an experimentally
accessible and genetically tractable organism. It is born as a tadpole larva with chordate
characteristics that settles within 24 hours and metamorphoses into a sessile invertebrate.
As an adult, it buds asexually to form zooids, genetically identical individuals that are
united by a common vasculature and embedded in a gelatinous tunic. Each zooid can
reproduce sexually and asexually. Thus, multiple experiments can be completed
simultaneously on genetically identical individuals. Second, B. schlosseri possesses a
naturally occurring blood-based transplantation reaction that is controlled by a single
genetic locus. As a colony grows asexually, it often comes into contact with another
colony of the same species. This can result in one of two possible outcomes: the
vasculature of the two colonies can fuse and a chimeric colony is formed, or a rejection
reaction occurs, causing the two adjacent colonies to retreat and grow in opposite
directions. Finally, as a member of the group that is the closest living invertebrate relative
to the vertebrate lineage, B. schlosseri is phylogenetically poised to address questions
regarding the origins of complex innate and adaptive immunity.
The outcome between two interacting colonies is a form of allorecognition. This
outcome is controlled by a single, highly polymorphic, co-dominantly expressed fusion¬
histocompatibility locus (FuHC) (Fig. 1). A specific gene within this locus, fuhc, was
found to encode the self-ligand that is recognized by the interacting colonies. This gene is
highly polymorphic, and the polymorphisms correlate absolutely with fusibility (De
Tomaso et al., 2005). In order to fuse, two colonies must share at least one of the fuhc
alleles. Another allorecognition gene, fester, has been identified in the FuHC locus
(Nyholm et al., 2006) and encodes a putative receptor for the fuhc ligand. Currently, the
function of another gene, uncle fester, which is also found in the FuHC locus, is being
studied in the allorecognition process (Mckitrick and De Tomaso, in prep). The FuHC
locus also contains a highly polymorphic Dnal-like gene. The function of this gene
(which I will refer to as DnaJ for simplification) is unknown, but the polymorphism
found in this gene in preliminary studies is unexpectedly high, making it a gene of
particular interest. Furthermore, its location in the FuHC locus makes it a candidate
allorecognition gene. This clustering of genes may be similar to the type of clustering
found in the MHC genes. The MHC region in vertebrates is a genomic region that
contains genes involved in immunity and allorecognition (Kindt et al., 2007).
This DnaJ gene encodes a protein that belongs to the DnaJHsp40 superfamily as
it contains a J-domain at its N-terminus. This J-domain is a 70 amino acid sequence that
is characteristic of all Dnal proteins (Walsh et al., 2004). DnaJ proteins are known to act
as co-chaperones to Hsp70 by binding to Hsp70s through their J domains and stimulating
ATP hydrolysis by Hsp70. Because they regulate Hsp70 function, they play an important
role in protein translation, folding, unfolding, translocation and degradation (Qiu et al.,
2006). Dnal proteins can also be molecular chaperones by themselves or regulate other
chaperones, such as Hsp90. Some DnaJ proteins may have acquired a J-domain in order
to recruit Hsp70 to their substrate, which may require this chaperone to carry out its
function. Proteins containing J domains are highly conserved across evolution and can be
found in bacteria, fungi, plants, animals and one class of DNA viruses (Kelley, 1998).
Therefore, understanding the function of DnaJ may contribute to understanding why J
domains are being used in so many proteins and if there are fundamental mechanisms that
underlie J-domain function.
This study will begin to determine the functionality of DnaJ by characterizing it
genetically. I will look at the patterns of polymorphism and amino acid variability, test
for population structure and determine the phylogeny of DnaJ. This will be achieved by
analyzing sequences of DnaJ from different B. schlosseri populations.
MATERIALS AND METHODS
Collection, amplification and sequencing of DNA
<span style='background-color:#fcc' title=' : NoMatch'>B. schlosseri colonies</span> were collected from the side of boat slips at Monterey
Marina, California. One system (7-11 zooids) was dissected out from each colony, and all
embryos and tadpoles were removed. Samples were frozen in liquid nitrogen and stored
at -80°C. RNA was extracted from each Monterey sample using a NucleoSpinû RNA
Extract II kit. CDNA was synthesized from the RNA using a reaction mixture of 1 uL of
Invitrogen Superscript III, 1 uL of oligo dTTis and 0.4 uL 25mM dNTP. The reaction
mixture was heated to 70°C for five min, put on ice for two min and heated to 50°C for
90 min. Pooled CDNA from individuals from Northern California (Half Moon Bay, Santa
Cruz, Moss Landing), Santa Barbara, CA, Eel Pond, MA and Sandwich Bay, MA, were
prepared previously.
Polymerase chain reaction (PCR) amplification was performed on 1488 bp of the
DnaJ using 0.6 uL DMSO, 4 uL 5X Phusion buffer, 0.15 uL Hot-start Phusion Taq, 0.4
uL of custom-made primers 5'Dnal (S’-GTATTAAGGTCATTTCTTAAACCC-3’) and
DnaJWR (S’-AACACTTGATTACATTTACCGCAAC-3’), 12.29 uL H20 and 2 uL of
CDNA. The cycling conditions included an initial denaturation of 98°C for 3 min, 40
cycles of 98°C for 30 sec, 50°C for 30 sec, 72°C for 45 sec, and a final extension step of
72°C for 3 min. PCR product was visualized on 1% agarose gels with 0.4 ug/mL of
ethidium bromide. PCR product was cleaned using SpinPrepTM PCR Clean-Up Kit
(Novagen). The cleaned product was A-tailed using 1 uL 1OX NEB Standard buffer, 0.08
uL 25 mM dNTP, 0.08 uL NEB Taq and 9.0 uL template. The reaction mixture was
incubated at 72°C for 20 min. This product was ligated into pGEM T vectors (Promega)
using 1.0 uL 10X ligase buffer, 1.0 uL T4 ligase, 0.5 uL pGEM vector and 7.5 uL of
template. The ligation reaction was carried out at room temperature overnight. The
ligated product was cleaned by N-butanol precipitation. One microliter of cleaned
plasmid was electroporated into 20 uL of Top10 electrocompetent <span style='background-color:#fcc' title=' : NoMatch'>E. coli cells</span> at 1.56
KV, 2.5 KV resistance and 186 resistance timing. One milliliter of SOC was added to the
electroporated cells, and the cells were shaken at 225 rpm for one h at 37°C. Twenty
microliters of the cells were plated onto agar plates containing ampicillin, X-gal and
IPTG. Cells were grown overnight at 37°C. White colonies were selected, and the gene
was amplified using T7 and SP6 primers (34 cycles, 53°C annealing). The PCR product
was purified using ExoSAP-IT (USB) and cycle sequenced in four directions and
primers T7 and SP6, or 5'Dnal and WR, and Dnal F3 (5'-
GGAATTAGAGAGAATCGCAACCGAAC-3’) and F14DnaJR (5'¬
ATTGAGAAACTCTTTCGTTGCAAC-3’). The product was cleaned using a 75%
isopropanol precipitation and sequenced using a Prism 3100 Genetic Analyser (Applied
Biosystems).
Sequence analysis
These were aligned, edited and trimmed to 1356 bp in Sequencher and analyzed
in the software ARLEQUIN 2.0. Standard measures of molecular diversity were
calculated, and sequences were analyzed for population structure. Nucleotide and protein
parsimony trees were generated in PAUP 4.0 using a heuristic search. Amino acid
sequences were obtained in MacClade.
RESULTS
Genetic diversity. From the seven Monterey individuals, five were
heterozygous, and eight haplotypes were sequenced. Six haplotypes were obtained from
Northern California, four from Santa Barbara, three from Eel Pond and six from
Sandwich Bay (Table 1). The mean number of pairwise differences for Monterey,
Northern California, Santa Barbara, Eel Pond and Sandwich Bay were 42.0-19.6,
32.6+16.3, 44.0-24.4, 17.2-8.9 and 31.0+15.8 (Fig.2A). The total numbers of
polymorphic sites for the same populations were 127, 73, 81, 51 and 81, respectively
(Fig. 2B). Polymorphic sites were found across the whole gene, but concentrated at the
3'end.
Genetic population structure and phylogenetic analysis. To determine the level
of genetic differentiation at DnaJ, a molecular analysis of variance (AMOVA) was
performed on populations structured into California and Massachusetts groups. The
AMOVA indicated no significant differentiation among populations (FST-O.O, p-0.0),
and 100% of the variation was attributable to within population variation (Table 2). The
locus by locus AMOVA did show significant differentiation among populations averaged
over all loci (Fsr=O.19286, p-0.0). The majority of the variation at each nucleotide was
attributable to within population variation and least attributable to among group variation
(Table 3). The loci with significant Fsr's were concentrated at the 3' end of the gene (Fig.
3). Tajima’s D was insignificant for all populations except for Eel Pond (Tajima’s D =-
1.4737, p=0.0264) (Table 1).
The nucleotide and protein trees show clades that are specific to California and
Massachusetts populations (Fig 4A,B), but the two groups of populations do not
segregate completely into separate clades. Each tree also shows a mixed California and
Massachusetts clade, and shared haplotypes between California and Massachusetts
populations.
Amino acid diversity. The nucleotide sequences were translated into their
amino acid sequence in MacClade and aligned to look for amino acid variation. A
variability plot (Wu and Kabat, 1970) shows that amino acid substitutions are found
across the whole protein, but concentrated at the C-terminus (Fig. 5).
DISCUSSION
DnaJ is a highly polymorphic gene, at both the nucleotide and amino acid level.
The amount of polymorphism found in DnaJ is similar to the amount of polymorphism
found in the allorecognition genes, fuhc and fester, which have been shown to be highly
polymorphic (De Tomaso et al., 2005; Nyholm et al., 2006). On average, Dna shows a
mean pairwise difference of 33.35 +4.77. For fuhc, the mean pairwise difference was 45
for Woods Hole and 37 for California, and for fester, the mean pairwise difference was
23 for Woods Hole and 34 for California (unpublished data). In contrast, COI, which is
evolving neutrally, has a mean pairwise difference of 1.4 for Woods Hole populations
and 2 for California populations (McKitrick, unpublished). The total number of
polymorphic sites is also unexpectedly high. In Monterey alone, nearly 10% of the locus
is polymorphic, with twelve alleles having 127 polymorphic sites. On average, DnaJ had
82.6 polymorphic sites over all alleles sequenced.
Although this gene has a considerable amount of genetic diversity within
populations, it lacks population structure, as indicated by the AMOVA tests. This lack of
population structure is reflected in the phylogenic protein and nucleotide trees, which
show a mixed California-Woods Hole clade. This observation could be explained by
three processes, either incomplete lineage sorting in which the gene may not be evolving
fast enough to show any differentiation, linkage disequilibrium and genetic hitchhiking to
the fuhc or balancing selection. As a comparison, fuhc (ectodomain) and COl, a neutral
gene, show population structure between California and Woods Hole (Fsr-0.29176,
p-0.0; FsT=0.92212, p-0.0), whereas fester does not show any structure (Fs-=-0.01506,
p-0.48680) (McKitrick & De Tomaso, unpublished). Balancing selection may also be
generating this genetic diversity. Although the Tajima's D for four of the populations
suggests gene neutrality, there may be selection occurring at portions of the gene. The
average Fsr over all loci indicates that there are divergent loci, particularly at the 3' end.
This pattern of divergent loci is similar to the pattern of amino acid variability, which is
concentrated at the C-terminus. This observation may suggest selection occurring at the
3’ end. Further analysis would be needed to determine whether selection is generating
this diversity. The unexpectedly high amount of variation and the possibility of selection
in this region of the protein may suggest that this region plays a crucial role in the
function of the protein. Investigation into whether the C-terminal ends of other Dnal
genes are similarly variable may provide clues as to what is occurring in this region.
Although a high degree of genetic diversity was found, the amount of
polymorphism for DnaJ may be underestimated in this study. Because only several
clones were sequenced for each Monterey individual, the number of heterozygotes and
alleles may have been underestimated. Picking more clones may reveal more alleles.
However, it is also possible that more than two alleles may be present in one individual.
This could be due to multiple gene copies of DnaJ or a phenomenon known as
chimerism, which results from stem cell parasitism in B. schlosseri (De Tomaso, 2006).
In addition, picking more clones from the other populations may also reveal more alleles
as the samples are pooled from many individuals. This would not only reveal more
alleles, but also more polymorphism at the nucleotide level.
This study has genetically characterized DnaJ. It has shown an unprecedented
amount of polymorphism particularly in the 3' end and amino acid variability at the C¬
terminus. Comprehension of the genetics of DnaJ will help us better understand how the
protein functions. Future studies of DnaJ will examine where the gene is expressed and
its functional role. Gene expression can be determined using in-situ hybridizations.
Because of its location in the fuhc locus, it can be hypothesized that DnaJ is involved in
the allorecognition process. It could possibly be involved in signaling as other Dnal
proteins have been found to be functional in androgen and estrogen signaling (Qiu et al.,
2006). Its function in allorecognition can be determined using siRNA knockouts on the
gene and observing its effect on allorecognition. Since Dnal proteins are also known to
be co-chaperones to Hsp70 proteins (Qiu et al., 2006), it should also be determined
whether our Dnal protein is associated with an Hsp70.
The results of this study also raise a question that is fundamental to evolutionary
biology. How is this high level of genetic variation created? Furthermore, how has this
level of genetic variation been maintained throughout evolutionary time? This question
can also be asked for fuhc and fester. The ascidian does not have any somatic
diversification mechanisms, such as recombination or somatic hypermutation. It has been
estimated that nearly 250 MHC alleles exist within the extant human population.
However, nearly 100 fuhc alleles have been found within Monterey Bay alone, and it has
been estimated that nearly 1000 alleles exist worldwide. Herein, we are reporting the
existence of another highly polymorphic gene, DnaJ, which is found within this locus.
Understanding the function of this gene and how the C-terminal end of the protein may
11
be involved in its function may elucidate what could be driving the generation and
maintenance of allelic diversity in the allorecognition locus of Botryllus schlosseri.
ACKNOWLEDGEMENTS
I would like to thank my advisor Anthony De Tomaso for his guidance and
support as well as Tanya McKitrick who has been particularly instrumental in the
realization of this project. I would also like to thank the other members of the De Tomaso
lab for their help and advice, specifically Snjezana Rendulic for teaching me lab
techniques and Kathi Ishizuka for sharing her dissection skills. Finally, I would like to
thank George Somero for teaching Bio175H and providing me with the opportunity to get
involved in research.
LITERATURE CITED
Burnet, F.M. 1971. "Self-recognition" in the colonial marine forms and flowering plants
in relation to the evolution of immunity. Nature 232, 230-235.
Buss, L.SW. 1987. The Evolution of Individuality. Princeton University Press, Princeton.
De Tomaso, A.W., Nyholm, S.V., Palmeri, K.J., Ishizuka, K.J., Ludington, W.B.,
Mitchel, K. and Weissman, I.L. 2005. Isolation and characterization of a
protochordate histocompatibility locus. Nature. 438, 454-459.
De Tomaso, A.W. 2006. Allorecognition polymorphism versus parasitic stem cells.
Trends Genet. 22, 485-490.
Kelley, W.L. 1998. The J domain family and the recruitment of chaperone power. Trends
Biochem. Sci. 23, 222-227.
Kindt, T.J., Goldsby, R.A. and Osborne, B.A. 2007. Immunology. W.H. Freeman and
Co., New York.
Maynard-Smith, J. and Szathmary, E. 1995. The Major Transitions in Evolution. W.H.
Freeman, San Francisco.
Medawar, P.B. 1957. The Uniqueness of the Individual. Methuen, London
Nasrallah, J.B. 2002. Recognition and rejection of self in plant reproduction. Science.
296, 305-308.
Nyholm, S.V., Passegue, E., Ludington, W.B., Voskoboynik, A., Mitchel, K., Weissman,
I.L. and De Tomaso, A.W. 2006. fester, a candidate allorecognition receptor from a
primitive chordate. Immunity. 25, 163-173.
Qiu, X.-B., Shao, Y.-M., Miao, S. and Wang, L. 2006. The diversity of the DnaJ/Hsp40
family, the crucial partners for Hsp70 chaperones. Cell. Mol. Life Sci. 63, 2560-2570.
Walsh, P., Bursac, D. Law, Y.C., Cyr, D. and Lithgow, T. 2004. The J-protein family:
modulating protein assembly, disassembly and translocation. EMBO Rept. 5, 567-
571.
Wu, T.T. and Kabat, E. A. 1970. An analysis of the sequences of the variable regions of
Bence Jones proteins and myeloma light chains and their implications for antibody
complementarity. Jl Exper.1 Med. 132, 211-250
Table 1: Indices of polymorphism, haplotype frequency and tests for gene neutrality
within B. schlosseri populations
Average no.
Tajima's D test
Total no. of
No. of
Nucleotide
Population
of pairwise
diversity
haplotypes
polymorphic sites
P-value
differences
0.51047
003140016
-0.0111
Monterey
42.0
127
0.55129
-0.3095
0.024 + 0.014
32.6
California
-0.043
0.58127
44.0
0.033 + 0.022
S. Barbara
-1.4737
0.0264
Eel Pond
0.013 4 0.008
172
Sandwich
-0.8135
0.26291
0.031 + 0.016
31.0
Bay
Table 2: AMOVA table. Analysis was performed on populations structured into
California and Massachusetts groups.
Sum of
Variance
Percentage
Source of variation
d.f.
Squares
components
of variation
Among groups
Among populations
within groups
0.5
100
Within populations
0.5
Total
Table 3: Locus by locus AMOVA. Analysis was performed on populations structured
into California and Massachusetts groups.
Variance
Sum of
Source of variation
components
Squares
Among groups
58.343
1.36285
Among populations
2.76937
106.846
within groups
518.131
17.29404
Within populations
683.32
21.42625
Total
FIGURE LEGEND
Fig. 1. FuHC locus in Botryllus schlosseri. Identified allorecognition proteins include
fuhc and fester. Uncle fester and DnaJ are currently being characterized.
Fig. 2. Nucleotide diversity in DnaJ. Mean number of pairwise differences (A) and total
number of polymorphic sites (B) in DnaJ (1356 bp) for each B. schlosseri population.
Errors are standard errors of the mean.
Fig. 3. Nucleotide divergence in DnaJ. Significant Fsr's (pS0.05) from the locus by locus
AMOVA were plotted across the length of the DnaJ sequence.
Fig. 4. Phylogenetic relationships between California and Massachusetts individuals.
Nucleotide (A) and protein (B) trees show California (Red) and Massachusetts (Blue)
clades as well as mixed population clades (Arrows).
Fig. 5. Amino variability of DnaJ. Variability is measured using the Wu-Kabat
coefficient of variability.
lester A clsde
uncle fexter
. —
+

E3-ubiquitin
ligse (Medd4)
fexter Bclade
——
Fig. 1
MNS knæe
func
a
wone - - — —

—


—

Fhosphelipase 2 pindin
DNAtopo
protein
20 k5
20
Monterey
p;

Populations
Santa Sandwich Eel Pond
California Barbara bay
Population
﻿60
Fig. 2
0.7
0.6
0.5
0.4
0.3
0.2
0.1
Fig. 3
1 102 203 304 405 506 607 708 809 910 1011 1112 1213 1314
Nucleotide position
California
Massachusetts
5 changes

Fig. 4
TH10.8
TH9.1
— TH10.6
TH9.3
- SB.3
- Sand.3
Sand.8
Sand.5
Sand6
Eel2b
TH4.3
TH5.1
— TH12.6
NC 4
INC.7
- SB.8
TH6.8
+ NC.5
NC3
NC.2
Eel.3b
NC.8
Eel.3
Eel2
Eel.7
Sand.10
Eel.5
TH12.2
IH32
1s.
SB.6
TH3.3
TH4.8
- NC.10
- Sand.2
TH5.1
NC.2
Eel.3b
NC.8
Eel.3
Eel.2
Eel.7
Sand.10
Eel.5
- Sand.3
Sand.8
Sand.5
Sand.6
- Eel.2b
- TH4.3
— TH12.6
NC.4
NC.7
SB8
TH6.8
- NC.5
NC.3
TH12.2
TH3.2
- SB.1
B.6
TH3.3
TH4.8
- NC.10
- Sand.2
- 5 changes
TH10.8
TH9.1
TH1O.6
TH9.3
SB.3
6
Fig. 5

0-
1 33 65 97 129 161 193 225 257 289 321 353 385 417 449
Amino acid position
</pre></body></html>