<html><body><pre style='margin:2rem auto;width:fit-content'>Acknowledgments
I thank my first reader, Dr. Lynna Hereford, for her patient guidance throughout the
course of both the research and writing stages of this thesis. Your excitement about this
project -- and your sense of humor! - have been invaluable.
Many thanks to Dr. Carol Boggs, my second reader and my academic advisor, for her
comments upon several drafts of this thesis and for offering her expertise in the
population genetics field.
I am also indebted to Dr. Dennis Hedgecock of UC Davis for providing the allozyme data
from his 1979 study and for offering tools and insight to aid in the data analysis. Drs.
Jeffrey Mitton and Patrick Gaffney were also influential in the data analysis.
Many thanks to Dr. Dennis Powers and his laboratory for providing resources and work
space and thanks to many others at the Hopkins Marine Station who assisted me in my
time there.
I would like to thank Gregory Morris for his contributions as my partner during the spring
course at the Hopkins Marine Station. The present work is an extension and completion
of our spring research.
I have also appreciated the guidance, reassurance, and administrative help of those in the
Human Biology office: Heidi Ballard, Jody Kamrowski, and Dr. Ellen Porzig.
Finally, I thank my family, friends, and future husband for their love and support.
Abstract
The decline of natural stocks of the commercially harvested red abalone, <span style='background-color:#cfc' title='Haliotis rufescens : Exact'>Haliotis
rufescens,</span> has led to land-based cultivation. To date, there has been little research on the
effects of domestication upon the genetic diversity of this species. In this study, protein
gel electrophoresis was used to estimate the extent of inbreeding and/or genetic drift
within cultured and natural red abalone populations. The present analysis of nine
populations indicates that there are significant differences between the genetic
compositions of the Santa Barbara 1992 population, believed to be an introduced
population, and the natural populations, but only slight differences between the cultured
and natural populations. This has implications for the success of the California abalone
enhancement programs of 1979-1980. Additional results of this study have relevance for
abalone aquaculture. The genetic drift analysis of one of the cultured populations
indicates that not all of the abalone used for each hatchery spawn had actually contributed
to the production of offspring.
Table of Contents
ACKNÖWLEDGMENTS ii
ABSTRAC III
TABLE OF CONTENIS I
LISTOF TABLES
LIST OF FIGURES VI
INTRODUCTION
MATERIALS & METHODS 4
Sample Collection 4
Sample Preparation.
Horizontal Starch Gel Electrophoresis5
Interpreting Gels0
Data Analysis
RESULS
Genetic Variation Analysis
Genetic Drift Analysis
DISCUSSION O
CONCLUSION4
Aquâculture14
Enhancement1
TABLES Z0
FIGURES 26
BIBLIÖGRAPH 30
List of Tables
Page
Table
1. Summary of Sampled Populations.. 20
2. Summary of Allozyme Analysis
21
3. Allele Frequency ChartZZ
4. Summary Statistics of Genetic Variation 23
5. Genetic Drift Analysis Summary 24
6. Comparison of Two Red Abalone Seedings.... 25
List of Figures
Page
Figure
1. California Red Abalone Landings (1931-1988) 20
2. Map of Collection SitesZ/
3. D-values by LocusZ0
4. Sources of PMI Female and Male Broodstock29
Introduction
California abalones have been harvested by humans for hundreds of years, but
large-scale commercial harvesting did not begin until 1916 (Tegner et al., 1992). Red
abalone, Haliotis rufescens, is preferred among the several native Californian abalone
species due to its larger size and distinct flavor. However, the natural history of the
abalone - a slow-growing, long-lived, fairly sessile creature - seems to facilitate
overfishing (Tegner, 1989). Between 1931 and 1968, state-wide landings averaged over
900 metric tons wet weight per year. Since 1968, commercial catches have declined to
15% of that: current landings are less thun 136 metric tons per year (Tegner et al., 1992).
(See Figure 1) This decline has occurred despite intensive effort by the California
Department of Fish and Game to restore the fishery.
Presently, commercial red abalone harvesting is centered in south-central California
- most specifically near the Channel Islands off the coast of Santa Barbara. Loss of some
formerly productive areas due to overfishing of natural stocks, expansion of the sea otter
range, environmental degradation of some mainland sites, and competition from
recreational divers have pushed the commercial divers to travel to the furthest Channel
Islands (Tegner, 1989). According to John Colgate, the President of the Abalone Divers
Association, San Miguel Island (see map Figure 2) is presently believed to have the largest
bed of abalone in southern California (personal communication). Northern California,
which has an even larger supply of abalone, is closed to commercial harvesting.
As the supply of abalone diminishes, their per unit value continues to increase.
Between 1973 and 1988, the price paid to divers has increased 800%, over twice the rate of
inflation (Tegner et al., 1992). Abalone uquaculture is a logical response to the diminishing
natural supply and continuing high demand that Californian, Japanese, and other seafood
connoisseurs have placed upon this gastropod. Because the present market for abalone is
twice as large as the current supply, cultivation of abalone is likely to continue to increase
(Sietsema, 1992).
Oysters, scallops and clams have been cultured for many years in the United States,
and Japan has been ocean ranching shellfish, including abalone, even longer. A small
laboratory constructed in 1964 at Morro Bay tested red abalone cultivation based on
Japanese abalone hatchery methods and showed that mass cultivation of abalone was
feasible for California industry (Ebert, 1984). Innovations in abalone culturing and the
increase in demand and price have now made abalone farming profitable. However, this
profit obviously relies on maintaining cost effective production. The most costly aspect of
abalone production is the organism's slow growth rate: abalone take three years to reach
market size.
In addition to being slow, abalone growth also varies among individuals. One
cohort in this study had a nearly 20-fold range in mass, from 0.3 grams to 5.85 grams.
This variable growth rate forces most cultivators to sort their abalone into size classes and
to discard the smallest 5-20%. Finally, the abalone’s high fecundity makes cultivated
breeding possible, but tempts aquaculturists to use a small number of adults to produce
thousands, and even tens of thousands of offspring. Over time, this continual loss of par
of the population and these hatchery breeding techniques may alter the gene pool,
potentially leading to inbreeding. Inbreeding often has devastating effects: homozygote
appearance of deleterious rare alleles, inability to have varied responses to changes in the
environment, loss of biodiversity, and a general loss of robustness. Ironically, the
breeding practices of abalone cultivators could produce abalone with even slower growth
rates than at present.
As abalone cultivation is a fairly new industry in California, little research has been
conducted upon the domestication of this species. However, several studies involving
electrophoretic work have been done in Japan, examining cultivation and even ocean
ranching of some other abalone species and of oysters (Fujino, 19781 & Il; Fujino and
Nagaya, 19771 & II; Fujino and Sasaki, 1984; Wada, 1986; Saito, 1984). Additionally
cultivation of other shellfish in the United States is used as a parallel to the more recent
cultivation of abalone. Electrophoretic studies found genetic diversity within and/or
between populations of the Coot Clam (Gaffney et al., 1990) and domesticated American
oysters (Hedgecock and Okazaki, 1984; Vrijenhoek et al., 1990). In addition to simply
comparing genetic diversity, several oyster studies pointed to the importance of measuring
the temporal variance of allelic-frequency in order to estimate the effective size of the
populations. These values provide a means to analyze genetic drift and can aid in
determining the extent and nature of genetic change in reproductively isolated populations.
Two studies of oysters and other marine shellfish estimated very low effective population
sizes for these cultivated populations even when the traditional methods showed little or no
significant inbreeding (Hedgecock and Sly, 1990; Hedgecock et al., 1992). A third study
indicated an additional value of estimating the effective population size. Work by Gaffney
et al. (1992) found that fewer oyster individuals than previously thought were actually
contributing to the offspring produced in large group hatchery spawns.
These studies show the importance of examining the population genetics of cultured
California abalone by comparing genetic variation and by measuring genetic drift. This
report analyzes nine populations of abalone: three separate spawns from a hatchery,
samples from the male and female culture broodstock, natural abalone from two areas of
the northern California coast, and samples of abalone collected in 1992 and in 1979 by
commercial divers near Santa Barbara, California. The goals for the present study are to:
determine the genetic composition of cultured and natural populations of the red abalone,
Haliotis rufescens, and estimate the extent of inbreeding and genetic drift within these
populations.
Additionally, anecdotal evidence indicates that the 1992 Santa Barbara population
analyzed in this study was the result of abalone seeding projects of the late 1970's and
early 1980's (Westlotorn, personal communication). In addition to aquaculture, another
response to the diminishing natural supply of abalone was artificial enhancement. Though
testing by the California Department of Fish and Game officially labeled abalone seeding as
unsuccessful, the present study provides some evidence to the contrary. A re-analysis of
the California abalone enhancement projects is presented.
Materials & Methods
Sample Collection:
Please refer to Figure 2 for a map of the collection sites and to Table 1 for a
summary of the populations.
A total of five populations of cultured red abalone, Haliotis rufescens, were
collected from Pacific Mariculture, Incorporated (PMI) in Santa Cruz, California.
Population 1 (N = 156), Population 2 (N = 120), and Population 3 (N = 122) were
samples from three separate 1991 spawns that occurred on March 28, August 21, and July
25, respectively. For each spawning, PMI used between 25-50 gravid abalone. The
females were chosen from a brood stock of 7-800 individuals while the males were chosen
randomly from the production tanks. Population 4 (N = 43) was from the female
broodstock and includes samples from 12, 5 and 6 tagged individuals that were known to
have contributed eggs to spawns 1, 2 and 3, respectively. As PMI has no separate male
broodstock, 2.5 inch (about 2.5 year old) male abalone from one production tank were
sampled for Population 5 (N = 39). Based on size and growth rate estimations, most of the
male abalone used in 1991 spawns had probably been sold before samples were collected
for analysis. However, this population does offer an interesting comparison of allelic
diversity. Further examination of the genetic history of the male and female abalone used
in PMI's spawns will be explored in the Discussion.
In addition to the culture populations, two abalone populations from northern
California and two from southern California were studied. Seventy mantle and gill tissue
samples were collected from abalone harvested by recreational skin divers near Gerstle
Cove (population 6) and 46 samples were from the coastline around Fort Ross (population
7).
Population 8 came from commercial divers who collected their abalone from the
south side of San Miguel Island off the coast of Santa Barbara in May 1992 (Westlotorn,
personal communication). Data from a 1979 Sea Grant study was used as a historical
comparison. Dennis Hedgecock (1979) analyzed samples of natural red abalone
populations collected from six sites in the Santa Barbara area, including the north and south
sides of San Miguel Island, and found very little variation from one collection site to the
next. Hedgecock's data were pooled to form Population 9 (N = 208) which represents the
natural abalone population off the coast of Santa Barbara. The cultivated abalone
populations in the study and in the other farms in California were ultimately derived from
the abalone in the Santa Barbara area (McBride, Oakes, McMullen, and McCormick,
personal communication).
Sample Preparation
Foot muscle, shell muscle, mantle, and/or gill tissue were analyzed (Table 2).
Tissue samples were placed in 1.5 mL Eppendorff tubes, with 20-40 microliters of buffer
and 5-10 mg of silicon beads. A buffer solution of O.1 MTris-HCl pH 7.0 buffer
containing O.1% B-mercaptoethanol was used for better resolution (Gaffney et al., 1992).
Samples were homogenized with an electric drill fitted with a glass grinding tip. To
prevent any denaturing of the proteins, samples were stored at -70 degrees C and placed on
ice during preparation and transport.
Horizontal Starch Gel Electrophoresis
The horizontal starch gel electrophoretic procedure was based on that described by
Schaal and Anderson (1974). No enzymatic activity was detected for isocitrate
dehydrogenase, lactate dehydrogenase, malic enzyme, and glucose-6-phosphate
dehydrogenase. The following enzymes gave positive staining but were not resolved well
enough or consistently enough to be scored: hexokinase, mannose-6-phosphate isomerase,
esterase, 6-phosphogluconate dehydrogenase, and tetrazolium oxidase. Leucine amino
peptidase was resolved, but it appeared monomorphic and required digestive tissue
samples, so assaying for this enzyme was not continued. Four enzymes -
glucosephosphate isomerase (PGI), phosphoglucomutase (PGM), glutamate-oxaloacetate
transaminase (GOT), and malic dehydrogenase (MDH) - were consistently resolved using
the Tris-citrate pH 5.8 buffer of Schaal and Anderson (1974). Additionally, these four
enzymes were analyzed in Hedgecock's 1979 study.
After centrifuging the samples for approximately five minutes, the solutions were
absorbed into filter paper wicks, blotted, and placed onto the gel. Gels ran 4-6 hours at
160 volts (0.3-0.4 amps. per gel) and were sliced in preparation for staining.
Interpreting Gels
Two loci were observed for PGM and MDH, and the more quickly migrating
isozyme was scored in both cases for it was more active and more polymorphic. These loci
were labeled PGM-2 and MDH-2, in accordance with the analysis and labeling of the 1979
data (Hedgecock). For each locus, the most common allele was labeled 100 and the other
alleles were designated by adding or subtracting the estimated number of millimeters they
varied from the chosen standard. For the PGM-2 locus, the allele chosen as the standard
differed between the 1979 data and the 1992 data, but this difference was accounted for in
the data analysis. To reduce scoring errors, questionable samples were run more than once
and samples from consecutive populations were loaded onto a single gel to ensure that the
allele designations were consistent among the eight populations assayed. Additionally, the
allelic designations were consistent with the 1979 data.
Data Analysis
Genotypes were tallied for each locus. In transferring data to table/graph form, the
original numerical allele designations were replaced with alphabetic ones. The most
common allele (originally 100) became A, the next common B, and so forth. Swofford
and Selander's BIOSYS-1 program (1981) was used to compute: allele frequency, Hardy
Weinberg equilibrium, mean number of alleles per locus (ni), observed and expected
heterozygosity (Ho and He), and the differences between them (D- Ho-He/Ho).
Another program (untitled, developed for Hedgecock et al., 1992) was used to analyze
genetic drift variables: temporal variance of allelic frequencies (F), estimated per¬
generation effective population size (Ng), an independent measurement of the numbers of
alleles expected to remain in each population (no, and a chi-square test of how ni compares
with the actual number of alleles (na). The genetic drift analysis includes an additional
variable, Np, the harmonic mean of the number of abalone spawned per hatchery-produced
generation. As the determination of Np depended not only on estimating how many
generations the abalone were removed from the wild, but also depended upon a compilation
of old, inadequate hatchery and sales records of several different companies, a range was
calculated using the lowest and then the highest possible variables.
Results
Genetic Variation Analysis
Table 3 shows the allele frequencies at each locus for all studied populations. Note
that all 60 samples collected in the 1992 Santa Barbara population were heterozygous at the
PGI locus. Goodness-of-fit to Hardy-Weinberg equilibrium was tested at each loci for
every population, using Levene’s correction for small sample size. For loci with several
alleles, a chi-square test was performed with all but the most common alleles pooled
together. Table 3 indicates that three culture populations contained one locus that was not
in Hardy-Weinberg equilibrium at the p « 0.05 level. The Santa Barbara 1992 population
had two such loci and the natural populations had none.
To examine possibilities of inbreeding in the populations sampled, the average
number of alleles per locus (ni) and the D-values for observed and expected heterozygosity
D = (Ho-He)Hel were compared. As can be seen in Table 4, the cultured populations
tended to have lower nj values than the natural populations, though the differences are not
statistically significant using the t-test. Notice that ni for the 1992 Santa Barbara
population, though not statistically significant, is much lower than the value for any of the
cultured populations. Unlike all of the other populations, it was completely monomorphic
at the MDH locus. Again, in comparing observed and expected heterozygosities and their
differences (Ho, He, and D), there were no statistically significant differences or possible
trends noticed between the cultured and natural populations. However, the 1992 Santa
Barbara Population had the highest observed heterozygosity and the next to lowest
expected heterozygosity, resulting in a large difference (D-.652). This difference is due
mostly to the PGI locus, but also to the PGM locus (Figure 3).
Genetic Drift Analysis
In addition to measuring the previous variables that have traditionally been used to
describe and compare population genetic structure, this study used a tool to analyze genetic
drift more specifically. This genetic drift test uses the allele frequencies of the derived and
progenitor populations to determine the allele frequency variance (F) and to estimate the
effective size of each derived population based upon how many generations it has been
separated from the progenitor (wild) population. In an independent test, Fis used to
estimate how many alleles are expected to remain in the population (nt). This expected
value is then compared with the actual number of alleles remaining (na) in a chi-square test.
If the observed value, na, is determined to be not significantly different from the expected
value, nt, then more validity is given to F and thus to the accuracy of the original estimated
population size. However, the chi-square test is not very accurate if either the expected
number of alleles remaining or lost is very small (£ 0.2).
Table 5 summarizes the information found for each population in the genetic drift
analysis. The 1979 Santa Barbara abalone population was used as the originating
population to compare with all of the populations collected and analyzed in 1992. As it was
very difficult to accurately determine how many generations separated the cultured abalone
from the wild, the genetic drift analysis was carried out with two best estimates per
population. The five culture populations from PMI did not come directly from natural
populations, but were produced from broodstock, or are actually seedstock, that were
purchased from several California abalone hatcheries. These other aquaculturists originally
obtained their abalone primarily from souther California waters around Santa Barbara or
the Channel Islands (McMullen, Oakes, and McCormick, personal communication), so it
seems fairly safe to assume that the 1979 Santa Barbara population represents the source
for the cultured abalone. The two northern California populations were analyzed as
controls, to see if the 1979 Santa Barbara population could be assumed to have a genetic
composition similar to a natural population from any location in California.
The three populations spawned ut PMI show a trend toward having lower allele-
frequency variance (F) with each subsequent sample: F = 0.130, 0.087, 0.027 for
populations 1, 2 and 3, respectively. The population of 2.5 year old male abalone at PMI
had a high allele-frequency variance (F = 0.136) and subsequently low effective population
size estimates. The female PMI broodstock abalone had a low allele-frequency variance (F
= 0.027) and subsequently had fairly high population size estimates, though the 95%
confidence limits contained a wide range that extended to infinity. For all five of these
populations, the independent comparison of na and ni found no significant differences
between the observed and expected values, using a x2 test with one degree of freedom.
For four of the five culture populations, the estimated effective population sizes are
well within the expected range of Np, the average number of abalone used to spawn each
hatchery-produced generation. However, the analysis of PMI spawn 1 does show a
discrepancy between these two values. Nk is between 2.13 and 28.42 while Nb is
between 61 and 66.
For the three natural populations collected in 1992, the number of generations
removed from the Santa Barbara 1979 population is based upon the size of the abalone
collected in 1979 and in 1992 and upon the growth rates of abalone in the wild. It is
evident that if the abalone collected from San Miguel Island in 1992 had been separated
from the rest of the natural Santa Barbara Channel Island since 1979 (1-2 generations) then
the allele-frequency variance was high (F = O.128) and the effective population size was
very low, having a 95% chance of being between 1.09 and 21.39 individuals. The genetic
drift analysis was as expected for the two northern California populations, with very low
allele-frequency variances (0.019 and 0.014) and subsequently high estimated effective
population sizes that include infinity in their 95% confidence limits. For two of these three
natural populations, the independent comparison of na and nt found no significant
differences, but a significant difference was found for the Fort Ross population. However,
this large x2 value can be explained by the small value for the expected number of alleles
lost (0.164).
Discussion
In the genotype variation analysis, three of the cultured populations had one locus
with a genotype frequency below the 5% probability required to rule out Hardy-Weinberg
equilibrium. The 1992 Santa Barbara population had two loci that were not in Hardy-
Weinberg equilibrium. This is the first, but definitely not the last, indication that this Santa
Barbara population is different than the other populations collected directly from the ocean.
Intuitively, a farm would seem to have a smaller population size and less random
mating than in nature. This could cause inbreeding, explaining the few deviations from
Hardy-Weinberg equilibrium found in those populations. Simply defined, inbreeding is
measured by a decrease in heterozygosity (Mitton, in press). However, though there was a
slight trend towards lower values for the mean number of alleles per locus in the cultured
populations as compared to the natural populations, the difference was not statistically
significant. Only in the 1992 Santa Barbara population does the low mean number of
alleles approach statistical significance. This loss of rare alleles and potential decrease in
mean alleles per locus in the cultured and introduced populations as compared to wild
populations could indicate a population bottleneck.
In an initially perplexing conflict, this potential loss of alleles has not yet led to any
decrease in heterozygosity. The cultured populations exhibited expected and observed
levels of heterozygosity that do not differ from the natural populations and do not have any
significant difference between them. However, the 1992 Santa Barbara population actually
exhibited much more heterozygosity than predicted by Hardy-Weinberg equilibrium. This
phenomenon - loss of alleles yet increase in heterozygosity - has been documented in
other cultured marine invertebrates, including hard clams, <span style='background-color:#cfc' title='Mercenaria mercenaria : Exact'>Mercenaria mercenaria</span> (Dillon
and Manzi, 1987) and oysters, <span style='background-color:#cfc' title='Crassostrea virginica : Exact'>Crassostrea virginica</span> (Vrijenhoek et al., 1990).
The increases in heterozygosity in the 1992 Santa Barbara population could have
resulted from a founder effect leading io genetic drift or from selection. While it is
theoretically possible that there could be strong selection pressures in the San Miguel area
for heterozygosity at the PGI locus, it seems very unlikely that a change this dramatic could
occur in only 1-2 generations by this means alone. (The present analysis of genetic drift
assumes that selection has not occurred.) The abalone that produced this population in
Santa Barbara (whether the spawn occurred in nature or in a hatchery) could have, by
chance, contained individuals with a higher concentration of the second-most common
allele. While losing overall allelic richness, the allele evenness could increase. For
example, the population from Santa Barbara in 1979 had allele frequencies for PGl of
A:O.890, B:0.073, and C:0.037 while in 1992 the Santa Barbara population in question
had PGI allele frequencies of A:0.500 and B:0.500. Though one allele was lost,
heterozygosity was greatly increased as was allelic evenness. As the departure from
Hardy-Weinberg equilibrium is signisicant, this situation is most likely with an extremely
small number of parents. This founder effect can then be analyzed as genetic drift.
A more accurate look at the changes in genetic diversity that could happen over time
with aquaculture and/or enhancement programs is found by quantifying genetic drift. As
discussed previously, decreases in heterozygosity may not be the only indication of
significant changes in genetic composition resulting from reproductive isolation or small
population size. In measurements of heterozygosity, an increase in the evenness of
frequencies of the remaining alleles can compensate for the loss of rare alleles in a
population bottleneck. Since population bottlenecks often affect overall allelic diversity
more than they affect heterozygosity, it makes sense to analyze the shifts in allele frequency
between progenitor and derived populations. In a study comparing hatchery and natural
oyster populations and in a re-analysis of the data from several previously published
electrophoretic studies (Hedgecock et al., 1992), it has been shown "that superficial
similarities in levels of genetic diversity scan belie significant genetic changes in cultivated
stocks." (Hedgecock and Sly, 1990)
This is obviously the case with the 1992 Santa Barbara population, for although it
exhibited an increase in heterozygosity, the genetic drift analysis indicates that this
population most likely results from fewer than 22 parents. This is the strongest indication
that these abalone were indeed the product of an enhancement program. First, the abalone
hatcheries that produced abalone seed for the various abalone enhancement programs have
been known to use as few as 2-4 highly fecund abalone to produce thousands of stock
(Oakes, McMullen, McCormick, personal communication). Secondly, no other allozyme
study of native Haliotis rufescens populations has ever indicated any type of localized
genetic variation from one cove to the next. This present study shows very little genetic
differentiation between the 1979 southern and 1992 northern California abalone
populations, Hedgecock's 1979 data showed very little genetic differentiation between
abalone populations on various Channel Islands, and a third study showed very little
variation in abalone populations that were sampled from all along the California coast
(McKean, 1980).
As for the cultured abalone at PMI, the conclusions from the genetic drift analysis
are not nearly as dramatic for these populations as they are for the Santa Barbara
population. Perhaps what is most interesting to note is that by purchasing several different
spawns of seed stock and brood stock from a variety of abalone hatcheries, PMI managed
to inadvertently lessen any inbreeding that might have been present in any one hatchery-
produced spawn. The female broodstock, which has been formed from a number of
spawns from a variety of hatcheries, had a high estimated effective population size. The
male abalone sampled, probably resulting from as few as one or two spawns, had a very
low estimated effective population size. Figure 4 presents the origins of these two
populations.
The fairly low variance in allelic frequency found between the progenitor population
and the female abalone and the fairly high variance of the male abalone are contrasted with
the moderate shift in the three spawn populations. Though the three spawns were not
produced from the males sampled, this could indicate that the males that were spawned
have experienced a more intense population bottleneck than the female broodstock.
However, Figure 4 demonstrates that the males used in the 1991 spawns were made up of
abalone produced in a large number of hatchery spawns, just like the female broodstock. If
these males had an effective population size comparable to that of the female broodstock,
then the lower effective population sizes of the three spawns could indicate that an amount
of genetic diversity has been lost with each subsequent generation. This effect could most
likely be counteracted by mixing spawns together. Additionally, the discrepancy between
Nb and Nk in PMI spawn 1 could indicate that not all of the abalone used for each hatchery
spawn have contributed to the production of offspring. Overall, it appears as if little
inbreeding can be detected in these culture populations after analyzing both the present
genetic variation and the genetic drift.
Conclusion
Aquaculture
The present study indicates that there are significant differences between the genetic
compositions of the Santa Barbara 1992 population and the natural populations, but only
slight differences between the cultured and natural populations. The short history of
abalone aquaculture paired with the long abalone generation time has prevented much
inbreeding and much genetic drift. However, it is always better to begin examining
potential losses in genetic diversity too early rather than too late. During the course of this
study, PMI realized that their present breeding practice: using a separate female broodstock
and spawning gravid males pulled from the production tanks - could not remain
permanent, for eventually the males pulled from the production tanks would be the progeny
from some of PMI's earlier spawns and they could potentially be performing back crosses.
Perhaps PMI could start using some wildstock male abalone in their breeding program, for
using natural abalone along with a large brood stock would minimize inbreeding and
genetic drift. The natural abalone would reintroduce rare alleles and the large brood stock
would help to maintain genetic diversity.
Additionally, evidence from PMI spawn 1 indicates a possible discrepancy between
the number of abalone thought to be contributing to spawns and the number that actually
are. Gaffney et al. (1992) found this phenomenon in oyster hatcheries and noted that this
problem could be alleviated by replacing large group spawns with a greater quantity of
spawns with fewer individuals. Abalone hatcheries could consider this breeding technique.
Finally, as abalone aquaculturists have begun to perfect their hatcheries, the time
might be appropriate for them to work together to begin to select for desired abalone traits
and start to breed various strains. (Keeping in mind that an outbred natural or cultured
population should remain as an antidote for any adverse inbreeding effects that could result
from domestication.) (Newkirk, 1983; Lester, 1983)
Enhancement
In a striking example of how quickly genetic diversity can be altered, the genetic
drift analysis shows dramatic changes in the allele frequencies of the 1979 and 1992
abalone populations from near San Miguel Island. This change is thought to be the result
of human intervention. The present data show that for such a large change to occur in such
a short time frame, then this population would have to be the result of fewer than 22
spawning adults. As explained previously, no other allozyme study of natural red abalone
populations has ever found any information to indicate that any genetic differentiation exists
between natural abalone populations, so it seems more plausible to assume that these could
be hatchery produced abalone that were introduced to the area several years ago. Between
the late 1970's and the early 1980's, a variety of groups, including the California Divers
Association, the California Department of Fish and Game, Atlantic Richfield Company,
and recreational divers groups, sponsored abalone enhancement programs which resulted
in hundreds of thousands of hatchery-produced abalone being seeded in southern
California water from Palos Verdes to many of the Channel Islands, including San Miguel.
It seems as if a controversy was inadvertently unearthed during the course of this
present study. The California Department of Fish and Game determined that enhancement
was not successful and therefore not to be relied upon as a way to protect this valuable
natural resource, but the professional abalone divers disagree. Many feel that the
Department of Fish and Game studies were merely inconclusive - due to some poor site
choices for abalone enhancement and to the great difficulties involved in being able to track
and find abalone in natural environments months and years after introducing them. The
divers experience with abalone has led them to believe that small and mid-sized abalone
tend to hide in rock crevices and only reappear to be easily observed when they have grown
much larger. Despite relating very low live seed recovery (£2.8%) for five large-scale
seeding experiments with red and green abalones, one Fish and Game study allows that
16
swamping the system with miniseed (very small, inexpensive abalone) may be an
economically feasible method of enhancement (Tegner and Butler, 1989).
The most complete study of red abalone seeding was conducted on the Palos
Verdes peninsula (Tegner and Butler, 1985). A total of 674 red abalone of two different
sizes and originating from two different hatcheries were planted in the study site in May
1981. One year later, only 1% of the live seed was recovered. Seed mortality was
estimated at 43% based upon the shells collected. Although over one half of the population
was not accounted for, the 1% live recovery rate did not give much encouragement to the
feasibility of future abalone enhancement. However, some evidence indicates that this
study site, while accessible to the researchers, did not provide the best habitat for abalone
and was already too damaged by human development. At the time of the planting, no
emergent (visible to divers without turning over rocks) abalone were found in 700 m2.
The average density of the other, cryptic (hidden) red abalone was very low. Density (1
S.E.) was 0.17 + 0.12 per square meter for 70 m2 area sampled.
This was not the only test site used to examine the feasibility of red abalone
enhancement. In the spring of 1979, several sites on the Channel Islands were examined to
select a suitable site for planting seed (DeFelice, 1979). Tyler Bight, on the south coast of
San Miguel Island, was selected as the planting site as it offered "excellent food supply
physical habitats, and workable depths." (Butler, June 6-11 1979). The June 1979 survey
of the habitat found both emergent and cryptic abalone. Additionally, the density of red
abalone was calculated in three different areas where 46, 70, and 90 m2 samples were
performed (Butler, June 19-25 1979). The density values were 1.09,0.70, and 1.00
abalone per m2, which is nearly a factor of ten greater than the abalone density in the Palos
Verdes sample (Table 6). This indicates that San Miguel Island offered a much better
habitat for red abalone than the mainland test site. In August 1979, 20,000 red abalone
seed were planted in two sites at Tyler Bight (Butler, August 1-8 & 16-22 1979).
However, bad ocean conditions prevented the survival rate from being determined. In
October of 1980, the Kelp Bass vessel went to San Miguel Island, but "large ground swells
prevented any sampling to be accomplished at San Miguel on two occasions during the
week" (Butler). It appears that the sampling was never performed.
While originally only anecdotal evidence from the abalone divers indicated that
abalone seeding could be successful, the present study provides some new, albeit
circumstantial, evidence that the 1979 seeding did provide abalone for the diving industry
1) According to John Colgate, the president of the California Abalone Divers Association,
the south side of San Miguel Island contains the largest bed of abalone in southern
California (personal communication). Anecdotal evidence among the divers indicates that
part of the prosperity of abalone in this region is due to the 1979 seedings. Additionally,
the divers who collected the abalone analyzed as the 1992 Santa Barbara population in this
study claimed that these samples resulted from enhancement (Westlotorn, personal
communication). 2) A study of red abalone growth rates determined that it took abalone in
southern California an average of 15 years, with considerable variability, to reach the
commercial minimum size of 197 mm (Tegner et al., 1992). The abalone planted in 1979
were around 30 mm (Butler, August 1-8 & 16-22 1979), indicating an age of
approximately one year. As the Santa Barbara population was sampled in 1992, then any
survivors from the seeding would be 14 years old. This is very comparable with the length
of time determined for abalone growth to minimum legal size. 3) It is very likely that the
abalone seed obtained for the 1979 seedings could have been produced from a very small
number of parents. Unfortunately, they were bought from a company that is now defunct,
Monterey Abalone Farms, so the exact hatchery records could not be obtained. However
several of the present abalone hatcheries have indicated that, if necessary, they could (and
have) obtained tens of thousands of abalone seed from as few as two large, fertile
individuals (Oakes, MCMullen, McCormick, personal communication). If the abalone
sampled were the seeded individuals, then a spawn of several females who were
homozygous for A at the PGI allele and one male who was homozygous for B at the PGI
allele could have produced offspring with allele frequencies like what was observed.
In conclusion, there are several possible explanations for the unusual genetic
variation found in the 1992 Santa Barbara population of abalone, including selection and
genetic drift. Genetic drift could have occurred naturally at that site, or could be the result
of a hatchery spawning only a very small number of individuals. Although an
electrophoretic study of the Australian blacklip abalone, <span style='background-color:#cfc' title='Haliotis rubra : Exact'>Haliotis rubra,</span> found regional gene
pools for these populations (Brown, 1991), no study has ever found any genetic variation
by distance between different red abalone populations. The present study shows very little
difference between samples collected in southern California in 1979 and in norther
California in 1992 and between these natural populations and several cultivated
populations. The most plausible explanation for the allele frequencies found in the 1992
Santa Barbara population is that these abalone were the result of the 1979 seeding. This
study indicates that abalone enhancement could be successful in California, possibly
helping to reverse the present decline in natural stocks. Further research could quantify the
effectiveness of seeding. In the course of this present work, several rare alleles were found
in individuals from the PMI broodstock that could be helpful to future studies. These
animals could be spawned and their offspring would then contain allozyme markers that
would indicate whether they were outplanted individuals with a greater than 90% assurance
(Hereford et al., 1992).
This study raises a final point. Although it is definitely possible to produce tens of
thousands of seed from only a handful of the highly fecund adult abalone and although it
appears to be possible for these hatchery-bred individuals to grow to commercial harvesting
size in the wild, the researcher must consider the ecological effects of enhancement. Before
any hatchery-bred abalone were outplanted, Dennis Hedgecock's study was conducted in
the late 1970's to ensure that there was no distinct genetic composition that varied from one
population to another. However, while great care was taken to ensure that no existing
population diversity would be altered by human intervention, it appears that the genetic
composition of the seeded individuals was not determined (until now). If enhancement
programs do occur in the future, this measure of biological diversity must be considered.
Population
Table 1
Summary of Sampled Populations
Description/Name
Location
PMI spawn 1
PMI, Santa Cruz
PMI spawn 2
PMI, Santa Cruz
PMI, Santa Cruz
PMI spawn 3
broodstock/
PMI, Santa Cruz
PMI females
adult males/
PMI males
PMI, Santa Cruz
native N. Calif. abs./
Gerstle Cove
Gerstle Cove
native N. Calif. abs./
Fort Ross area
Fort Ross
Santa Barbara abs./
south side of
SB 1992
San Miguel
six sites in the
Santa Barbara abs.
SB area, including
SB 1979
north and south
sides of San Miguel
Date Collected
April 1992
May 1992
July 1992
July 1992
July 1992
May 1992
May 1992
May 1992
1978-1979
Population
PMI 1
PMI 2
PMI 3
PMI females
PMI males
Gerstle Cove
Fort Ross
SB 199.
SB 1979
Table 2
Summary of Allozyme Analysis
Tissue sampled
Locus
foot
PGI
GOT
PGM
MDH
PGI
foot
GOT
PGM
MDH
PGI
foot
GOT
PGM
MDH
mantle
PGI
GOT
PGM
MDH
PGI
mantle
GOT
PGM
MDH
PGI
mantle
GOT
PGM
MDH
PGI
mantle
GOT
PGM
MDH
shell muscle
PGI
GOT
PGM
MDH
PGI
mantle
GOT
PGM
MDH
Sample size
156
﻿156
﻿156
156
120
120
120
110
﻿122
60
208
169
﻿138
Table 3
Allele Frequency Chart
Allelic frequencies. Asterisks indicate significant (pe0.05) deviations from Hardy-
Weinberg equilibrium (using chi-square test with pooling of rare alleles when appropriate
and using Levene’s correction for small sample size).
PG
GOT
PGM
MDH
Population
Allele
593
.990
.990
PMI 1
279
.010
215
.010
128
﻿592
959
871
PMI 2
338
129
.041
.204
071
430*
914
987
PMI 3
.852
455
135
.086
.013
.115
﻿012
95.
90
.884
.360
PMI Females
047
093
.488
105
﻿152
﻿012
PMI Males
195
﻿692
205
.013
102
013
979
.307
893
Gerstle Cove
086
521
107
021
150
.007
014
007
967
﻿326
848
946
Fort Ross
﻿576
141
.054
033
098
011
21
500*
SB 1992
.975
1.000
17
500
025
.008
SB 1979
989
266
﻿856
968
.120
592
011
﻿032
014
130
010
012
Table 4
Summary Statistics of Genetic Variation
Mean number of individuals assayed per locus (S), mean number of alleles per locus (ni),
mean heterozygosity observed per locus (Ho), mean heterozygosity expected per locus
under Hardy-Weinberg equilibrium (He), and the difference in heterozygosities
(calculated as D-Ho - He/He).
1979
PMI PMI PMI PMI PMI Gerstle Fort
1992
Statistics
3 females males Cove Ross SB SB
Number of
Loci Studied 4 4 4 4 4 4 4 4 4
﻿46
60 148.3
156 117.5 121.3 43
(0) (2.5) (0.8) (0) (0) (0) (0) (0) (27.4)
(S.E.)
3.0
2.5 2.0
3.0
2.3
2.5 2.5
2.5
2.3
(0.3) (0.3) (0.3) (0.3) (0.5) (0.7) (0.3) (0.4) (0.6)
(S.E.)
Ho
289 .267
340 .239 245 .375 .208
.255 .281
(141) (084) (.147) (101) (.120) (115) (.104) (231) (111)
(S.E.)
259 .272
314 .255 247 227 225
He
233 .291
(.131) (095) (.122) (117) (101) (125) (112) (121) (.124)
(S.E.)
094 -034 .116 -018 .083 -.063 -008 652 -.076
Table 5
Genetic Drift Analysis Summary
Estimates of temporal variance in allelic frequencies (F) and variance effective population
size (Np). St is the sample size of the derived populations (mean over loci); t is the number
of generations separating the derived populations from the progenitor population, Santa
Barbara 1979; Np is a range estimating the mean breeding number per generation; np is the
number of alleles in the progenitor population, na is the number of alleles in St, and nt is the
number of alleles expected to remain in a population of size Ny after t generations of random
drift; goodness-of-fit of na vs nt is tested by x2 with one degree of freedom (note that this
value is not reliable when the expected number of alleles remaining or lost is «0.2).
Interval
Nb
np,na x2,1d.f.
(95% c.l.s) range
n navsnt
SB 1979"-males PMI
4.16
12,10
0.136
0.924
5-21
(1.03 - 10.86)
(7.9
8.33
(2.07 -21.72)
1
41.87
120-430
SB 1979-females PMI 43
0.027
12,10
1.571
(5.98 - inf.)
(11.4)
83.74
(11.96 - inf.)
SB 1979-PMI spawn 1
156
2 0.130
8.10
61-66
0.140
12,9
(2.13 - 18.95)
(7.9
12.16
(3.19 -28.42)
SB 1979-PMI spawn 2 117,5 2 0.087
12,9
0.000
12.51
48-51
(3.19 -30.85)
(8.5)
18.76
(4.79 -46.28)
51.87
89-99
12,10
SB 1979-PMI spawn 3 121.3
2 0.027
0.071
(10.8)
(11.06 - 208.99)
77.78
(16.58 -313.49)
60
1
4.30
12,8
0.082
SB 1979-SB 1992
0.128
(1.09 - 10.70)
(8.0)
8.60
(2.18 -21.39)
70
0.332
SB 1979-Gerstle Cove
0.018
65.18
12,12
1
(10.05 - inf.)
(11.8)
130.36
(20.09 - inf.)
11.038
46
0.014
12,10
SB 1979-Fort Ross
(11.8) p «0.05
(13.55 - inf.)
inf.
(27.11 - inf.)
* SB 1979 mean sample size is 148.3
Table 6
Comparison of Two Red Abalone Seedings*
Palos Verdes
Tyler Bight. San Miguel
Density of Abalone
before seeding
(abalones/m2)
0.17 + 0.12
1.09, 0.70, 1.00
Emergent abalone
visible before?
yes
no
46m2, 70m2, 90m2
70m2
Area surveyed
f of abalones seeded
﻿674
20,000
August 1979
May 1981
Date of seeding
% of seed recovered
1% live, 43% of shells
never sampled
* data from Tegner & Butler, 1985 and cruise reports: Butler, 1979-80
Figure 1
California Red Abalone Landings (1931-1988)
Commercial red abalone landings from California for 1931 to 1988. For 1951 to 1988,
data is broken down into south-central California and the San Francisco area.
(figure from Tegner et al., 1992)
1800-
• SAN FRANCISCO AREA
• <span style='background-color:#cfc' title='South-central : Exact'>SOUTH-CENTRAL</span> CALIFORNIA
1600-
a ALL CALIFORNIA
1400
1200-

8 000-
800
600-
400-


200


o+
1970
﻿1950
930
YEAR
26
1990
Figure 2
Map of Collection Sites
ORICON
-------------
CALIFORNIA
<span style='background-color:#cfc' title='Sonoma : Exact'>Sonoma</span> County
San Francisco Bay
Monterey Bay
A-Pacific Mariculture Incorporated
B -Gerstle Cove
C - Fort Ross
D - San Miguel Island
Santa Barbard

P.
CHANNEL ISLANDS
PACIFIC
OCEAN
mies 20
0 attias 80
——
--—
—
BAJA
MALIFORNIA
27
1.0
0.8 -
0.6:
0.4
0.2

0.0-
0.2 +
ka
Figure 3
D-values by Locus
gtt
—4
Sample Populations
— PG
—— O
—n— PO
—0— MH
28

Bibliography
Brown, L.D. (1991). Genetic variation and population structure in the
Blacklip Abalone, Haliotis rubra. Australian Journal of Marine and
Freshwater Research 42: 77-90.
Butler, Robert. (June 6-11, 1979). Cruise report 79-KB. Experiemental Abalone
Enhancement Program California Department of Fish and Game
Operations Research Branch, Long Beach.
Butler, Robert. (June 19-25, 1979). Cruise report 79-X-4. Experiemental
Abalone Enhancement Program California Department of Fish and
Game Operations Research Branch, Long Beach.
Butler, Robert. (August 1-8, 1979). Cruise report 79-A-5. Experiemental
Abalone Enhancement Program California Department of Fish and
Game Operations Research Branch, Long Beach.
Bulter, Robert. (August 16-22, 1979). Cruise report 79-A-6. Experiemental
Abalone Enhancement Program California Department of Fish and
Game Operations Research Branch, Long Beach.
Butler, Robert. (October 6-10, 1980). Cruise report 80-KB-2. Experiemental
Abalone Enhancement Program California Department of Fish and
Game Operations Research Branch, Long Beach.
Colgate, John. President of the California Abalone Divers' Association. Santa
Barbara. personal communication, September 1992.
DeFelice, Richard. (March 26-30, 1979). Cruise report 79-X-7. Experiemental
Abalone Enhancement Program California Department of Fish and
Game Operations Research Branch, Long Beach.
Dillon, R. and Manzi, J. (1987). Hard clam, Mercenaria mercenaria,
broodstocks: genetic drift and loss of rare alleles without reduction in
heterozygosity. Aquaculture 60: 99-105.
Ebert, E. and Houk, J. (1984). Elements and innovations in the cultivation of
red abalone, Haliotis rufescens. Aquaculture 39: 375-392.
Fujino, K. (1978). Genetic studies on the Pacific Abalone <span style='background-color:#cfc' title='Haliotis discus hannai : Exact'>(Haliotis discus
hannai</span> ) - I: Inbreeding and overdominance as evidenced by
biochemical polymorphism in a wild population. Bulletin of the
Japanese Society of Scientific Fisheries 44(4): 357-361.
30
Fujino, K. (1978). Genetic studies on the Pacific Abalone <span style='background-color:#cfc' title='Haliotis discus : Exact'>(Haliotis discus</span>
hannoi Ino) - II: Excessive homozygosity in deficient animals. Bulletin
of the Japanese Society of Scientific Fisheries 44(7): 767-770.
Fujino, K. and Nagaya, N. (1977). Biochemical polymorphisms in the Pacific
Oyster <span style='background-color:grey' title='Crassostrea gigas : Fuzzy'>(Crassostre gigas</span> ) -I: Variants in myogen and esterases.
Bulletin of the Japanese Society of Scientific Fisheries 43(8): 983-988.
Fujino, K. and Nagaya, N. (1977). Biochemical polymorphisms in the Pacific
Oyster (Crassostre gigas) -II: Variants in Tetrazolium Oxidase and
Leucine Aminopeptidase. Bulletin of the Japanese Society of Scientific
Fisheries 43(12): 1455-1459.
Fujino, K. and Sasaki, K. (1984). Age association of genotypic proportions of
isozymes in the Pacific Abalone (Haliotis discus hannai). Bulletin of the
Japanese Society of Scientific Fisheries 50(1): 11-15.
Gaffney, P.M., Scott, T., Koehn, R., and Diehl, W. (1990). Interrelationships of
heterozygosity, growth rate and heterozygote deficiencies in the Coot
Clam, <span style='background-color:#cfc' title='Mulinia lateralis : Exact'>Mulinia lateralis.</span> Genetics 124: 687-699.
Gaffney, P.M., Davis, C.V. and Hawes, R.O. (1992). Assessment of drift and
selection in hatchery populations of oysters (Crassostrea virginica).
Aquaculture 105: 1-20.
Hedgecock, D., Chow, V. and Waples, R.S. (1992). Effective population
numbers of shellfish brood stocks estimated from temporal variance in
allelic frequencies. Aquaculture 108: 215-232.
Hedgecock, D. and Okazaki, N.B. (1984). Genetic diversity within and between
populations of American oysters <span style='background-color:#cfc' title='Crassostrea : Exact'>(Crassostrea).</span> Malacologia 25(2): 535-
549.
Hedgecock, D. and Sly, F. (1990). Genetic drift and effective population sizes
of hatchery-propagated stocks of the Pacific oyster, <span style='background-color:#cfc' title='Crassostrea gigas : Exact'>Crassostrea gigas.</span>
Aquaculture 88: 21-38.
Hereford, L., Powers, D., Kirby, V., and Scriviani, P. (1992). A proposal to the
California Abalone Divers' Association.
Lester, L. James. (1983). Developing a selective breeding program for penaeid
shrimp mariculture. Aquaculture 33: 41-50.
McBride, Susan. (April-September 1992). Director of Pacific Mariculture, Inc.
hatchery. Davenport, CA. personal communication.
McCormick, Thomas. (September 1992). President of McCormick & Assoc.
Ojai, CA. personal communication.
Mckean, M. (1980). Electrophoretic analysis of four California red abalone
(Haliotis rufescens ) populations. for Animal Science 119 at UCDavis,
taught by Dennis Hedgecock.
McMullen, John. (September 1992). President of Ablab. Port Hueneme, CA.
personal communication.
Mitton, J. (1992-3, in press). Theory and data pertinent to the relationship
between heterozygosity and fitness. In: The Natural History of
Inbreeding and Outbreeding, ed. by N. Thornhill. University of Chicago
Press, Chicago, IL, 1-45.
Newkirk, Gary F. (1983). Applied breeding of commercially important
molluscs: A summary of discussion. Aquaculture 33: 415-422.
Oakes, Frank. (September 1992). President of Abfarm. Cayucas, CA. personal
communication.
Saito, K. (1984). Ocean ranching of abalones (Haliotis discus hannai) and
scallops <span style='background-color:#cfc' title='Patinopecten yessoensis : Exact'>(Patinopecten yessoensis</span> ) in northern Japan. Aquaculture 39:
361-373.
Schaal, B.A. and Anderson, W. (1974). An outline of techniques for starch gel
electrophoresis of enzymes from the American oyster Crassostrea
virginica Gmelin. Georgia. Marine Sciences Technology Report Ser. 74.
Sietsema, T. (April 22, 1992). California’s abalone industry is thriving. San
Francisco Chronicle.
Swofford, D.L. and Selander, R.B. (1981). BIOSYS-1: A FORTRAN Program
for the comprehensive analysis of electrophoretic data in population
genetics and systematics. Journal of Heredity 72(4): 281-283.
Tegner, Mia J., DeMartini, John D., and Karpov, Konstantini. (1992). The
California red abalone fishery: A case study in complexity. In: Abalone
of the World: Biology, Fisheries, and Culture, ed. by S.A. Shepherd, M.J.
Tegner, and S. Guzman-del Proo. Fishing News Books, Oxford, 370-383.
Tegner, Mia J. (1989). The California abalone fishery: Production, ecological
interactions, and prospects for the future. In: Offprints from Marine
Invertebrate Fisheries: Their Assessment and Management, ed. by John
F. Caddy. John Wiley & Sons, Inc., 401-420.
Tegner, Mia J. and Butler, Robert A. (1989). Abalone seeding. In: Handbook
of Culture of Abalone and Öther Marine Gastropods. Kirk O. Hahn (ed.),
CRC Press, Inc., Boca Raton, Florida, 157-181.
Tegner, Mia J. and Butler, Robert A. (1985) The survival and mortality of
seeded and native red abalones, Haliotis rufescens, on the Palos Verdes
peninsula. California Fish and Game 71(3): 150-163.
Vrijenhoek, R.C., Ford, S.E. and Haskin, H.H. (1990). Maintenance of
heterozygosity during selective breeding of oysters (Crassostrea
virginica ) for resistance to MSX <span style='background-color:#cfc' title='Haplosporidium nelsoni : Exact'>(Haplosporidium nelsoni</span> ) disease.
Journal of Heredity 81: 418-423.
Wada, K.T. (1986). Genetic variability at four polymorphic loci in Japanes
pearl oysters, <span style='background-color:#cfc' title='Pinctada fucata martensii : Exact'>Pinctada fucata martensii,</span> selected for six generations.
Aquaculture 59: 139-146.
Westlotorn, Scott. (May 23, 1992). Processor of commercially harvested
abalone. Santa Barbara. personal communication.
</pre></body></html>