<html><body><pre style='margin:2rem auto;width:fit-content'>Detection of Sodium Channel Messenger RNA in the
Stellate Ganglion, Optic Lobe, and Mantle Muscle
of the Squid <span style='background-color:#cfc' title='Loligo : Exact'>Loligo</span> Opalescens
ABSTRACT
Different types of Na channels may be distinguished not only
between species but also within individual organisms. Investigations into
the differences between Na channel isoforms may be useful in elucidating
mechanisms by which Na channels carry out their specific functional roles
within cells. In this study, two RNA probes transcribed from segments of
CDNA encoding a putative squid giant axon Na channel (GFLNI) were
employed in in situ hybridizations and RNase protection assays to detec
Na channel messenger RNAs in the stellate ganglion, optic lobe and mantle
muscle of the squid <span style='background-color:#cfc' title='Loligo opalescens : Exact'>Loligo opalescens.</span> The results of this study indicate
the presence of additional Na channel mRNAs with partial sequence
homology to GFLNI mRNA in squid stellate ganglion and optic lobe, and
provide evidence for the expression of Na channel mRNA in mantle muscle
cells of <span style='background-color:#fcc' title=' : NoMatch'>L. opalescens.</span>
INTRODUCTION
Na channels-the voltage-dependent ionic channels responsible for
generation of action potentials-display remarkably similar Na currents
across different phyla (Shaller et al, 1992). Despite this conservation in
functional properties, important differences in toxin sensitivity, antibody
labeling and primary structure have served to distinguish different types
of Na channels not only between species but also within individual
organisms (Shaller et al, 1992). Putative Na channels cloned and
sequenced from the optic lobe (SOSR22/2-7; Sato & Matsumoto, 1992) and
giant axon (GFLNI; Rosenthal & Gilly, 1993) of two presumably closely
related species of squid <span style='background-color:#cfc' title='Loligo bleekeri : Exact'>(Loligo bleekeri</span> and Loligo opalescens,
respectively), for instance, display sequence differences in functionally
important regions of the channel protein and little overall amino acid
sequence identity (only 32%; Rosenthal & Gilly, 1993). Indeed, GELNI
demonstrates higher identity with Na channels found in vertebrate tissues
such as that isolated from electric eel electroplax (43%; Rosenthal & Gilly.
1993).
Little is presently known about multiple Na channel subtypes in
squid, but distinct Na channel species are likely to exist in different tissues.
Thus, GFLNI is expressed only at extremely low levels in the optic lobe of
L. opalescens (Rosenthal & Gilly, 1993), a complex region of the nervous
system undoubtedly rich in Na channels. In addition, voltage-clamp
expériments on L. opalescens in our laboratory have indicated the
presence of Na channels with distinct kinetic properties in several types of
non-giant-axon-forming squid neurons and in mantle muscle cells. These
Na channels may have a different molecular identity from the GFLNI-type
channels in the giant axon of the same species. Further investigation into
molecular differences between these Na channels will be useful in
elucidating the mechanisms by which Na channels fulfill their specific
functional roles in cells of many species.
In this study, two RNA probes transcribed from segments of CDNA
encoding GFLNI were employed in in situ hybridizations and RNase
protection assays to test for the presence of GFLNI-related Na channels
mRNAs in the stellate ganglion sincluding giant fiber lobe (GFL)I, optic lobe
and mantle muscle of L. opalescens. The results of these experiments
indicate the existence of additional Na channels with partial sequence
homology to GFLNI in the stellate ganglion and optic lobe of L. opalescens.
and provide additional evidence for the presence of a similar type of Na
channel mRNA in mantle muscle cells in this particular species of squid.
MATERIALS AND METHODS
Synthesis of RNA probes. CDNA encoding peptides 484-576 of the
second half of interdomain I-Il of GFLNI was released from pNCSETIO (a
vector created by Josh Rosenthal in our laboratory for fusion protein
generation in the development of Na channel antibodies) with Hind III and
Eco RI. This CDNA was then ligated into Bluescript II KS- phagemid
(double digested with Hind III and Eco RI) to create pHMSI. pNZS¬
(encoding the 3'end of GFLNI mRNA; see Fig. 1A) and pNC5' (encoding the
5'end of GFLNI mRNA; see Fig. 1A) have been described previously
(Rosenthal & Gilly, 1993). 358-labeled antisense HMSI (containing 282 nt
coding and 69 nt vector sequence), antisense NZ5- (containing 135 nt
coding, 102 nt untranslated and 30 nt vector sequence) and sense NCS
(containing 161 nt untranslated and 143 nt vector sequence) RNA probes
for in situ hybridizations were obtained from pHMSI, pNZS- and pNCS
respectively. Each was transcribed (after linearization with the
appropriate restriction enzymes; see Fig. 1A) with T7 RNA polymerase in
the presence of 100 uCi of 13581 UTP by use of a kit (Riboprobe system;
Promega). 32P labeled HMSI and NZ5- probes for RNase protection assay
were similarly transcribed from these plasmids in the presence of 125 uG
of 32PJ UTP.
In situ hybridizations. Adult stellate ganglion (including GFL), adult
optic lobe and 4-month-old mantle muscle tissues were dissected from live
squid (collected from Monterey Bay) and fixed for 1 hour in fresh, ice cold
4% paraformaldehyde in artificial sea water (ASW). Fixed tissue samples
were washed twice for 5 minutes in ASW, once for 5 minutes in a 1:1
dilution of ASW and PBS (pH 7.0), and once for 5 minutes in PBS (pH 7.0).
Samples were soaked twice for 40 minutes in 20% sucrose in PBS at 4°C (to
cryoprotect tissue from ice crystal damage during sectioning), frozen in
Polyfreeze (Polysciences. Inc.) using dry ice, and stored at -70'C. Tissue
was sectioned at 10 microns with a cryostat and affixed to silane coated
glass slides (Sigma).
In situ hybridizations with 358-labeled RNA probes were performed
on the tissue sections using the protocol outlined by Simmons et al (1989).
Following the hybridization procedures, slides were dipped in NBT-2
emulsion (melted and diluted 1:1 with dH20), allowed to dry vertically at
room temperature for 3 hours, stored at 4’C in light-proof containers with
desiccant for 12 days, and developed in Kodak D-19 for 4 minutes, dH20
stop bath for 15 seconds, and Kodak film strength rapid fixer for 4
minutes. Slides were then rinsed in running tap water for 1 hour, stained
in 0.5% cresyl violet and mounted with Permount (Fisher Scientific). For
each tissue type, one slide was stained with Multiple Stain (Toluidine Blue
and Basic Fuchsin; Polysciences, Inc.) and mounted with Permount
immediately after sectioning for comparison with the in situ hybridized
slides.
RNA extraction. Stellate ganglion (including GFL), optic lobe, mantle
muscle and gill tissues were dissected from live adult squid (collected from
Monterey Bay) and immediately frozen in liquid nitrogen and stored at
70C. Approximately equal amounts of each tissue were homogenized
with à motorized tissue homogenizer (Brinkman, Inc.) into 2 ml of
guanidine thyocyanate solution (2.6 M guanidine thyocyanate, 9.2 mM n¬
laurel sarcosine, and 12.5 mM sodium citrate). 1.5 ml of homogenate was
layered over 550 ml of 5.7 M cesium chloride and centrifuged in a
Beckman RP 55-S Swinging Bucket Rotor for 3 hours at 55,000 rpm and 23
C. The RNA pellet was resuspended in 100 ul of TE-1% SDS, extracted with
phenol/chloroform, and ethanol precipitated. RNA concentrations were
determined using a spectrophotometer, and a small aliquot of RNA from
each tissue type was run on a 1% agarose gel containing ethidium bromide
to ensure there was no degradation.
RNAse protection assay. 105 cpm of 32P-labeled antisense HMSI probe
were hybridized to 10 ug of each type of RNA in 3 ul of hybridization
solution (4 M Nacl, 400 mM PIPES, and 10 mM EDTA) and 24 ul of
deionized 100% formamide. In addition, 105 cpm of HMSI probe were
hybridized to 10 ul of 10 mg/ml tRNA as a negative control. All
hybridization mixtures were heated to 85C for 5 minutes and then
incubated at 48°C overnight. RNase digestions were carried out by addition
of 300 ul of RNase solution (300 mM NaCl, 5 mM EDTA, 10 mM Tris, 10
ug/ml RNase A, and 0.4 ug/ml RNase TI) to each sample and incubation at
room temperature for 1 hour. The reaction was stopped by addition of 20
ul of 10% SDS and 2 ul of proteinase K (25 mg/ml) and incubation for 15
minutes at 37°C. Each sample was then phenol/chloroform extracted,
ethanol precipitated using 25 ug of tRNA as a carrier, resuspended in S ul
of loading buffer (80% deionized formamide, IXTBE, and 0.1%
bromothymol blue), and run on a 6% denaturing acrylamide gel. 1000 cpm
of undigested HMSI and NZ5- probes were also run as markers. Kodak
XAR-S film using an intensifying screen was exposed to the gel overnight
at -70°C.
RESULTS
In situ hybridizations. Stellate ganglion (including GFL) is a known site
of GFLNI mRNA production (Rosenthal & Gilly, 1993) and was thus
selected as a positive control for both probes. In the stellate ganglion
sections, localization of probe was expected to be noted only in regions
containing cell bodies and not in regions consisting of neuropil (Fig. 24).
For both antisense probes (NZ5- and HMSI), this was seen to be the case
(Fig. 2B-C). Sections hybridized with sense NC5' probe displayed very little
background labeling (Fig. 2D).
Although both probes had a similar labeling pattern in the stellate
ganglion, an important difference in labeling between the two antisense
probes was apparent. Whereas NZ5- labeled GFL and non-GFL cell bodies
quite evenly (Fig. 2B), HMSI produced much stronger labeling in the non-
GFL portion of the ganglion (Fig. 20). This suggests that HMSI may detect
additional Na channel mRNAs not recognized by NZS¬
Probe-specific patterns of localized labeling were also evident in the
optic lobe with both antisense probes. In contrast, only very slight and
evenly distributed labeling by sense NC5' could be noted (Fig. 3B). NZS-
was observed to exclusively label very few cells along the inner granular
layer (Fig. 30). HMSI displayed a similar labeling pattern in the inner
granular layer, but also reliably labeled cells in the outer granular laver
and in the medulla (Fig. 3D-E). Thus, HMSI appears to label more cell
types than does NZ5- in the optic lobe as well as in the stellate ganglion.
In transverse sections of mantle muscle, labeling above background
of the glass slide was not observed with sense NC5' (Fig. 44). In contrast.
both NZ5- and HMSI exhibited labeling in the tissue sections above this
background level, although labeling appeared to be evenly distributed and
was not localized to any specific regions of the tissue (Fig. 4B-C).
RNase Protection Assay. RNase protection assays were performed in
which HMSI probe was hybridized with stellate ganglion (including GFL)
optic lobe, gill, or mantle muscle RNA, and with control tRNA, and then
digested with RNase. Both tRNA and gill (which is known to lack Na
channels) RNA served as negative controls; a band was not noted for either
of these. Three bands were observed for stellate ganglion; these included
an apparently fully protected band of 282 nt, and two partially protected
bands estimated to be 255 nt and 210 nt (Fig. 5). Bands were not noted
for either optic lobe or mantle muscles (even after additional 3 day
exposure; data not shown)
These hybridization results suggest that, in addition to an mRNA with
complete sequence identity to GFLNI mRNA in the second half of
interdomain I-Il (indicated by the fully protected band), at least one
additional mRNA species is expressed in the stellate ganglion that is highly
homologous (but not identical) to GFLNI.
DISCUSSION
The most important finding in the present study is that the HMSI
probe detects several types of putative Na channel mRNAs in stellate
ganglion and optic lobe tissues. This is indicated by the less specific
labeling of HMSI probe in in situ hybridizations (relative to that of NZS-
and by the appearance of several partially protected bands in RNase
protection assays employing HMSI probe. RNase protection assays
conducted by Rosenthal & Gilly (1993) on stellate ganglion and giant fiber
lobe RNA using antisense NZ5- probe detected only a single presumably
fully protected band for either tissue. The highly specific nature of the
NZ5- probe is not surprising, since it contains a large portion of 3
untranslated sequence for GFLNI. In contrast, HMSI probe consists
entirely of coding sequence and may thus be less specific than NZ5- and
capable of detecting Na channel mRNAs which are closely related to GFLNI
in the interdomain I-Il region. This explanation would account for the
differing results in the in situ hybridizations and RNase protection assays
with the two probes.
As noted in Figure 1B, HMSI probe corresponds to the second half of
interdomain I-Il of the Na channel protein, whereas NZS- probe relates to
the C-terminal end. Both of these regions lie in the cytoplasmic portions of
the protein (Fig. 1B). Comparisons of protein sequences in a variety of
isolated Na channels have shown that the cytoplasmic and extracellular
portions tend to be the most variable regions of the protein (Schaller et al.
1992). The degree of variability in the interdomain I-Il region for mRNAs
encoding squid Na channels requires further investigation, but the RNase
protection assay results imply that one or two species must be identical to
GFLNI over a sizable portion of this region.
Na channel isoforms containing related but not identical amino acid
sequences in interdomain I-Il could arise from transcription of separate
genes, alternative splicing of mRNAs, or posttranslational modifications of
the proteins themselves. In studies conducted on related Na channel
mRNAs identified in rat brain and rat muscle, several forms were found to
be the consequence of alternative splicing in a region corresponding to the
second half of the interdomain I-Il portion of Na channel protein (Shaller
et al, 1992). Alternative splicing in the first cytoplasmic loop of a Na
channel mRNA has also been noted in <span style='background-color:#cfc' title='Drosophila : Exact'>Drosophila</span> (Lougney et al, 1989).
In addition to being the site of structural diversity, the interdomain
l-Il loop is also the site of potential functional regulation. CAMP¬
dependent phosphorylation at multiple sites present in this region (Rossie
et al, 1987) reduces peak Na currents (Li et al, 1993). The possibility thus
emerges that alternative splicing in this region could selectively express
sites that act to regulate functional aspects of channels and thereby
cellular excitability. In addition, it has been postulated that variations in
cytoplasmic domains such as the interdomain I-Il loop could play a role in
specifying the localization of Na channels to different areas within cells
(Schaller et al, 1992). The Na channels related to GFLNI found in this
study in the <span style='background-color:#fcc' title=' : NoMatch'>L. opalescens stellate</span> ganglion and optic lobe show
considerable sequence homology in interdomain I-Il and may also result
from RNA splicing in this region. Again, changes in the amino acid
sequence in this cytoplasmic loop may endow Na channels with specialized
properties within each tissue type.
Clearly attributing a specific role to each Na channel isoform based
on the results of this study is impossible. It is likely, however, that at least
one of the putative Na channel mRNAs detected in the stellate ganglion and
giant fiber lobe using GFLNI related probes corresponds to the much
studied Na channel of the giant axon. The larger cells in the inner and
outer granular layer regions of the optic lobe have distinctive
morphologies and patterns of positioning and are thought to be involved in
early-stage sensory integration within the optic lobe, whereas the more
poorly characterized, smaller cells in the medulla carry out additional
integration and transmit outputs to other portions of the nervous system
(Young, 1974). Distinct labeling was observed with both antisense probes
only in the inner granular layer of the optic lobe in large cells spaced quite
far apart. This result indicates that a putative Na channel mRNA identical
to GFLNI may be present in these cells. This conclusion is further
supported by the RNase protection assay results of Rosenthal & Gilly
(1993) noting a single fully protected band of extremely low abundance in
optic lobe RNA using NZ5- probe. Additional Na channel mRNAs in optic
lobe detected by HMSI probe appear to be more abundant and may relate
to the SOSR22/2-7 Na channel CDNA isolated in optic lobe of a similar
species of squid (Sato & Matsumoto, 1992). The absence of bands relating
to optic lobe Na channel mRNAs in the RNase protection assay using HMSI
probe can be attributed to a low ratio of Na channel mRNA species to total
RNA. A longer exposure time in the RNase protection assay is most likely
necessary to detect signal for Na channel mRNA in this tissue.
In in situ hybridizations carried out on mantle muscle tissue, both
antisense probes demonstrated labeling above the background level
produced by the sense probe. This observation suggests that Na channel
mRNA related to GFLNI mRNA may be expressed in mantle muscle cells.
Localization of labeling to one of the two zones of identified circular muscle
fiber types of mantle muscle (Bone et al, 1981) was not noted, however,
and only diffuse labeling by either antisense probe in all portions of tissue
could be seen. This may be due to poor resolution in the detection of 32P
signal using emulsion coating of the slides. Additional in situ
hybridizations of mantle muscle employing a higher resolution technique
may be necessary to detect localization of probe within mantle muscle
tissue. As with optic lobe, the abundance of Na channel mRNA in mantle
muscle tissue is probably very low, and lengthy exposures are most likely
required in order to detect the presence of Na channel mRNA in mantle
muscle tissue using the RNase protection assay.
ACKNOWLEDGMENTS
1 would like to express my sincere gratitude to Professor Gilly for his
encouragement and guidance in focusing my ideas in my project
throughout the entire quarter. I owe an especially big thank you to Taylor
Liu for unselfishly devoting numerous hours helping me step by step
through techniques and for always providing great company and being an
invaluable source of advice. I am greatly indebted to Marie Perri for also
giving up much of her time helping me with the RNase protection assays
and for her cheerful companionship in the lab. I would like to thank Matt
McFarlane and Josh Rosenthal for their great senses of humor and always
ready willingness to help me out. Lastly, I would like to thank Simone,
Tony, <span style='background-color:#cfc' title='Zora : Exact'>Zora,</span> Adam and Don for helping make my experience in the Gilly lab
à fun and memorable one.
LITERATURE CITED
Bone, Q., Pulsford, A., & Chubb, A.D. (1981) Squid mantle muscle. J. mar
biol Ass. U.K. 61, 327-342.
Li, M., West, J.W., Numann, R., Murphy, B.J., Scheuer, T., & Catterall, W.A.
(1993) Convergent regulation of sodium channels by protein kinase C
and CAMP-dependent protein kinase. Science 261, 1439-1442.
Lougney, K., Kreber, R., & Ganetzky, B. (1989) Molecular analysis of the
para locus, à sodium channel gene in Drosophila. Cell 5 8, 1143-1154.
Rosenthal, J., & Gilly, W. (1993) Amino acid sequence of a putative sodium
channel expressed in the giant axon of the squid Loligo opalescens.
Proc. Natl. Acad. Sci. USA 90, 10026-100030.
Rossie, S., Gordon, D., & Catterall, W.A. (1987) Identification of an
intracellular domain of the sodium channel having multiple CAMP¬
dependent phosphorylation sites. J. Biol. Chem. 262, 17530-17535.
Sato, C., & Matsumoto, G. (1992) Primary structure of squid sodium
channel deduced from the complementary DNA sequence. Biochem.
Biophys. Res. Commun. 186, 61-68.
Shaller, K.L., Krezmien, D.M., Mckenna, N.M., & Caldwell, J.H. (1992)
Alternatively spliced sodium channel transcripts in brain and muscle.
J. Neurosci. 11, 918-927.
Simmons, D.M., Arriza, J.L., & Swanson, L.W. (1989) A complete protocol
for in situ hybridization of messenger RNAs in brain and other tissues
with radiolabeled single stranded RNA probe. JHistotech. 1 2(3), 169-
181.
Young, T.Z. (1974) The central nervous system of Loligo. Phil. Trans. of
Roy. Soc. of London 267, 268.
LEGEND TO FIGURES
Figure 1. A. GFLNI and related CDNAs employed in synthesizing probe
The approximate locations of Nde I, Hind III and Nco I restriction
enzyme sites used in linearizing pNZ5-, pHMS1, and pNCS'are
shown. Figures are not drawn to scale. B. Schematic
representation of the regions of GFLNI to which each probe
corresponds. Note that all three probes relate to cytoplasmic
portions of the protein.
Figure 2. All photos are taken at 4X magnification. A. 10 micron thick
section of adult L. opalescens stellate ganglion stained with Multiple
Stain (Toluidine Blue and Basic Fuchsin; Polysciences, Inc.). The
section consists of both the giant fiber lobe (I) and stellate ganglion
(11). B. Adult stellate ganglion section hybridized with NZS¬
antisense probe. Diffuse staining in the cell body regions of the
giant fiber lobe and stellate ganglion (*) may be noted. C. Adult
stellate ganglion section hybridized with HMSI antisense probe.
Diffuse staining in the giant fiber lobe (1) and heavy staining in the
stellate ganglion (II) are seen. D. Adult stellate ganglion section
hybridized with NC5' sense (negative control) probe; labeling is not
apparent.
Figure 3. Photos A, B, C and D are taken at 10X magnification; photo E at
20X magnification. A. 10 micron thick section of adult L.
opalescens optic lobe stained with Multiple Stain (Toluidine Blue
and Basic Fuchsin; Polysciences, Inc.). Dark stained regions include
the outer granular layer (1), inner granular layer (II), and medulla
(III). B. Adult optic lobe section hybridized with NC5' sense probe
only diffuse background labeling can be seen. C. Adult optic lobe
section hybridized with NZ5- antisense probe; the probe labeled
localized regions along the inner granular layer (*). D. Adult optic
lobe section hybridized with HMSI antisense probe. Localized
labeling is apparent along the inner granular layer (1), outer
granular layer (II), and in the medulla (III). E. Magnified view of
localized labeling by HMSI probe in the medulla of an adult optic
lobe section.
Figure 4. All photos are taken at 40X magnification. A. 4-month-old
mantle muscle section (transverse) hybridized with NC5' sense
probe. Labeling is not observed above the background labeling of
the glass slide. B. 4-month-old mantle muscle section (transverse
hybridized with NZ5- antisense probe. Diffuse labeling in the tissue
is notable above background labeling. C. 4-month-old mantle
muscle section (transverse) hybridized with HMSI antisense probe.
Diffuse labeling in the tissue above background levels is apparent.
Figure 5. Results from RNase protection assays on extracted RNAs using
52P-labeled HMSI probe (overnight exposure). Samples run include
undigested HMSI probe (A), undigested NZ5- probe (B), negative
control tRNA (C), stellate ganglion RNA (D), mantle muscle RNA (E)
optic lobe RNA (F), and negative control gill RNA. Bands are noted
only for undigested HMSI probe, undigested NZ5- probe, and for
stellate ganglion RNA. The three bands corresponding to stellate
ganglion consist of a presumably fully protected band (282 nt) and
two partially protected bands (approximately 255 nt and 210 nt).
The lower mobility of the fully protected band relative to the
undigested HMSI probe is due to removal of 69 nt of vector
sequence in the probe.
Figure 1
GFLNI
—
WA
NZ5
—
HI
HMSI
Ncol
NCS

Vuntranslated sequence
coding sequence
— vector sequence
B.
— —
oufside

inside
5
PHRS!


Ozawa/Na Channels
VA
NdeI

EcoRI
—17
0
+ ——
+-
N25-
﻿16
Figure 2
A.
Ozawa/Na Channels
Figure 2 continued
C.
Ozawa/Na Channels
Figure 3
A.
Ozawa/Na Channels
III
B.
Figure 3 continued
Ozawa/Na Channels
20
Figure 3 continued
E.
Ozawa/Na Channels
Figure 4
A.
Ozawa/Na Channels
22
Figure 4 continued
C.
Ozawa/Na Channels
Figure 5
Özawa/Na Channels

EF
</pre></body></html>