<html><body><pre style='margin:2rem auto;width:fit-content'>ABSTRACT
<span style='background-color:#cfc' title='Gillichthys mirabilis : Exact'>Gillichthys mirabilis,</span> the longjaw mudsucker, is a Gobiid fish that lives under a
wide variety of temperature ranges in bays and estuaries along the Pacific coast of North
America. A population genetics study could suggest whether genetic differences
associated with differential physiological adaptations have occurred. Samples of
Gillichthys mirabilis from three locations: Hayward, CA, San Diego, CA, and San Felipe,
Mexico, were analyzed using isozyme electrophoresis and Amplified Fragment Length
Polymorphism (AFLP), a recently developed DNA fingerprinting technique. Three
questions were posed: 1) Are the populations of Gillichthys mirabilis separate or
interbreeding? 2) If the populations are interbreeding, to what extent does interbreeding
occur?, and 3) Is AFLP a sensitive and reliable technique to use for a population genetics
study of this species?
In the isozyme work, staining for malate dehydrogenase revealed an allele that was
present only in the San Diego fish, suggesting little gene flow among the three
populations. Preliminary AFLP results show some differences between the Hayward and
San Diego samples, as well as the potential for AFLP to be a powerful technique. More
in-depth studies, however, will be needed to draw definitive conclusions about the
population structure of Gillichthys mirabilis
INTRODUCTION
Gillichthys mirabilis, the longjaw mudsucker, is a fish of the <span style='background-color:#cfc' title='Gobiidae : Exact'>Gobiidae</span> family. It is
commonly found in bays and estuaries along the Pacific coast of North America, from
Tomales Bay (about 50 miles north of San Francisco) to Baja California, as well as in the
Sea of Cortez (Fields, 1995) (Fig. 1). Samples from three locations: Hayward, California,
San Diego, California, and San Felipe, Mexico (Fig. 2), were analyzed to determine
whether the populations at these locations were separate or interbreeding. Tused two
different methods to analyze the samples: isozyme electrophoresis, which enabled me to
study the samples at the protein level, and Amplified Fragment Length Polymorphism
(AFLP), a recently developed DNA fingerprinting technique. A third purpose of the
project was to determine whether AFLP is a sensitive and reliable technique for studying
population genetics of this species.
<span style='background-color:#cfc' title='Gillichthys : Exact'>Gillichthys</span> can reside under a wide range of temperature and water conditions. It
can burrow in the mud to regulate its body temperature, and has been known to survive at
temperatures as high as 35° C, though it prefers temperatures from 9 to 23° C (Love,
1991). The temperature of its natural habitat can vary greatly. Annual temperatures on
the coast of Northern California might range from 10 to 18° C, while in the Sea of Cortez
the range might be from 9 to 30°C (Fields, 1995). Gillichthys can also survive in a wide
range of water conditions, ranging from water that is nearly fresh to water that is two and
a half times saltier than sea water (Love, 1991). A population genetics study of
Gillichthys might suggest differential physiological adaptations that have occurred among
Gillichthys populations, especially since it can live in such varied environments.
Current literature on the population genetics of Gillichthys is very limited. Fields
(1995) found no differences between the enzyme kinetics, amino acid sequences, and
CDNA sequences of Az lactate dehydrogenase in Gillichthys from the San Diego area and
the northern part of the Sea of Cortez. Based upon his studies, Fields suggests that
Gillichthys uses behavioral thermoregulation rather than biochemical adaptation to survive
in the warm northern Sea of Cortez and that the isolation of the California and Sea of
Cortez fish from each other may have occurred relatively recently. The study of one
protein, however, is not enough to draw conclusions about population genetics; more in-
depth studies are needed. Cathleen Davies (personal communication) is currently studying
the population structure of Gillichthys using protein electrophoresis, but she has not yet
published her results. No one has used the more sensitive DNA techniques to study
Gillichthys's population structure.
We decided to use both isozyme electrophoresis and AFLP to analyze the samples
of Gillichthys. Studying the fish at both the protein and DNA levels provided several
advantages. Isozyme electrophoresis is an established method from which it would be
likely for me to get results in the short eight week time frame of this study. Using a DNA
technique on Gillichthys, however, provides the potential to detect differences that protein
work probably could not, since with DNA fingerprinting, many loci are easily studied at
once and post-translational effects are not a factor.
THEORY
Isozyme Electrophoresis
Isozyme electrophoresis involves three main steps: extraction of protein, resolution
of the protein on a non-denaturing polyacrylamide gel, and staining for the activity of the
protein of interest (Fig. 3). When proteins are subjected to gel electrophoresis, they
migrate different distances based mainly upon their net charge and size. A more
negatively charged, smaller protein will migrate faster.
Because proteins with the same function that have been coded for by different loci
or by different alleles within a locus usually migrate at different rates and produce different
banding patterns, one can deduce the genotype of an individual for a particular protein
using isozyme electrophoresis. For example, the top section of Figure 4 shows a
monomeric protein. If an individual is homozygous for the faster allele, this is represented
by the single band which has migrated farther. If the individual is homozygous for the
slower allele, this is represented by the single band that has not migrated as far. Finally, if
the individual is heterozygous, it will have two bands, one for each allele. The situation is
similar for a dimeric protein, except that the heterozygote will have three bands: one for
each of the homodimers and one (the middle band) for the heterodimer, which consists of
one component of each of the homodimers. These patterns are due to allozymes, which
are encoded by allelic variations at a single locus. The situation increases in complexity
(five bands for the heterozygote of a tetrameric protein, etc.) as the number of subunits in
of the protein increases (Richardson et al., 1986).
With proteins that have more than one locus coding for them, each isozyme, as the
product of individual loci are called, will show a banding pattern as described above.
Proteins are present in different amounts in different tissue, so the number of and intensity
of isozymes will vary among tissue types (Richardson et al., 1986). In my study,
however, the same type of tissue was analyzed for each fish, so the type and intensity of
isozymes present should be similar.
The proteins are stained through a chemical reaction, where the enzyme of interest
catalyzes an oxidation-reduction reaction which is coupled with a dye (Richardson et al.,
1986)
AFLP
AFLP was developed by Vos et al. (1995). The method involves multiple steps:
isolation of genomic DNA, double restriction digest, ligation of adapters, preamplification,
selective amplification and labeling, and autoradiography (Fig. 5).
Genomic DNA is cut by a four base cutter (Msel) and a six base cutter (EcoRl).
<span style='background-color:#fcc' title=' : NoMatch'>Synthesized oligo</span> nucleotide adapters are then ligated to the ends of the restricted
fragments (Fig. 6). In the preamplification step, primers whose sequence are identical to
the adapter sequence and sticky end sequence with an additional random base on the 3
end are used. During the PCR reaction, the double stranded fragments with adapters on
the end denature, and the primers bind selectively to the fragments with the
complementary adapter/sticky end sequence as well as the complement to the random base
(Fig. 7). The PCR reaction amplifies these fragments. This provides selectivity by
amplifying only fragments with adapters followed by the complement to the random base.
In the selective amplification and labeling step, the primers are similar, except they
have an additional two random bases to increase selectivity. In this reaction, only the
fragments with both the complementary adapter sequence/sticky end sequence and three
bases complementary to the three on the primer are amplified (Fig. 8). This increases
selectivity again by amplifying only a subset of the fragments that were amplified in the
preamplification. Labeling of the fragments with radioactivity also occurs in this step, as
the EcoRl primer in this step is labeled with3
The product of the selective amplification and labeling step is resolved on a
polyacrylamide sequencing gel. Since each fragment is a unique size, it should migrate at
a unique rate and appear as a unique band when visualized. Figures 9 and 10 show what
AFLP banding patterns might look like. The gel is visualized by autoradiography.
In summary, AFLP produces a banding pattern that is different for each individual,
but that is more similar between more closely related individuals. The function and origin
of each amplified fragment is not known. Different banding patterns can be produced for
the same individual by using primers with different random bases.
Differences between individuals can be evaluated by calculating a difference index
= + different bands/+ total bands. Pair-wise comparisons could then be made, and
populations compared, by using one way ANOVA.
MATERIALS AND METHODS
Eight Gillichthys specimens were caught using baited minnow traps in the
estuaries around San Felipe, Mexico. Over thirty specimens each were caught in the same
manner from the estuaries around San Diego, California and in the Cargill salt ponds in
Hayward, California. The fish were kept in an aquarium which was filled with sea water at
ambient temperature (18° C). When the fish were needed for analysis, they were removed
from the aquarium and immediately stored in a -70° C freezer.
Isozyme Electrophoresis
As mentioned above, for the isozyme work the major steps involved extracting the
protein, resolving the proteins on a non-denaturing polyacrylamide gel, and staining the
gel for activity of the protein of interest (Lin, 1993) (Fig. 3). Two proteins were stained
for: lactate dehydrogenase (LDH), a glycolytic enzyme, and malate dehydrogenase
(MDH), an oxidative enzyme. We dissected approximately 0.25 grams of white muscle
tissue from each individual. The tissue was ground in 0.75 mL of homogenization buffer,
which consisted of 20% glycerol, 20 mM imidazole (pH 7.0), and 3 mM B¬
mercaptoethanol. The homogenized samples were spun in a 4° C centrifuge at 14000 g
for 30 minutes to 1 hour. The supernatant was immediately removed and stored on ice or
in a 4° C refrigerator until used. If the supernatant was stored for more than a few hours
before use, it was centrifuged again for 30 minutes at 14000 g immediately before use.
The polyacrylamide gel was composed of two layers: the running gel and the
stacking gel. The stacking gel solution for one gel was composed of 3.56 mL double
distilled (dd) H2O, 1.50 mL 4X stacking gel buffer (0.5 M Trizma base, pH 6.9), 0.60 mL
30% acrylamide/1% bis-acrylamide, 6 uL TEMED, and 40 uL 10% ammonium persulfate.
The running gel solution for one gel was made up of 4.725 mL dd HO, 4.125 mL 4X
running gel buffer (1.5 M Trizma base, pH 8.0), 4.125 mL 30% acrylamide/1% bis¬
acrylamide, 8.25 uL TEMED, and 0.15 mL 10% ammonium persulfate (Lin, 1993). The
running gel was poured into the gel plates and allowed to polymerize. The stacking gel
solution was then poured onto the polymerized running gel.
To resolve the samples, we placed the gel in a gel rack and filled the chambers with
electrophoretic buffer. The type of electrophoretic buffer varied depending on the protein
we planned to stain. For MDH the buffer consisted of 191 uM glycine, titrated with
Trizma base to pH 8.95, while for LDH the buffer was titrated to a pH of 8.3. I prepared
the sample solutions by adding 15 uL of loading buffer to 40 uL of the supernatant.
Blank solutions with water instead of supernatant were also prepared. For MDH gels, 55
uL of sample solution were loaded, while for LDH gels 25 uL were loaded.
The gels were run in a 4° C cold room, at 12 mA until the dye reached the running
gel layer at which time the voltage was increased to 25 mA, or at 12 mA overnight and 25
mA for a few minutes before the gel was disconnected.
When the dye front had run off the bottom of the gel, the electrophoresis was
stopped, and the gel was removed from the apparatus for staining. I made staining
cocktails with the following reagents: for MDH, 44 mL pH 9.0 staining buffer (0.2 M
Tris-HCl), 10 mg NAD', 100 uL (1.25 mg) phenazine methosulfate (PMS), 2 mL nitro
blue tetrazolium (NBT), and 1 mL 2 ML-malate pH 7.0. For LDH, 45 mL pH 8.0
staining buffer (0.2 M Tris-HCl), 10 mg NAD', 100 uL (1.25 mg) PMS, 2 mL NBT, and
110 uL lactic acid were mixed. The gel was placed in the staining solution at room
temperature until all the bands appeared; this usually required about 10 minutes. At this
point, the staining solution was replaced with dd HO and the gel was washed twice for 10
minutes.
AFLP
The main steps of AFLP were isolation of genomic DNA, double restriction digest,
ligation of adapters, preamplification, selective amplification and labeling, resolution on a
polyacrylamide gel, and visualization through autoradiography (Fig. 5). The protocol was
based upon the methods described by Vos, et al. (1995), and many of the modifications
were made upon suggestion by Tony DeTomasso.
Our main DNA extraction method was Macherey-Nagel Nucleospin C& T DNA
extraction from cells and tissue (see Macherey-Nagel for protocol). We also used a
phenol and 1:24 chloroform/isoamyl extraction method and a salt extraction method
(courtesy of Dr. Grant Pogson). All of these methods produced comparable results.
The double restriction digest was set up by incubating 500 uM DNA, 5 units
EcoRl, and 5 units Msel with 3 uL New England Biological (NEB) Buffer 2 and 3 uL
IOX NEB BSA. An appropriate amount of dd HO was added to make a final reaction
volume of 30 uL. Adapters (Vos et al., 1995) were ligated by incubating the double
digest product with the following reagents at room temperature overnight: 2 uL 20 mM
ATP, 4 uL 10X Boehringer-Mannheim ligation buffer, 1 uL 50 uM Msel adapter, 1 uL 5
MM EcoRl adapter, 1 uL Boehringer-Mannheim ligase, and 1 uL dd H2O. The single
strands of the adapters should be annealed together before use. This is done by heating
the appropriate concentration of adapter to 95° C for five minutes and letting the mixture
cool to room temperature before freezing it.
I made the preamplification reaction by first diluting the ligation product 2:3. 60
uL of O.IXTE (10 mM Tris-HCl and 0.1 mM EDTA, pH 7.5) were added to each
ligation product to make a total solution of 100 uL. Each preamplification reaction was
then made by mixing 2 uL 1OX PCR buffer, 2 uL 2 mM dNTP's, 0.5 uL 30 uM Msel
primer, 0.5 uL 30 uM EcoRI primer, 4 uL 2:3 diluted ligation reaction solution, 0.09 uL
Taq polymerase, and 10.91 uL dd H2O. The random bases for the Msel and EcoRl
primers were thymine and adenine, respectively. Before the PCR cycles were started, the
samples were heated at 72° C for two minutes. The PCR cycles were as follows: 94° C
for 45 seconds, 56° C for 45 seconds, and 72° C for 1 minute, all for twenty cycles. Al
mL solution of the lOX PCR buffer consists of 670 uL 1 M Trizma Base, 67 uL 1 M
MgCh, 83 uL 2 M(NH2)SO4, 7 uL 14 M B-mercaptoethanol, and 173 uL dd HO
NaOH was then added to adjust the pH to 8.6.
When the preamplification cycles were complete, I ran 8 uL of the preamplification
solution on a 1% agarose gel to check the efficacy of the preamplification step. The
remaining 12 uL of the preamplification solution were diluted 1:20 with O.IXTE. 3 uL
of this diluted preamplification solution were mixed with 1 uL IOX PCR buffer, 1 uL 2
mM dNTP's, 0.25 uL 10 uM labeled EcoRl primer (random bases AGG or ACT), 0.25
UL 30 uM Msel primer (random bases TGA or TAC), 0.05 uL Taq, and 4.45 uL dd H2O
to give a total reaction volume of 10 uL. We prepared the reactions on ice, spun them for
a few seconds in a microfuge, transferred the samples back to ice, and then transferred the
samples directly into a PCR machine preheated to 95° C. The 36 PCR cycles were as
indicated in Table 1.
The labeled EcoRl primers were prepared beforehand. For a 25 uL sample, 11.25
uL dd HO, 2.5 uL 100 uM primer stock, 2.5 uL polynucleotide kinase buffer, 7.5 uLy
5P ATP, and 1.25 uL T4 polynucleotide kinase (9.30 U/uL) were added together. The
solution was incubated at 37° C for 30 minutes and then at 65° C for 15 minutes. We then
stored the labeled primers in a lead pig in a freezer designated for radioactivity.
Aster the selective amplification and labeling step was completed, the samples were
prepared for loading. We added 10 uL loading dye to each sample and denatured the
samples at 95° C for five minutes. We then transferred the samples to ice and loaded 5 ul
of each sample into the gel. The gel was run at 55 W until the first dye front ran
completely off the gel, which took two to three hours. The dried gel was visualized by
autoradiography.
RESULTS
Isozyme Electrophoresis
On the gels that were stained for LDH, all of the individuals from Hayward, San
Diego and San Felipe showed the same simple two-band pattern. The lack of variation
between the San Diego and San Felipe populations is consistent with Fields's (1995)
results. Figure 11, with one individual from San Felipe and eight from San Diego shows
the banding pattern that was characteristic for all the individuals that were stained for
LDH.
All of the individuals from Hayward and San Felipe also showed the same banding
when stained for MDH. The slowest band is the darkest, and there are two lighter bands
below the darkest one (Fig. 2 and 3). The individuals from San Diego, however, show
variation. As can be seen in Figure 14, the San Diego individuals show variation in the
fastest band. The individuals show three variations, which are evident in the individuals
labeled SDI, SD2, and SD3. SDI shows the same banding patterns as the Hayward and
San Felipe fish. SD2, however, shows three bands instead of one at the fastest position.
SD3 shows one band at the fastest position, but it has migrated farther than it did than in
the Hayward and San Felipe individuals. SDI is therefore homozygous for the slower
allele, SD2 is heterozygous, and SD3 is homozygous for the faster allele. Table 2 shows
the number of individuals with each genotype for the populations. 10 individuals from
Hayward, 30 individuals from San Diego, and 8 individuals from San Felipe were
analyzed.
The San Diego population was tested for Hardy-Weinberg Equilibrium (Appendix
1). It was the only population tested since it was the only one that exhibited variation.
The population was almost exactly in Hardy-Weinberg Equilibrium, with a insignificant X¬
test statistic of 0.1391 (critical value was 3.841) and expected values that matched the
observed values almost exactly. (18 homozygous slow, 10 heterozygous, and 2
homozygous fast were observed, while 17.8, 10.6, and 1.6 were the expected values.)
AFLP
Figure 9 shows a gel obtained when we were trying to optimize the AFLP
conditions. All the lanes are from the same sample, but each lane reflects different
reaction conditions. Each lane shows the same bands, although some bands are lighter in
some lanes than others.
Figure 10 shows a gel on which there are several different individuals. H2 and H3
are individuals from Hayward while SD14, SD17, and SD19 are from San Diego. The
two Hayward fish have bands in common with each other, as do the three San Diego
individuals. The Hayward and San Diego fish also have bands in common with each
other, yet there are also several distinct differences between the banding patterns. One
difference is the bands indicated on the San Diego individuals which are lacking in the
Hayward fish.
DISCUSSION
Isozyme electrophoresis
All of the LDH gels showed a simple two-band pattern. The lighter band on top is
most likely the heart type homotetramer and the dark band is most likely due to the muscle
isoform.
MDH is a dimeric protein and is coded for by three loci: one mitochondrial locus
and two cytosolic loci. Hayward individuals, San Felipe individuals, and some San Diego
individuals exhibit a simple three-band pattern, one band for each isoform. The darkest
band on top is due to the mitochondrial isoform of MDH, and the two lighter bands below
are due to the two cytosolic isoforms of MDH. The second, faster cytosolic isoform
shows allelic variation in the San Diego population, with some individuals homozygous for
the slow allele, some heterozygous, and some homozygous for the fast allele.
The San Diego population did not significantly deviate from Hardy-Weinberg
Equilibrium expectations. This means that the assumptions underlying the Hardy¬
Weinberg Equilibrium, namely,: random mating between phenotypes, large population, no
selection between genotypes, no differential migration, and no mutation are generally
being met. This suggests that MDH is not under selection in San Diego. If it were being
selected upon, it could not be used as a marker for population structure and dynamics
because it would not be neutral.
The Hayward and San Felipe populations are fixed for the slow allele, suggesting
that there is very little or no gene flow among the three populations. The rationale behind
this conclusion is as follows: since the fast allele is not present in either the Hayward or
San Felipe populations, neither of these populations is interbreeding with the San Diego
population. Since San Diego is located south of Hayward and north of San Felipe, it is
highly unlikely those two populations would interbreed with each other if they were not
interbreeding with the San Diego population, due to the geographic distance separating
them.
It should be noted that my sample sizes of 10, 30, and 8 for the Hayward, San
Diego, and San Felipe populations, respectively, are relatively small. It is possible that I
did not detect the fast allele in the Hayward and San Felipe populations simply because of
chance. However, Dr. Jen-jen Lin has also analyzed both San Diego and San Felipe
Gillichthys for MDH. Of the fisty individuals from San Felipe she analyzed, all had the
same simple pattern.
The MDH results are somewhat unexpected if one were looking for a temperature-
related cline; the samples in the coldest and warmest environments show no variation from
each other while the samples from the intermediate environment show variation from both.
There are several possible theories that could explain the presence of the fast allele
in San Diego only. One possibility is that the slow allele is actually the ancestral form and
the mutation for the fast allele is one that has occurred in the San Diego area only.
Another possibility is that the frequency of the fast allele is actually increasing among
Gillichthys farther south down the Pacific coast, but that I cannot see this effect because I
did not have any samples from the Pacific side of Baja California. If this were case, the
absence of the fast allele in San Felipe can be explained either by founder effect or by
random genetic drist. Founder effect means that the few fish that journeyed (geographic
movement would most likely be from oceanic dispersal of larvae rather than the swimming
of adult fish) around the tip of Baja California into the Sea of Cortez to found the
population happened to all be homozygous for the slow allele. The individuals from San
Felipe would mostly be descended from the same founding fish and therefore are all
homozygous for the slow allele. For random genetic drift to cause the homozygosity of
the slow allele in San Felipe, the few fish that journeyed around the tip of Baja California
into the Sea of Cortez had both the slow and fast allele. Because the small population size,
these fish would not be in Hardy-Weinberg Equilibrium, and the genotype frequencies
would not necessarily remain stable over time. By chance, the frequency of the fast allele
could have been reduced to nothing or almost nothing. From that point on, all the fish in
the San Felipe area would be homozygous for the fast allele (Hartl, et al., 1989).
Although the analysis of the MDH results suggest that there is low gene flow
among the three populations, the study is far from complete. Larger sample sizes, and
samples from more locations, particularly the Pacific coast of Baja California would be
needed before a firmer conclusion about the pattern of MDH genotypes among Gillichthys
can be drawn. In addition, analysis of many more protein loci would also be needed to
draw definitive conclusions about the population structure and dynamics of Gillichthys.
AFLP
The data from the AFLP technique were not sufficient to permit a firm conclusion
to be drawn about population structure. As shown in Figure 10, however, there were
some differences between the Hayward and San Diego individuals. Although these results
are preliminary, they do show promise that AFLP can potentially find differences between
samples of Gillichthys.
Figure 9 also shows promise, not in terms of population differences but in terms of
the reliability of AFLP. Figure 9 shows several lanes run with the same individual, with
each lane reflecting slightly different reaction conditions. That the lanes show the same
banding patterns, with only differences in the intensities of the bands, suggests that AFLE
was working as planned; that the fragments being amplified are the ones we want to
amplify, and are not just random fragments. If random fragments were being amplified,
the fragments and therefore the bands would not be consistent over several trials.
It should be noted that although we did produce promising AFLP results,
numerous modifications and optimization efforts were required before consistent results
were obtained. The modifications we made are included in the materials and methods, but
several which we found to be particularly important involved: the amount of DNA used,
the temperature of the ligation reaction, the concentration of primer, and the preparation
of the selective amplification and labeling step PCR reaction on ice.
Although technically the amount of restriction enzyme should determine how many
fragments are cut, we found that adjusting the DNA concentrations affected the results.
500 uM gave the best results for Gillichthys, but for my colleague Maya Hayden, who
was using the AFLP technique on the squid <span style='background-color:#cfc' title='Loligo opalescens : Exact'>Loligo opalescens,</span> 300 uM DNA gave the
best results. It is important that the concentration of DNA not be too high, because an
excess of DNA can inhibit AFLP by remaining uncut and thereby interfering with the
ligation reaction.
Decreasing the ligation temperature from 37° C for three hours to room
temperature overnight may also have improved our results. This modification produced a
less harsh environment which may have increased the likelihood of the proper DNA
fragments, adapters, and enzymes coming into contact. It probably would also decrease
the likelihood of the adapters denaturing.
We added more Msel primer than EcoRl primer in the selective amplification and
labeling step for the same reasons that more Msel adapter is added than EcoRl adapter
Because Msel cuts much more frequently, there are many more Msel-Msel cut fragments.
Supplying more Msel primer ensured that there were adequate Msel primers for both the
many Msel-Msel cut fragments and the less abundant Msel-EcoRl cut fragments.
The PCR reaction for the selective amplification and labeling step was prepared on
ice. This was important to protect the Taq from denaturing and to prevent unplanned
reactions from occurring.
In summary, AFLP results may are consistent with a difference between the
Hayward and San Diego populations. AFLP was able to provide consistency within an
individual and to show variation between individuals, which suggests that it can be a
powerful technique. A downside of AFLP, however, is that it may require a lot of
modification before consistent and reliable results are obtained.
CONCLUSIONS
MDH results, which show the presence of a fast allele of the second cytosolic
isoform exclusively in San Diego individuals, provide some evidence that there is very
little or no gene flow between the Hayward, San Diego, and San Felipe populations of
Gillichthys mirabilis. The presence of the fast allele in San Diego may be explained by
several possible theories. Preliminary AFLP results, which suggest variation between the
Hayward and San Diego populations, do not disprove the no gene flow hypothesis.
However, more in-depth studies that include larger sample sizes, samples from more
locations, and analyses of more protein loci are required before definitive conclusions
about the population structure and dynamics of Gillichthys mirabilis can be drawn.
Finally, the ability of AFLP to provide consistency within individuals and show variation
between individuals suggests that this method has the potential to be a powerful DNA
fingerprinting technique. The original protocol, however, may require some modification
to make it suitable for the particular species being studied.
ACKNOWLEDGMENTS
I received help from numerous people throughout this project and would like to
thank some of them here. Thanks to my advisors Dennis Powers and George Somero for
initiating the project and for their support throughout the quarter. A huge thanks to my
mentor Gary Villa for his patience, help, and support. Thanks to Jen-jen Lin, who taught
me the isozyme electrophoresis techniques and to Tzung Yang, who took me Gillichthys-
trapping. Trish Schulte and the other members of the Powers lab provided a lot of input
and support; thank you all. Thanks also to Tony DeTomaso for his help with AFLP and
to Grant Pogson for his DNA Salt Extraction protocol. Finally, I would like to thank
Maya Hayden, who was my partner in developing the AFLP technique, for her company
and support.
LITERATURE CITED
Barlow, G.W. (1961). Intra- and interspecific differences in rate of oxygen consumption
in gobiid fishes of the genus Gillichthys. Biol. Bull. 121: 209-229.
Fields, P.A. (1995). Adaptation to environmental temperature in two genera of coastal
fishes, <span style='background-color:#cfc' title='Paralabrax : Exact'>Paralabrax</span> and Gillichthys. PhD thesis, University of California, San Diego.
Hartl, D.L., and Clark, A.G. (1989). Random Genetic Drist. In Principles of Population
Genetics, Second Edition. Sunderland, MA: Sinauer Associates.
Lin, J.-J. (1993). Thermal adaptation of cytoplasmic malate dehydrogenases of teleost
fishes. PhD thesis, University of California, San Diego.
Love, R.M. (1991). Longjaw Mudsucker (Gillichthys mirabilis). In Probably more than
you want to know about the fishes of the Pacific Coast, pp. 167-168. Santa Barbara,
CA: Really Big Press.
Murphy, R.W., Sites, Jr., J.W., Buth, D.G., and Haufler, C.H. (1996). Proteins: Isozyme
Electrophoresis. In Molecular Systematics, 2nd Edition (ed D.M. Hillis, C. Moritz,
and B.K. Mable), pp. 51-120. Sunderland, MA: Sinauer Associates.
Richardson, B.J., Baverstock, P.R., and Adams, M. (1986). Electrophoresis. In
Allozyme Electrophoresis, pp. 15-30. San Diego, CA: Academic Press.
Vos, P., Hogers, R., Bleeker, M., Reijans, M., van de Lee, T., Hornes, M., Frijters, A.,
Pot, J., Peleman, J., Kuiper, M., and Zabeau, M. (1995). AFLP: a new technique
for DNA fingerprinting. Nucleic Acids Research. 23, 4407-4414.
APPENDIX 1
The Hardy-Weinberg Equilibrium Distribution model is a method of interpreting
the population genetics of a sample. If a sample is in Hardy-Weinberg Equilibrium, it is
believed to meet the following assumptions:
1) There is random mating between phenotypes
2) The population is large
3) There is no selection between genotypes
4) There is no differential immigration or emigration of genotypes
5) There is no mutation.
In Hardy-Weinberg equilibrium, the genotypes will maintain a stable equilibrium with
genotype frequencies of AA = p’, Aa = 2pq, and aa = q', where A and a are alleles, p is
the gene frequency of A calculated from observed values, and q is the gene frequency of a
also calculated from observed values (Murphy, et al., 1996).
The test for Hardy-Weinberg equilibrium involves comparing the expected number
of genotypes, which are calculated from the values of p", 2pq, and qu', with the observed
numbers using the X2- test (Richardson, et al., 1986). The calculations for testing the San
Diego population for Hardy-Weinberg Equilibrium were as follows:
ÖBSERVED VALUES: 18 Homozygous Slow, 10 Heterozygous, 2 Homozygous Fast
p (slow) = (2 x18 + 10)/2n = 0.77
q (fast) = 1 - 0.77 = 0.23
p==0.593, 2pq =0.354, q’=0.053
EXPECTED VALUES = (expected genotype frequency) x (+ of individuals screened)
17.8 Homozygous Slow, 10.6 Heterozygous, 1.6 Homozygous Fast
X* Value = L (Observed-Expected)'/Expected = 0.1391
Critical value (o5, 1) = 3.841
0.1391 is less than 3.841, so the X* Value is not significant.
Therefore, the population is in Hardy-Weinberg Equilibrium
If the population was not in Hardy-Weinberg equilibrium, this would indicate that
the variation has a non-genetic basis or that one or more of the assumptions are not being
met
Table 1-PCR cycles for the selective amplification and labeling step
Duration
Temp
Duration
4of
Duration
Temp
Temp
cycles
1 min
45 sec
72°C
45 sec
650C
940 C
45 sec
72°C
1 min
64°C
940C
45 sec
45 sec
1 min
45 sec
630C
72°C
94° C
1 min
45 sec
72° C
45 sec
62°C
94°C
72°C
1 min
45 sec
45 sec
61°C
94° C
72°C
45 sec
1 min
45 sec
60°C
94° C
45 sec
720C
1 min
45 sec
590 C
94° C
45 sec
45 sec
72°C
1 min
580C
1
94°C
72°C
45 sec
1 min
45 sec
570C
94°C
1 min
45 sec
72°C
27
45 sec
560C
94° C
Table 2 - The number of individuals with each genotype.
Homozygote  Heterozygote
Slow
10
Hayward
10
18
San Diego
8
San Felipe
Homozygote
Fast
2
gaa
k

L




2
4






—

— —




—
M
n

Ldn
Figure 2
Hayward

L



San Diego
San Felipe
Figure 3
Extraction of Protein
Non-Denaturing Polyacrylamide Gel
Electrophoresis
Staining for Activity of the Protein
Figure 4
A
AjA
Monomer
A2 E
A
Dimer
A
AjA2
AA
A2 E
A2A2
Figure 5
Isolation of Genomic DNA
Double Restriction Digest
Ligation of Adapters
Preamplification
Selective Amplification
and Labeling
Polyacrylamide Gel
Electrophoresis
Autoradiography
2
2 1
15


i
5

1
+



1
2
0
L
1

—


E

1
Figure 9
.
SD8
Figure 10


















..



































.
















W
.


.



.



.
.

H2 H3 SD SD SD
14 17 19
Bands that are present in the
San Diego individuals but
not the Hayward individuals


Figure 13

FIGURE LEGENDS
Figure 1 - Distribution of Gillichthys mirabilis: the coast of California and
Baja California, and the Sea of Cortez.
Figure 2-Samples were obtained from Northern California, Southern California, and Sea
of Cortez locations.
Figure 3 - The major steps of isozyme electrophoresis.
Figure 4 -Possible gel patterns of monomeric and dimeric proteins.
Figure 5 - The major steps of AFLP.
Figure 6-After the double restriction digest, adapters are ligated to the ends of the
fragments.
Figure 7 -Preamplification involves the denaturing of double-stranded DNA, selective
annealing of primers, and amplification.
Figure 8 -Primers used in the selective amplification labeling step have three random
bases to increase selectivity. EcoRI primers are labeled with radioactivity.
Figure 9-The lanes indicated are the same sample under different reaction conditions.
The lanes show identical banding patterns, with differences in the intensity of bands.
Figure 10 - There are differences between the banding patterns of Hayward individuals
(H2 and H3) and San Diego individuals (SD 14, SD17, and SD19).
Figure 11 - LDH gel with the simple, two-band pattern. These individuals are from San
Felipe and San Diego; however, all the individuals sampled from the three location
exhibited this same pattern.
Figure 12 -MDH gel with Hayward samples. Each individual has the same, three-band
pattern.
Figure 13 - MDH gel with San Felipe samples. As with the Hayward samples, each
individual has the same, three-band pattern.
Figure 14 - MDH gel with San Felipe and San Diego individuals. The San Diego
individuals show variation at the second cytosolic isoform. SDI is homozygous for the
slow allele, SD2 is heterozygous, and SD3 is homozygous for the fast allele.
</pre></body></html>