<html><body><pre style='margin:2rem auto;width:fit-content'>Abstract
A highly conserved portion of the RyR channel region was amplified.
cloned and cycle sequenced from <span style='background-color:#cfc' title='Brachiostoma lanceolatum : Exact'>Brachiostoma lanceolatum</span> (amphioxus) and
<span style='background-color:#cfc' title='Mustelus californicus : Exact'>Mustelus californicus</span> (smoothhound dogfish). Two isoforms, RyRI and RyR3,
are expressed in most vertebrate skeletal muscle, while RyRI is the sole isoform
expressed in mammalian and some reptilian skeletal muscle. The purposes of
this study were 1) to investigate how primitive the two isoform condition is, and
2) to determine what clues the sequences of primitive vertebrate RyRs provide
about structure and/or function in the channel region. In comparisons with
published RyR sequences, amphioxus and shark displayed an overall high level
of sequence identity with all isoforms in this region. Molecular phylogenetic
analyses clustered the amphioxus and shark sequences with the published RyRl
sequences. One region of 50 contiguous amino acids located within a possible
transmembrane region did display high variability among species. However,
hydrophilicity profiles of amphioxus, shark, fish RyRI, and frog alpha sequences
in this region revealed similar patterns of hydrophobicity. This common
characteristic indicates that the amino acid differences most likely do not impact
on structure or function in this region.
Introduction
Intracellular calcium release in muscle cells is controlled by the ryanodine
receptor (RyR), a large protein located in the sarcoplasmic reticulum (SR). The
ryanodine receptor allows calcium, stored in the SR, to be released into the
cytosol during excitation-contraction (E-C) coupling. The ryanodine receptor is
localized at triad junctions, sites where the SR associates with specialized
inpocketings of the muscle cell membrane called transverse (T) tubules. At these
triad junctions, the cytosolic domain of the RyR forms "foot" structures which
span the gap between the SR and T tubules. The RyR is necessary for signal
transduction between these two membranes in E-C coupling (Sorrentino et al.,
1993).
A combination of four identical monomers of more than 5,000 amino acids
each forms the calcium release channel. Of these 5,000, only 1,000 residues
make up the pore, or channel, which is inserted into the SR. These pore amino
acids are found at the carboxy terminus of the RyR and are highly conserved
between isoforms and species. The bulk of the protein closest to the N-terminus
is cytosolic, and is thought to form the foot structure (Sorrentino et al., 1993;
Grunwald et al., 1995)
Two models for RyR control of calcium release have been proposed. In
the first model, direct mechanical coupling of an RyR to a dihydropyridine
receptor (DHPR) in the T tubule membrane triggers calcium release by the RyR
upon excitation (O'Brien et al., 1995) Calcium-induced calcium release, the
second method of modulation, requires extracellular calcium. In this proposed
pathway, a calcium current through the DHPR activates the RyR. (Herrmann-
Frank et al., 1991) When RyRs are located next to each other, a two component
mechanism has been proposed. Calcium release via direct mechanical coupling
may trigger calcium-induced calcium release in a neighboring ryanodine receptor
(O'Brien et al., 199
At least three different RyR isoforms have been found in muscle and other
cell types. RyRI was first isolated from skeletal muscle and is proposed to
function via direct mechanical coupling (Takeshima et al., 1989; Zorzato et al.,
1990). RyR2 and RyR3 were first found in cardiac and brain tissues respectively.
These two isoforms are thought to employ calcium-induced calcium release
(Hakamata et al., 1992)
A pattern of isoform expression in skeletal muscles has been described by
O'Brien et al (O'Brien et al., 1993). In most vertebrates that have been surveyed,
two RyR isoforms (alpha and beta) which correspond to RyR 1 and RyR2 are
expressed. However, only the RyRI isoform has been isolated from skeletal
muscles in mammals and some reptiles. Based on this pattern of expression it has
been suggested that the more primitive condition is the expression of two
isoforms in skeletal muscle.
Little is known about isoform expression prior to shark along the vertebrate
phylogenetic tree. This raises the question: how primitive is this two isoform
condition? Since the condition of having two isoforms appears to be more
primitive, then this coexpression is expected in species at the based of the
vertebrate tree. In an attempt to begin answering this question, one purpose of
this study was to extend observations of isoform expression further back in time
by obtaining sequence from the RyRs of primitive vertebrates such as
Brachiostoma lanceolatum (amphioxus), <span style='background-color:#cfc' title='Myxine glutinosa : Exact'>Myxine glutinosa</span> (hagfish), and
<span style='background-color:#cfc' title='Botryllus schlosseri : Exact'>Botryllus schlosseri</span> (tunicates). Sequencing of Mustelus californicus
(smoothhound dogfish) skeletal muscle RyR was also attempted to try to verify
the presence of two isoforms.
Prior sequencing of RyRs indicated that a high level of homology existed
among isoforms and species in portions of the channel region (Hakamata et al.,
1992). PCR primers specific to one of these regions in <span style='background-color:#cfc' title='Makaira nigricans : Exact'>Makaira nigricans</span> (blue
marlin) RyRI sequence were used to amplify sequence RyR in the species
mentioned above. Attempts at isolating other existing isoforms were done in this
conserved region because the primers were more likely to anneal to all of the
isoforms.
Two patterns of transmembrane sites have been proposed for the RyR
channel region: the first hypothesizes that four transmembrane locations exist
while the second proposes twelve transmembrane sites (Zorzato et al., 1990;
Takeshima et al., 1989) . Location of transmembrane sites has been predicted by
the pattern and hydrophobic character of amino acids in a portion of the
molecule. The region sequenced in these experiments has been suggested to
contain three of the twelve predicted transmembrane sites and one of the four
hypothesized locations. Recent experiments done by Grunwald, et al. using site
directed antibodies, provide support for the four site model (Grunwald et al.,
1995) Comparisons of RyRs from higher vertebrates with RyRs of more
primitive vertebrates may provide another approach to examining this structural
question. This leads to a second question for investigation: what clues do the
sequences obtained in this study provide about structure and/or function in this
region of the RyR?
Expression of RyRI in amphioxus and shark was determined in these
experiments. Two amphioxus clones were sequenced and found to have different
amino acid translations. Alignment of amphioxus and shark sequences with RyR
from other species showed a high level of homology with the exception of a
variable region that was approximately 50 amino acids. This variable region
contained a hypothesized transmembrane region. Despite the residue changes.
the overall hydrophobicity of this region prevents rejection of this hypothesis.
Parsimony and distance trees indicated that amphioxus and shark sequences wer
most similar to each other as well as to RyRl sequence from fish (blue marlin).
Materials and Methods
Tissue Isolation
Amphioxus and hagfish tissue samples were obtained from Pacific
Biomarine (Venice, CA). Muscle tissues from smoothhound shark and blue
marlin were taken from specimens collected by the Block lab <span style='background-color:#cfc' title='Kona : Exact'>(Kona,</span> HA, 1993).
Tunicate specimens were a kind gift of Dr. Doug Stoner, Hopkins Marine Station,
Pacific Grove, CA.
Blue marlin, shark, hagfish and amphioxus tissues were freeze clamped
and stored at -80’C. Tunicate specimens were used immediately upon acquisition
and did not need to be freeze clamped for storage purposes.
RNA Isolation
Tissue samples of 200 mg to 500 mg were placed in Tri Reagent
(Molecular Research Center, Inc, Cincinnati, ÖH). Samples were minced
thoroughly with a Tekmar homogenizer and RNA was isolated according to the
manufacturers specifications. RNA pellets were washed with 70% EtÖH (1 ml
per sample) and centrifuged at room temperature for 5 minutes at 14,000 rpm.
The pellets were dried, resuspended in Diethylpyrocarbonate (DEPC) -treated
H9O (Sigma, St Louis, MO), and combined to form one sample of 100 ul total
volume.
CDNA
Total RNA (20-40 vg) in 18 ul of DEPC-treated H2O was used to make
CDNA. To this RNA, 0.05 vg/vL Oligo dT’s (2 vl) were added and the resulting
solution was heated to 95°C for five minutes and then quick chilled on ice. 5 x
Buffer (12 vl), 10 mM dNTPs (2 vl) (Promega, Madison, WI), and DEPC-treated
L.0 (4 ul) were added. After mising RNAsin (1 uh and MMLV Reverse
Transcriptase ( o) Cromega.Madison, WI) the reaction vas ineubated at tom
tempenatur for S minuesand hen a 97C for 45 10 50 mintes. 40 lof 103
Tris Eihylenedamine tetraceic acid EDTA)/ Sigma, St Louis, Mo wvere used
to stop he reaction and the cDNa was hen precipialed wih 10 olof 3 Mi
MaOAc and 30 01 1006 O for at least 2 hous at 20 C. Thepelet was
washed with 106, EOH as described under RNA isolation procedures.
PCR
Blue marin specifi primers, RyR Bam (S CTOCATCCTCATCTCAICS
GEVERSt) and BR EO S COACATTCTOACTCTIOC CORVARD), vere
used at l mnd concentaions each. 2 5 mM MgCh, 20 500ng of DNA
template and 125 M dNTPS wer ed Abot sant procedure wasdone fortne
reactions in which Su (units) Tag DNA polymense vas aded once the
reactions had reached 80 C ina Pefkin Elner a80 thermaleyder. Pok
condions were 3S opde a1 95 C for Imin 20sec. 30 C for lmin. 0 ec. 120.
for 1 min, 72-C for 1 min, 30 sec. 72-C for 7 min delay¬
POR products were precipitated as descrbed forcDNA and were run on a
16 Tak lov meling agarose gel Productsat he prediced sie daprok 430 bp/
were cut from the gel and sored in labeled microcennifuge ubes at 20C.
Cloning and Plating
POR products were cloned using the pOEM-T Vectorsystem (Promega.
Madson W) acoding o the mantacmer s pecicaions vilh some indicded
changes. 20 Vlofligheffcieney KIl. Blue <span style='background-color:grey' title='Escherichia coli : Fuzzy'>Echerichia coli</span> competentceis
vere addedto 10 U ofthe lgaed PCR produGC NOEM T Vecorand ineubated
on ie for l hour. The cels werethen plated on B wih amnpeilin 0 ogimn
and incubated overnight at 37°C. Color selection was used to determine which
colonies had successfully taken up the vector
PCR Detection and Plasmid Purification
PCR detection of recombinants was done using 1 mM M13-20 (forward)
primer, 1 mM M13-Reverse primer, and 3 mM MgCl. PCR cycle conditions
were the same as described above. 2 vl of LB with 50 ug/ml ampicilin were used
to grow up the transformed cells. The purifications of the plasmid DNA were
done with the Wizard Miniprep DNA Purification Systems (Promega, Madison,
WI following manufacturer's procedures.
Cycle Sequencing
1 ug of each clone, PRISM Ready Reaction Terminator mix (Applied
Biosystems), and either 3.2 pM M13-20 and M13-R were used for the sequencing
reactions. For PCR of the sequencing reactions, 25 cycles of the following
conditions were used; 95°C for 5 min; 95°C for 1 min; 55°C for 1 min, 30 sec;
70
12°C for 2 min; 72°C for 7 min. Following PCR the unincorporated nucleotides
were removed using G50 Sephadex spin columns (Promega, Madison, WI).
Polyacrylamide denaturing gels (6%) were used for sequencing the clones
on an Applied Biosystems 373A DNA sequencer. Sequences were edited with
SeqEdl.03 from Applied Biosystems.
Analysis of Sequences
Comparisons of the experimental sequences with those listed in
GENBANK were done via the NCBI BLAST E-mail server (Altschul et al.,
1990). Consensus sequences were made in AssemblyLign (International
Biotechnologies, Inc.) The program Mac Vector version 4.0 (International
Biorechnologies, lnc) was used to tanskte and align sequenes. Hygdrophnlichy
pofles vere also geaed in Mae Vector using the kyte Doolule scake ot
hydrophilicity.
Sequences were aligned in CLUSTALVin an apropriae fomnatforthe
Phlogeny lnferene Package PHVLIP 35) Cniversty of Vaskingon Wa). A
parsimony re vilh 30 boistrap resamplings was derived using he Seqo.
Propns, and Consense progansin PINVLP. The PINVLIP Protdistand
Deichbor prograns were used togenerate adistance urce. Thealgorihin vsedio
derermine branch lengihs wes povded by he Dayhof PaM manik hich makes
atransition pobabilty mati based on the sequenes. This matrik wveignis
amino acid swviches by predieing he pobablity of changing from one amno
acidio anoher. The pobabilies aeued odemine banch discances scaled
in unts of erpected fraction of amino acds changed. The PfINLIP pograms,
Davgram and Dravue were usedto prodice picures of ach tye of te.
Results
cDNA concentrations as determined by UV spectrophotometry, ranged
from 180 ng/ol to 600 ng/bl. PCR products of the expected 450 bp size were
obtained for blue marlin, hagfish, shark, amphioxus. The photograph of the
products from PCR detection of recombinants shown in Figure l indicates that
cloning was successful with blue marlin, shark, and amphioxus. A faint band at
approximately 670 bp (450 bp of clone plus 220 bp of poly linker) from the
hagfish products can be seen in lane 12, however, sequencing of this clone
determined that there was no insert. RyRl sequence was amplified from the
three other species. Since the Block lab had already sequenced blue marlin in
this region, the sequence obtained for blue marlin in this study was not used in
comparisons.
The nucleotide sequences of shark and amphioxus are given in Figure 2.
These are consensus sequences derived from the overlap of the forward and
reverse primed sequences of a clone. Clean sequence for two amphioxus clones,
Amphe and Amphl0, and three shark clones, Shark17, Shark 20 and Shark21
were obtained. However, only the sequences from Shark17, Amph6 and Amphlo
are shown in Figure 2 because consensus sequences could only be determined for
these clones. Amph6 was sequenced twice and the two consensus sequences
showed 98.1% identity. Comparisons with GENBANK showed that the
sequences with the exception of Shark2l were most likely RyRI <span style='background-color:#cfc' title='Mus musculus : Exact'>(Mus
musculus).</span> BLASTX results for Shark21 indicated that this clone was probably
Ryk3 (Homo sapien). However, since a consensus sequence could not be
completed for this clone, its sequence was not used in analysis.
An alignment of the translated sequences is given in Figure 3. Sequence
identity is greatest among blue marlin, amphioxus and fish RyRI sequences
10
(Table 1). As the alignment indicates, the most variation among all sequences
occurs between aa 10 and aa 30 and between aa 40 and aa 90, while the rest of
the sequence region has a high level of homology. Hydrophilicity profiles reflect
this overall high level of homology among amphioxus, shark, blue marlin
(fishRyR1), and frog alpha (Figure 4).
Figure 5 is a parsimony tree including the same species shown in the
alignment (Figure 3). The tree was resampled by bootstrapping 500 times and
the numbers at each node represent the percentage of trees in which this node
occurred. In 45.2% of the trees, rabbit cardiac, rabbit brain, and frog beta
sequences were further separated into their own clade. Since this bootstrap value
was less that 50%, this node was collapsed back. A distance tree based on the
number and types of amino acid changes among sequences was derived for these
species as well (Figure 6). The lengths of the branches correlate to distance in an
evolutionary time scale.
Discussion
Sequences obtained in these experiments are just a beginning step in
explaining isoform expression in early and pre-vertebrates. While the data do not
indicate clearly whether amphioxus and shark specifically have one or two
isoforms, they do strongly suggest that amphioxus and shark express RyRl. This
result is not surprising since the expression of RyRI seems to be common to all
vertebrate skeletal muscles that have been characterized. Further support for the
presence of RyRI in amphioxus is the similarity in E-C coupling mechanisms in
amphioxus twitch muscle and vertebrate skeletal muscle suggesting that they
have similar ryanodine receptors (Benterbusch et al., 1992). Expression of
RyRl in shark is expected based on immunoblotting experiments done by
O'Brien et al. which indicated the presence of both RyRI and RyR3 in shark
skeletal muscle (O'Brien et al., 1993).
Comparisons of the hydrophilicity profiles for amphioxus and shark with
those of fish alpha and frog alpha reflect the sequence variability between aa 40
and aa 70 of the sequences shown in Figure 4. The presence of hydrophilic
amino acids located between aa 60 and aa 65 could suggest that this portion of
the protein is not in the membrane of the SR (Figure 6). The hydrophilic
character of this region may also indicate that this small region is more likely
found in the cytosol, perhaps on the surface of the folded protein. However, the
overall hydrophobicity seems to be maintained in these sequences from
amphioxus and shark. Therefore, the numerous changes in amino acids in this
region are unlikely to impact structural or functional interpretation of this region
to a high degree.
12
Previous alignments of the entire RyR sequences were used by the Block
lab to construct a parsimony tree. In this tree, fish alpha and frog alpha were
grouped in the same clade. The parsimony tree completed for the regions
sequenced in this experiment places fish alpha, amphioxus and shark into a
separate clade. This raises the questions, 1) would the same division occur ir
more of the sequences of amphioxus and shark were compared and if this is the
case. 2) where in the phylogenetic tree does this split between frog and these
other species occur? Isolation of RyR from species such as lungfish, bowtin and
birchirs which branch off of the vertebrate phylogenetic tre between bony fisn
and amphibians might help answer this second question. The distance tree also
groups amphioxus, shark and fish alphas apart from frog alpha, posing similar
questions.
In the distance tree, an amino acid change is weighted based on the
frequency with which that amino acid occurs in the entire sequence. Branch
lengths are meant to reffect distance on an evolutionary time scale. The large
branch lengths separating amphioxus and shark from fish suggest that these two
groups are more evolutionarily distant than the parsimony tre might indicate.
However, a distance tree which weights amino acid changes based on
composition, polarity, and molecular volume may provide a better representation
of relative branch distances.
There are several possible explanations for the lack of a POR product of
predicted size with tunicates. The most obvious problem may have been with the
quality of tunicate RNA Or CDNA. However, UV spectrophotometty of each orf
these indicated that the both the RNA and CDNA were of high quality. Four
attempts at priming with RyR Bam and RyR Eco were made, but perhaps more
PCR trials should be conducted trying different PCR conditions. Another
13
possibility is that the RyR of tunicates may differ enough from blue marlin RyRI
that the primers will not anneal.
The Amphó and Amph10 clones are most homologous to each other and
differ from the other RyR sequences to a similar extent. The differences between
the Amph6 and Amph10 clones in this sequenced region may be due to
sequencing problems. However, the possibility of these clones coming from
different types of RyR cannot be ruled out at this point. Resequencing of these
clones would be necessary for demonstrating the veracity of these sequences.
Since these sequences are from CDNA made from RNA, inconsistencies in how
introns are spliced from this region during transcription might also explain the
sequence differences. However, when this portion of the sequence is aligned
with <span style='background-color:#cfc' title='Drosophila : Exact'>Drosophila</span> RyR, the entire region falls within an exon on the Drosophila
sequence (Takeshima et al., 1994).
The next step in determining the pattern of isoform expression in
amphioxus might be to sequence further along the RyR. If the two amphioxus
clones are from different RyR types, comparisons at some of the more diverse
regions may show a step along the way in the branching off of the different
isoforms. Isolation of RyR from tunicates would be very useful in drawing a
more complete picture because it is a relatively close ancestor to amphioxus.
Obtaining RyR sequence from larval tunicates might be more feasible since at
this stage in life, the tunicates have tails designed for locomotion. With
adulthood the tunicates become benthic and resorb their tails, losing the muscle
present in this appendage. Although adult tunicates do require some sort of
musculature to pump water for feeding, it is presumably less than that of the
larval tail (Minkoff, 1983).
RyR3-specific primers would be a useful tool in determining the presence
of this isoform in tissues where it is coexpressed with RyRI and RyR2.
﻿14
However, it is this coexpression which makes development of a probe very
difficult. RyR3 seems to be expressed at much lower levels than RyRI and RyR2
in all tissues (Sorrentino et al., 1993). Therefore, the chances of isolating RyR3
from any tissue using non-RyR3 specific probes are not high. These factors, in
addition to the difficulties in finding a distinct yet highly conserved region of the
RyR3, make this project a difficult undertaking.
Results from this study have shown that amphioxus and shark do express
RyRI. In alignments with other RyRl sequences, a small region of high
variability was present in amphioxus and shark. However, as the hydrophilicity
profiles indicated, these amino acid differences have not appreciably changed the
pattern of hydrophobicity in this region. Overall, sequences obtain for these
species demonstrated high level of homology with RyRI sequences from other
vertebrate species.
15
Literature Cited
Altschul, S.F., W. Gish, W. Miller, E.W. Myers, D.J. Lipman. 1990. Basic local
alignment search tool. J. Mol. Biol. 215: 403-410.
Benterbusch, R., F.W. Herberg, W. Melzer, R. Thieleczek. 1992. Excitation¬
contraction coupling in a pre-vertebrate twitch muscle: the myotomes of
the <span style='background-color:#cfc' title='Branchiostoma lanceolatum : Exact'>Branchiostoma lanceolatum.</span> J. Membrane Biol. 129: 237-252.
Grunwald, R., G. Meissner. 1995. Lumenal sites and C terminus accessibility of
the skeletal muscle calcium release channel (ryanodine receptor). J. Biol.
Chem. 270: 11338-11347.
Hakamata, Y., J. Nakai, H. Takeshima, K. Imoto. 1992. Primary structure and
distribution of a novel ryanodine receptor/calcium release channel from
rabbit brain. FEBS. 312: 229-235.
Herrmann-Frank, A., E. Darling, G. Meissner. 1991. Functional characterization
of the Ca2+gated Ca2+ release channel of vascular smooth muscle
sarcoplasmic reticulum. European Journal of Physiology. 418: 353-359.
Minkoff, Eli C. 1983. Evolutionary Biology. Reading, Massachusetts:
Addison-Wesley Publishing Company. 485-491.
O’Brien, J., G. Meissner, B.A. Block. 1993. The fastest contracting muscles of
nonmammalian vertebrates express only one isoform of the ryanodine
receptor. Biophysical Journal. 65: 2418-2427.
16
O'Brien, J., H. Valdivia, B.A. Block. 1995. Physiological differences between
the alpha and beta ryanodine receptors of fish skeletal muscle. Biophysical
Journal. 68: 471-482.
Sorrentino, V., P. Volpe. 1993. Ryanodine receptors: how many, where and
why? TiPS. 14: 98-103.
Takeshima, H., S. Nishimura, T. Matsumoto, H. Ishida, K. Kangawa, N.
Minamino, H. Matsuo, M. Ueda, M. Hanaoka, T. Hirose, and S. Numa.
1989. Primary structure and expression from complementary DNA of
skeletal muscle ryanodine receptor. Nature (Lond.). 339: 429-445.
Takeshima, H., M. Nishi, N. Iwabe, T. Miyata, T. Hosoya, I. Masai, Y. Hotta.
1994. Isolation and characterization of a gene for a ryanodine
receptor/calcium release channel in <span style='background-color:#cfc' title='Drosophila melanogaster : Exact'>Drosophila melanogaster.</span> FEBS
Letters. 337: 81-87.
Zorzato, F., J. Fujii, K. Otsu, M. Phillips, N.M. Green, F.A. Lai, G. Meissner,
D.H. Mac Lennan. 1990. Molecular cloning of cDNA encoding human
and rabbit forms of the Ca2+ release channel (ryanodine receptor) of
skeletal muscle sarcoplasmic reticulum. J. Biol. Chem. 265: 2244-2256.
17
Figure Legends
Figure 1. PCR detection of recombinants. PCR products were fractionated on
1% agarose gel, stained with ethidium bromide and visualized with UV light.
Bands at approximately 670 bp represent vectors containing clones. Molecular
weight markers are given to the left of the figure.
Figure 2. Nucleotide and derived amino acid sequences of shark and amphioxus.
Figure 3. MacVector 4.0 multiple sequence alignment of derived amino
sequences. Dots indicate sequence identity to the top sequence (Shark17). Gaps
in sequences are represented as dashes. The region of high variability referred to
in the text is located between aa 40 and aa 90. Lines over portions of the Sharkl /
sequence indicate locations of hypothesized transmembrane sites (Zorzato et al.,
1990; Takeshima et al., 1989).
Figure 4. Hydrophilicity profiles generated by MacVector 4.0. Values above the
zero line indicate regions of hydrophilic residues. Program utilizes à kyte¬
Doolittle algorithm.
Figure 5. Parsimony tree generated from a CLUSTALV multiple alignment of
derived amino acid sequences by PHYLIP Protpars. Values given at nodes of
tree are percentages of 500 bootstrap replicates in which the node occurred.
Drosophila is the designated outgroup.
18
Figure 6. Distance tree generated by PHYLIP Protdist and Neighbor programs
using a CLU
ISTALV multiple alignment of derived sequences. The algorithm to
determine branch lengths was provided by the Dayhoff PAM matrix which
weights amino acid changes by predicting the probability of a specific amino acid
switch. Drosophila is the designated outgroup.
19
8
Figure 1.
53b
078 bp
872 bp
603 bp
310 bp
271 bp



enen eien




2.
aaaaaata-
ann e anan
Figure 2.
Shark17

. . . . . . . .
a TRANSLATION OF SENRKIT AJ
taaaa
aaaaaaaaaa-
..0.1 "
.. . . . . a TRANSLATION OF SHARKI7 (A a a a a a à à à à a a

. . . . . . . . . . a TRANSLATION OF SRARKI7 (A) a . . . . . . . . . . 3
aaaa-
m e   ur
s 5
. . . . . . . . . . a TRANSLATION OF SHARK7 (Al a a a a a a 4 a a à à
ETT
M TETETT
. . . . . . . . a TRANSLATION OF SHARKI7 (A a a a a a a a aa a
AI TAC CI COT CI CET CTATT
a TRANSLATION OF SMARKI7 (AI  . .
Amph6
a-
CCATTTA
0A  TRANSLATTON OE ANRG taj aaaaaia
1.*. . . .
Saa
E AS TE C AT TATECECT ACT CET CEAC TTTTTA
6
.. . .  Rstrro or Mn (a  . . . . . . . . . ..
ataa
ATETTATTAT

. . . . . . . . TRANSLATION OE AHE (A . . . . . . . . . .

aaaaaa-
aaa-
. . . . a a = à à
. . . . . . . . . . . TRANSLATION OE AH6 1A
s   d d   e a ne qoe oge a eug er g  Te T

. . . . . . . . . . * 3
. . . . . . . . . . RNSLATION OE MPHE (A)
r
2 a TRANSLATION OF MRS IAI a.
Amph10
AT
C G Tr   e
o a 4 . . . . . .
TCAT CATC
11
CATA TCAATTNTETETCRACOTCTTTTTA

...... or o ti . . . . . . . . . .
aataa-
SCTTETTCTRCETTETATE
... . . . . TRNSIATO OE NMO (à . . . . . . . . . . .
aaaa-
A
.. . . . . . . TRMSLATION OF MRIO (4 . . . . . . . . . . .
taaa-
TTMTcardcce n  rr
o 0
" a TRANSLATION OF ANPHIO (AJ 4 . . . . . . . . . . .
. . . . . . . .
r  r  er r
N OF APHIO (A)
Figure 3.
SHaEk17 (52 VOLASLOVSOOHEKKOEEPDNSVFAWVTSIDIKYOIWKFGWVFTDCYFPFISCCYLNAVF
. F . . * * * * * * * * * *
LCVRRVLSV.
2Ls
. .* * * * * * * * * * * * * * *
VCARRRVLSV0
. * . . . . . . . . .
2
R11
VNNS. LYLV-WVASI
M.EITSNG..P-
MEITARN. R.P-O PPGLLT.LN..V..
VINS. LYLG-W.NVNSL
71NS LYLC-WNVMSL
MT EITARN. R.P-N.PPGLLT.LN.V
LVNS. LYLA-WTTNSI
L.KNA..F.-pv..TKI.-EC.LVS. LS..M.
RaDBE
1 19
LKNA.F.PVE TT-AA-ALVS. LS.M.HLV NS LYLA-W.-TTMS»
95 1
M.KAF.OAR. KPKK.S LS. VLNVM.LVNS. LYLA-W.ATMSI
OrosophA
SEETGI KYINN. R.V AVT. NA. LY- LW.FSES-»
( 394
—
Sharkl7 (52 GRDNYNNFFLPAWSSCWIFAWVVODSCAPILSSVTANAKOLMMTVGLLAVVVYLYTVVAF
ALRLP1LRr
LGHCHLLOI. NC. Krt-RT6
100
Anphé
82
FR-LVT LI ReCERLLRT
( 628
..****.*..  2
-17FA-RL-LOI. NG. KTL-RT.
d
L3nFA.-AL-LOI.NG. KTL-RTv
-13FA-AL-LOI.MG KTL-RTv.
100
60
500
-LCRFA.-AL-LOI.ACFKTL-RTVL.
100
tabBrainA
FA-AL-LOI NGFKTL-RTVL

496 VLG
5FAAL-LOI NCFRTL-ATvt.

VncFFA-AL Lov CrkrLarvLr1..
Shark17(2 NFFRKFYNKSEDEDEPONKCOONMTCYLFRMYVGVRAGGCIC
120

* **** *** ****. L.
12
120
h6
626
2

10 120
* * * * * * * * * * * * * * * * * * * * * **
* * * . . . .
140

1
. . .
0...
6

1

10 EOEvKKLv1
Figure 4.
Shark17
5.00
4.0

3.00
2.00
1.00
0.00
-1.00
-2.00
-3.00
-4.00 -
-5.00
Amph6
5.00
4.00
3.00-
2.00-
1.00-
0.00
-1.00 -
-2.00
-3.00
-4.00
-5.00
Amph10
5.00
4.00
3.00
2.00
1.00-
0.00
-1.00 -
-2.00 -
-3.00 -
-4.00
-5.00 -
Fish RyRI
5.0
4.00
3.00
2.00
1.00-
0.00-
-1.00
-2.00-
-3.00
4.00
-5.00 -
Frog alpha
5.00
4.00-
3.00
2.00 -
1.00-
0.00
-1.00
-2.00 -
-3.00-
-4.00 -
-5.0
Hydrophilicity Window Size - 7

Hydrophilicity Window Si:

4
Hydrophilicity Window Size

Hydrophilicity Window Size


Hydrophilicity Window Size = 7


Scale = Kyte-Doolittle

V
120 140
160
100
Scale = Kyte-Doolittle
W

140 160
120
(yte-Doolittle



140
﻿160
120
Scale = Kyte-Doolittle

V
L1—
120
100
Scale = Kyte-Doolittle


140
Figure 5.
100.0
— Drosophila
90.5
76.4
64.1
94.4
—
—Amphioxus6 alpha
55.1
100.0
— Amphioxus10 alpha
— Shark alpha
— Fish alpha
— Human skeletal
98.4
— Rabbit skeletal
— Frog alpha
— Frog beta
— Rabbit brain
— Rabbit cardiac
Figure 6.
Frog alpha
Fish alpha
Amphioxus 10 alpha
anpehinsaha
— Shark alpha
Rabbit skeletal
Humanskeletal
— Rabbit brain
— Frog beta
— Rabbit cardiac

— Drosophila
==.089% expected change b/w two aa sequences
</pre></body></html>