<html><body><pre style='margin:2rem auto;width:fit-content'>Blazyk1
ABSTRACT
Potential for gene flow in marine environments is often much higher than what is actually
observed, due to various barriers to dispersal. Understanding the divisions between populations
of organisms is important to fishery regulations and designing effective marine reserves. <span style='background-color:#cfc' title='Tegula funebralis : Exact'>Tegula
funebralis</span> have a pelagic larval phase, lasting 5-14 days, giving them a moderate potential for
dispersal. Samples ranging from Oregon to Santa Barbara were collected from seven geographic
locations. A 700 base pair region of the mitochondrial cytochrome oxidase I gene (COI) was
sequenced for an average of 9 individuals from each location. Comparing these sequences, there
is no clear overall population structure, although there seem to be some slight patterns
corresponding to geographic location. In particular, snails from Oregon are more similar to each
other and are more different from snails from other populations, suggesting a lower level of
dispersal between <span style='background-color:#fcc' title=' : NoMatch'>T. funebralis</span> from Oregon and the more southern locations. These smaller
signals could not be shown to be significant with the current sample sizes, using Fsr values or
chi-squared tests, and had p values of between 0.07 and 0.10. Overall, there appears to be a
fairly high level of gene flow across the range studied, which makes it unlikely that there are
major barriers to dispersal in this range. However, there may be subtle population structure in T.
funebralis along the west coast of North America that will emerge when larger sample sizes from
each location are used, a larger portion of the gene is sequenced, and populations further to the
north and south are sampled.
Blazyk 2
INTRODUCTION
Many marine invertebrates are sessile and may live their entire adult lives within a few
square meters of space. Therefore, the genetic makeup of populations is largely due the nature of
larval dispersal and development (Hoskin 1996). The pelagic larval phase of some invertebrates
is the one chance that animal has to spread its genes to another (possibly quite distant) location.
It is then to be expected that invertebrates that lay egg cases would have genetically distinct
populations on a much smaller scale than species that have pelagic larvae.
Pelagic larvae of invertebrates can drift in the oceans for anywhere between a few days to
several months. Off the west coast of North America, where the California Current moves at an
average rate of 10 cm/s (Debenham et al. 2000), larvae have the potential to move from tens to
thousands of kilometers. However, in many cases, species exhibit a more limited dispersal than
is predicted by the duration of their pelagic larval phase. This can be explained by natural
selection, larval behavior, including vertical migration and swimming, eddies and other
counteracting currents, and by possible physical boundaries to dispersal.
Because it is very difficult to directly measure larval dispersal distances, other, indirect
measures are used. One such tool is analyzing genetic differences between populations. This
evaluation, when compared to genetic variation within populations, gives a good idea of the
amount of larval exchange between populations. One migrant per generation prevents the
accretion of large genetic differences, and ten migrants per generation stops all but very minor
differences from appearing.
Possible barriers to dispersal include Point Conception, where the California coast turns
sharply eastward, moving away from the stream of the California Current, Monterey Bay, and
San Francisco Bay. However, these features seem to not always impede dispersal.
Blazyk 3
Miner (2002) found that although there appear to be two distinct physiological races of
<span style='background-color:#cfc' title='Pollicipes polymerus : Exact'>Pollicipes polymerus</span> (gooseneck barnacle, with a pelagic larval phase of up to 40 days), which
divide at Point Conception, there were no differences in the genetic makeup of individuals
spanning this range. <span style='background-color:#cfc' title='Tetraclita squamosa rubescens : Exact'>Tetraclita squamosa rubescens</span> (pink barnacle) appear to have high gene
flow along a range from San Francisco to south of Point Conception. However, the population at
Moss Landing, at the center of Monterey Bay, does seem to have a higher level of isolation than
the rest, which could indicate more larval retention in some populations than others (Ford and
Mitton 1993).
Debenham et al. (2000) shows that in <span style='background-color:#cfc' title='Strongylocentrotus franciscanus : Exact'>Strongylocentrotus franciscanus</span> (red sea urchin,
with a pelagic larval phase of 61 to 131 days), there is high gene flow throughout the species
range from Alaska to Baja California. Samples from Alaska did seem to deviate slightly from
the rest, possibly indicating a more limited exchange of larvae, selection, or evidence of selective
mating.
In a study of <span style='background-color:#cfc' title='Haliotis cracherodii : Exact'>Haliotis cracherodii</span> (black abalone, with a pelagic larval phase of 5 to 15
days), Hamm and Burton (2000) conclude that there are significant differences between
populations of abalone between Santa Cruz and Santa Barbara, although genetic difference does
not correlate with geographic distance. This indicates that black abalone might be fairly poor in
dispersing larvae even moderate distances, which is consistent with observations of pink and
green abalone dispersal.
Buonaccorsi et al. (2002) studied <span style='background-color:#cfc' title='Sebastes caurinus : Exact'>Sebastes caurinus</span> (copper rockfish, with a pelagic
phase of several months) from Canada to south of Point Conception. Among these samples,
there was evidence of significant divisions between populations, as well as a positive correlation
between geographic and genetic distance.
Blazyk 4
Discovering the population structure of various species may be important for its own
sake, but it has implications for fisheries policy and designing marine reserves as well. In many
areas, for example, black abalone populations are severely depleted and are being protected from
human harvesting. However, because they have very limited dispersal (and hence have few
incoming larvae from other, less depleted areas), these populations may not be able to recover on
their own (Hamm and Burton 2000)
Marine reserves are designed to have positive effects beyond their boundaries, especially
when meant to restore commercial species, where the economic value of the reserve depends on
export from nearby areas where fishing is allowed. If there is little dispersal between a reserve
and the area outside of it, this will not improve the overall fishery. However, if a high
percentage of larvae or adults migrate out of the reserve, the reserve loses its effectiveness. A
successful self-seeding reserve needs to be as big as the mean larval dispersal of the targeted
species. This is not always possible (especially since some species have potential dispersal of
hundreds of kilometers), so in some cases, a better alternative is a series of stepping stone
reserves (Palumbi 2003).
Tegula funebralis has a pelagic larval period of about 5 to 14 days (Moran 1997), which
gives it a moderate potential for dispersal (about 40 to 120 kilometers). It ranges from
Vancouver Island south along the west coast to central Baja California. Since it spans several
geographic features proposed to be barriers to dispersal and has a relatively short pelagic period,
it seemed that Tegula funebralis had the potential for interesting population structure. Here, I
compared a region of the mitochondrial cytochrome oxidase 1 (COl) gene of seven populations
of Tegula funebralis between Oregon and southern California. I found no significant differences
between populations, although there was a significant positive correlation between genetic and
Blazyk 5
geographic difference, which indicates that more structure may resolve with larger sample sizes
and additional populations further to the north and south.
MATERIALS AND METHODS
Populations sampled
T. funebralis were collected from seven sites along the west coast of the United States
(Table 1; Fig. 1). These areas include Strawberry Hill, Oregon (44°15’42'’N, 124°07’36°W),
Bodega Bay, California (38°19’N, 123°04’W), just outside the breakwater at Pillar Point, near
Half Moon Bay, California (37°29’42"’N, 122°29’42"’W), exposed and protected regions of the
rocky intertidal at Hopkins Marine Station, Pacific Grove, California (36°37’N, 121°54’W), the
rocky intertidal at Soberanes Point, California (36°27’092’N, 121°55’45’’W), San Luis Harbor,
San Luis Obispo, California (35°10’36'’N 120°45’36’’W), and Lompoc, California (34°43’N,
120°34’)
DNA isolation
To isolate DNA, all snails were relaxed in a 1:3 mixture of isotonic MgClz and seawater.
Once relaxed, small bits of tissue were snipped from the edge of the snails' feet. This tissue was
then processed with the Nucleospin kit, extracting relatively clean DNA. The products were run
on a 2% agarose gel, showing bright bands corresponding to DNA. The extracted DNA was
diluted with water to a 1:10 solution, which was used in PCR.
mtDNA amplification
1 ul of the 1:10 DNA was used as a template in a 25 ul polymerase chain reaction
comprising 2.5 ul 1Ox AmpliTaq buffer, 2.5 ul 1Ox dNTPs, 0.30 ul AmpliTaq polymerase, and
1.25 ul each of the forward and reverse primers. Amplification conditions were as in Hellberg
Blazyk 6
(1998)—50 sec at 94°C, 90 sec at 50’C, and 90 sec at 72°C, but for 30 cycles instead of 25 and
700
without the 8 min extension time at /z’C. The Folmer et al. (1994) primers: HCO2198 (5’-
TAACTTCAGGGTGACCAAAAAATCA-3’) and LCO1490 (S’GGTCAACAAATCATA
AGATATTGG-3’) were used to amplify a section of cytochrome c oxidase I (COI). Some other
primers were tried, without success, including LCO/HCO (which are the same as 1490/2198 but
are not degenerate), COla/COlf, and 2198/conch2f. PCR products were run in 2% agarose gels
to identify the samples that successfully amplified, producing a distinct band, about 700bp in
length.
DNA sequencing
To remove excess dNTPs and primers, 5 ul of PCR products of successfully amplified
samples were then added to 2 ul shrimp alkaline phosphotase (SAP), 1 ul exonuclease 1 (EXÖ),
and 0.5 ul SAP buffer. The samples are then held at 37°C for 30 minutes and 80°C for 15
minutes. To add fluorescent markers for sequencing, 2 ul of the SAP/EXO products were added
to 6 ul dd H20, 1 ul big dye, 1.5 ul 5X buffer, and 0.5 ul of either 1490 or 2198 (for forward and
reverse sequences). The samples were then cycle sequenced with 25 cycles of heating to 96°C
for 10 seconds, 50°C for 5 seconds, and 60°C for 4 minutes.
After the samples cycle sequenced, 40 ul 75% isopropanol was added to each, and they
were let to stand for 15 minutes. The samples were then spun at room temperature at about 3000
rpm for an hour to form DNA pellets. The tubes were then uncapped, turned over, and spun for
two minutes at 700 rpm to remove excess fluid. To resuspend the DNA, 20 ul of hi-di was
added to each tube. The tubes were then vortexed to ensure the DNA mixed in the hi-di and then
briefly spun to bring all the liquid to the bottom of the tubes. The samples were heated at 96°C
Blazyk 7
for two minutes to denature the DNA. Äfter cooling to room temperature, the samples were
transferred to a sequencing plate and sequencing using an ABI3 100 Genetic Analyzer.
Post-sequencing analysis
Äfter sequencing, the files were cleaned and sorted, using Sequencher 4.1. The forward
and reverse files for each snail were combined to create one consensus sequence per snail. These
consensuses were accumulated into a master contig, which was used in making trees and in
statistical analysis. PAUP*4.Ob10 (PPC) was used to make parsimony trees with branch lengths
in terms of numbers of nucleotide changes. Structure among clades was analyzed using
contingency tables and chi-squared values with the program ChiSquare v1.0. To compare
diversity within sequences from one location to diversity of sequences among two or more
locations, Fsr values were calculated. This was done using heap big Power Mac, programmed
by Stephen Palumbi. Pairwise Fsr values (between each pair of locations) were then correlated
with distance between populations to examine whether values increased with distance, as would
be expected by the isolation by distance model.
RESULTS
Amplification and Sequencing
The first several PCR conditions tried were unsuccessful, producing no visible bands of
products, including the positive control (Fig. 2). In these unsuccessful reactions, I was using
0.15 ul of enzyme (both AmpliTag and DNAzymeEXT), as well as a number of different primer
pairs, including 1490/2198, LCO/HCO, COla/COIf, and conch 2F/2198. After doubling the
amount of enzyme per reaction to 0.30 ul, and using the degenerate primers 1490 and 2198, I
was finally able to consistently produce successful reactions (Fig. 3). In this figure, bands of
Blazyk 8
amplified DNA about 700bp long are visible. About 90% of the PCR products sequenced
produced clean enough data to be used in analysis. In total, there were 67 sequences from snails
from seven locations, giving an average of about ten sequences per location.
Parsimony and Minimum Spanning Trees
I first created a parsimony tree with the 67 sequences I had produced, plus the published
sequence for Tegula funebralis COI found on GENBANK, which represents an individual from
Baja California (Fig. 4). This tree is one of a hundred randomly generated trees, all of which
contained the same major clades, including matching groupings of snails. These trees represent
the most parsimonious relationship between the snails, meaning that each base change occurs as
few times as possible (ideally, just once). This same data can be viewed in another way as a
minimum spanning tree (Fig. 5). Here, identical sequences are grouped together in circles, with
divergent sequences radiating out from those circles on branches proportional in length to the
number of nucleotide changes that occur in the sequence.
Statistical Analysis
First, I selected three major clades from the parsimony tree (Fig. 4). I chose these clades
because they seemed to be the largest groupings of sequences. Most of the other branches on the
tree led to only one or two individual snail sequences. I then examined the makeup of these
major clades by creating a contingency table (Table 2). To test whether the distribution of clades
in any location was significantly different from the total, I used a chi-squared test. The true
value of chi-square was 9.828, with p = 0.08 + 0.05 (not significant).
To compare diversity between locations to diversity within locations, l used the measure
of Fsr. Including all seven populations as separate demes, Fsr was negative (although not
significantly different from zero). I then calculated pairwise Fsr values for all pairs of locations
Blazyk 9
(Tables 3 and 4) and graphed these values over distance (Fig. 6). Here, the linear regression line
had an R“ of 0.5035 and a slope of 1x10“, with p = 0.000. Changing negative values of Fsr to
zero (Fig. 7), the linear regression had an R* of 0.5894 and a slope of 5x10%, still with p - 0.000.
DISCUSSION
From the parsimony tree, no striking structure or clades corresponding to locations are
evident. There appear to be individuals from each location spread throughout the tree. However,
upon closer inspection, some subtle patterns are noticeable. For example, Clade II is made up of
eight individuals, four of which are from the Oregon population. None of the other four
members of this clade is from south of Soberanes. With current sample sizes, this is not
conclusive, yet may suggest that structure is not truly lacking.
The results of the chi-squared test show that there is not a significant difference between
the distribution of snails from Oregon, Lompoc, and the remaining samples across the three
clades specified. The test was not too far from significant, with p - 0.08, giving further reason to
think that a larger sample size could resolve some hints of differences in structure between
populations.
Using the principles of the island model, in which all populations have an equal chance
to exchange individuals, Wright and Malécot first developed analyses of the relatedness of
populations in the 1940s. Wright, in fact, derived the formula for Fsr, relating it to population
size and migration rate (Fsr = 1/(4Nm+1)). The “st“ stands for the relationship between the
subpopulation and the total population, and is a type of "inbreeding coefficient" (Templeton
2003). This has now become one of the most standard measures in population genetics to show
population structure.
Blazyk 10
To examine the relationship between genetic variation within and between populations,
calculated Fsr values. To obtain an Fsr for two or more populations, the ratio between average
genetic variation within populations and between populations is subtracted from 1. If there is a
large difference between populations compared to within, the Fsr will approach 1, while if two
populations are not genetically different, Fsr will be approximately 0. In this study, comparing
all seven populations produced a negative Fsr (not statistically different from zero), which
indicates that there is a high level of gene flow between the populations.
1 thought that this might be a result of some populations obscuring differences in others,
so I decided to calculate pairwise Fsr values for all sets of locations. The majority of these were
negative, except the values between the Oregon populations and the other locations. This led me
to believe that there might be a correlation between Fsr and geographic distance between
populations. Strawberry Hill, Oregon is much farther from the other locations than they are from
one another. Indeed, when I graphed pairwise Fsr values against distance and ran a linear
regression, there was a significant positive slope, both when 1 included negative values of Fsr
and when I changed the negative values to zero (which should be the minimum value of Fsr).
By examining the slope of the regression lines on the plots of Fsr versus distance, it is
possible to estimate the average dispersal distance of that species. In both Fig. 6 and Fig. 7, the
Fsr is about 0.04 at a distance of 1000km. I compared this value to a calibration graph (Palumbi
2003), which predicted a mean larval dispersal of about 40 kilometers. This value is much less
than the hypothetical maximum (over 120 km), which may suggest that factors other than the
California current affect dispersal.
The graphs of Fsr versus distance indicate that although there may be a fairly high level
of gene flow along the American west coast (based on the fairly homogenous phylogenetic trees
Blazyk 11
and the mostly small/insignificant Fsr values), there is probably an isolation by distance effect
acting as well. Isolation by distance is based on the stepping stone population model, which is
the theory that gene flow is restricted with increasing geographic distance, resulting in a genetic
structure. This is likely more realistic than the island model, since populations very far apart
would be expected to exchange fewer individuals than closer populations, even if the species has
a moderate or high dispersal potential.
I then calculated values of Nm for each of the pairwise combinations including Oregon
that had a positive Fsr (using the formula Fsr = 1/(4Nm+1)). The Oregon and Lompoc
populations gave an Nm of about 3, which represents the number of migrants per generation
between those two populations (Bohonak 1999). Exchange rates between Oregon and the other
locations had an average of about 7 migrants per generation, although one site suggested an Nm
as high as 40. Even with a rate as low as 3-7 migrants per generation, any major genetic
differences between populations would be prevented.
Numerous studies mentioned previously, as well as my results, support the likelihood that
many species with pelagic larvae have relatively high gene flow along this coast. Ford and
Mitton (1993) and Debenham et al. (2000) showed that Point Conception and Monterey Bay do
not obstruct dispersal of the pink barnacle or the red sea urchin, respectively. In the present
study, suggested barriers such as Monterey Bay and San Francisco Bay appear to not impede
dispersal very significantly, at least not to an extent detectable with the statistical power of my
data.
The connection between <span style='background-color:#cfc' title='Tegula : Exact'>Tegula</span> population structure and implications for marine reserves
may not be immediately apparent, since it is unlikely a reserve would ever be created for the
purpose of protecting these snails. However, many other marine species (including several
Blazyk 12
crustaceans, algae, mollusks, and many fish) have a pelagic larval phase of similar duration or
have comparable dispersal distances (Shanks 2003). This means that although <span style='background-color:#ffcc00' title='Tegula funebralis : PartialExact'>Tegula funebralis
may</span> not be a target species for a reserve, their population structure may be representative of
many other species that do need to be protected.
Most species with pelagic larvae appear to have dispersal distances greater than 20
kilometers, with many on the order of hundreds of kilometers (Shanks et al. 2003). This means
that it may often be impractical to create individual reserves of ideal size, remembering that the
ideal is equal to the average larval dispersal distance for targeted species. Currently, there is no
single no-take zone in the United States that is as large as 20-50 km (Palumbi 2001), let alone as
large as would be required for species with long-lived pelagic larvae. An alternative design is a
network of smaller reserves, although these must be close enough to allow significant dispersal
and recruitment between reserves (Airamé et al. 2003).
Another difficulty is that fact that delineating habitat as no-take reserves usually means
directly taking away that area from fishermen. At the moment, no-take reserves only comprise
about 0.1% of marine habitat in the US. In order to have controlled fish population sizes, it has
been estimated that this area should increase to as much as 50%. This value is impractical, at
least at this time, since it would result in serious declines in fishery harvests, despite growth of
overall fish populations (Palumbi 2001). It is clear that some equilibrium must be met, for the
current trend is that of fishing populations until they all but disappear and then moving on to
another species. It is still unclear, however, whether setting aside 20% of marine habitat as
reserves would provide a happy solution to both fishermen and declining fish populations, or if it
would merely hurt the economic value of the fisheries while not being enough to regenerate
declining marine populations.
Blazyk 13
To increase the usefulness of this study, I would like to increase sample sizes from each
location, while adding additional locations to the north and south, to investigate further the
apparent effect of isolation by distance. Also, I plan to sequence additional sections of COI, or a
section of another gene to isolate a greater number of polymorphic sites that would add power to
fürther resolve genetic differences between populations. With current populations and sample
sizes, there are no apparent significant differences, suggesting fairly high levels of gene flow.
However, Fsr does increase significantly with distance, which shows that there is not unlimited
exchange of larvae between all locations studied. Rather, there is a balance between fairly high
larval exchange and isolation by distance in Tegula funebralis between Oregon and southern
California.
Blazyk 14
ACKNOWLEDGEMENTS
I would like to thank Stephen Palumbi for being a great advisor and knowing the level of
involvement that allowed me to learn and explore things myself, without ever feeling completely
lost. Also, thanks to Emily, who also worked on snail genetics, and who knows everything about
scientific labs, having spent many summers in such thrilling environments. I also owe a lot to
everyone in the Palumbi lab—Vollmer, Julie, Laura, Cathy, Roxanna, Tom, and Adam. You
guys always helped me out, answered my questions, and even took the time to speculate with me
about Tegula genetics. I would also like to thank the other 175H professors—George Somero
for being a wonderful advisor and helping me get snails from various locations, and Mark Denny
and Jim Watanabe for general advice and input. Lastly, thanks to all the sea kids for making the
first five weeks of the quarter raucous and fun, and to all the other 175H/176H students for the
lovely, although somewhat quieter, times the rest of the quarter.
Blazyk 15
LITERATURE CITED
Airamé, Satie, Jenifer E. Dugan, Kevin D. Lafferty, Heather Leslie, Deborah A. McArdle, and
Robert R. Warner. 2003. Applying ecological criteria to marine reserve design: a case
study from the California Channel Islands. Ecological Applications. 13: S170-S184.
Bohonak, Andrew J. 1999. Dispersal, gene flow, and population structure. The Quarterly Review
of Biology. 74:21-45.
Buonaccorsi, Vincent P., Carol A. Kimbrell, Eric A. Lynn, and Russell D. Vetter. 2002.
Population structure of copper rockfish (Sebastes caurinus) reflects postglacial
colonization and contemporary patterns of larval dispersal. Can. J. Fish. Aquat. Sci. 59:
1374-1384.
Debenham, Patty, Mark Brzezinski, Kathy Foltz, and Steven Gaines. 2000. Genetic structure of
populations of the red sea urchin, Strongylocentrotus franciscanus. J. of Experimental
Marine Biology and Ecology. 253: 49-62.
Folmer, O., M. Black, W. Hoeh, R. Lutz, and R. Vrijenhoek. 1994. DNA primers for
amplification of mitochondrial cytochrome c oxidase subunit I from diverse metazoan
invertebrates. Mol. Mar. Biol. Biotech. 3: 294-299
Ford, Michael J. and Jeffry B. Mitton. 1993. Population structure of the pink barnacle, Tetraclita
squamosa rubescens, along the California coast. Mol. Mar. Biol. Biotech. 2: 147-153.
Hamm, D. E. and R. S. Burton. 2000. Population genetics of black abalone, Haliotis cracherodii,
along the central California coast. J. of Experimental Marine Biology and Ecology. 254:
235-247.
Hellberg, Michael E. 1998. Sympatric sea shells along the sea’s shore: the geography of
speciation in the marine gastropod Tegula. Evolution. 52: 1311-1324.
Blazyk 16
Hoskin, M. G. 1997. Effects of contrasting modes of larval development on the genetic structures
of populations of three species of prosobranch gastropods. Marine Biology. 127: 647-656.
Miner, Benjamin G. 2002. Are the two physiological races of Pollicipes polymerus <span style='background-color:#cfc' title='Cirripedia : Exact'>(Cirripedia)</span>
genetically divided along the California coast? Invertebrate Biology. 121: 158-162.
Moran, A. L. 1997. Spawning and larval development of the black turban snail Tegula funebralis
<span style='background-color:#cfc' title='Prosobranchia : Exact'>(Prosobranchia:</span> <span style='background-color:#cfc' title='Trochidae : Exact'>Trochidae).</span> Marine Biology. 128: 107-114.
Palumbi, Stephen R. 2001. The Ecology of Marine Protected Areas. pp. 509-530 in Mark D.
Bertness, Steven D. Gaines, and Mark E. Hay, eds. Marine Community Ecology, Sinauer
Associates, Inc., Sunderland, Massachusetts.
Palumbi, Stephen R. 2003. Population genetics, demographic connectivity, and the design of
marine reserves. Ecological Applications. 13: S146-S158.
Shanks, Alan L., Brian A. Grantham, and Mark H. Carr. 2003. Propagule dispersal distance and
the size and spacing of marine reserves. Ecological Applications. 13: S159-S169.
Templeton, A. 2003. Gene flow and population subdivision. pp.6-1 to 6-34 in A. Templeton
Population Genetics and Microevolutionary Theory, Wiley & Sons, in press.
Blazyk 17
APPENDIX 1: TABLES
Table 1: This table organizes data about the samples used in this project. It includes the "snail
1D" used in labeling samples in various trees and tests. Also listed are the collection locations,
with latitude and longitude. Finally, the number of snails from each site is recorded, as well as
the number of those that were successfully sequenced.
Snail ID
Collection Location
Latitude
Longitude
sequenced
124°07°36”W
ORI-ORIS
44°15°42”N
Strawberry Hill, OR
11
38°19'N
123°04’W
BBII-BB20
Bodega Bay, CA
10
37°29’42'’N
122°29°42°’W
10
HMBI-HMB10
Half Moon Bay, CA
Hopkins Marine
HPI-HP10;
36°37'N
20
121°54’W
15
Station, Pacific Grove,
HEI-HE10
CA
36°27°09°N
121°5545°W
Soberanes Point, CA
11
10
SPI-SP11
35°10’36”N
120°45°36”W
10
SLOI-SLO10
San Luis Obispo, CA
LMII-LM20
34°43'N
120°34’
Lompoc, CA
Blazyk 18
Table 2: This table shows the contingency table used in analysis. The phylogenetic tree (Fig. 4)
was broken into 3 major clades—I, II, and III, with everything else as "Rest." The samples from
Oregon (OR), Lompoc (LM), and everywhere else grouped together (Rest) are listed below with
appropriate distributions between clades.
Clade
I
II
III
Rest
4
OR
2
2
4
5
Rest
15
24
4
LM
Blazyk 19
Table 3: This is a list of average nucleotide distance (diversity) within each location.
Average diversity within location
Location Code
Location
OR
0.00430
Strawberry Hill, OR
BB
0.00530
Bodega Bay, CA
HMB
0.00556
Pillar Point, Half Moon Bay, CA
Hopkins Marine Station, Pacific
HE/HP
0.00458
Grove, CA
SP
0.00408
Soberanes Point, CA
San Luis Harbor, San Luis
SLO
0.00504
Obispo, CA
0.00259
LM
Lompoc, CA
Blazyk 20
Table 4: Below are average nucleotide differences within and between pairs of locations, which
are used to calculate Fsr values, also listed. In addition, distances between the pairs of locations
are given in kilometers.
Diversity between
Distance between
Avg. diversity
Locations
FST
locations
locations (km)
within locations
0.003445
0.00368
1103.4
0.06556
ORLM
0.00485
0.05675
ORSLO
0.00507
1049.5
0.00419
ORSP
0.00411
-0.01946
887.4
ORH
0.00444
870.0
0.03986
0.00462
ORHMB
0.00493
0.00496
0.00605
764.3
0.00502
ORBB
0.00480
666.5
0.04382
0.00383
-0.03003
0.003945
458.1
BB/LM
-0.07000
0.00535
0.00539
404.8
BB/SLO
-0.05157
0.00469
BB/SP
0.00446
230.2
-0.04000
0.00494
0.00475
BB/H
215.1
-0.09919
0.00494
BB/HMB
0.00543
104.2
0.00379
353.9
-0.07520
HMB/LM
0.004075
0.00539
-0.01670
HMBSLO
0.00548
300.8
-0.08559
HMB/SP
0.00444
126.3
0.00482
0.00475
110.9
-0.06737
HMB/H
0.00507
-0.01558
0.00353
0.003585
243.0
HLM
0.00499
190.1
0.00500
H/SLO
0.00200
-0.05353
H/SP
0.00411
18.4
0.00433
-0.01988
0.00327
0.003335
228.8
SP/LM
0.01660
SP/SLO
0.00474
0.00482
176.6
0.01396
0.003995
0.00394
54.1
SLO/LM
Blazyk 21
APPENDIX 2: FIGURES
Fig. 1: Map of the west coast of the United States with collection sites marked in red.
Fig. 2: Example of an unsuccessful PCR, with products run on a 2% agarose gel. There are no
clear bands, even for the positive control. The lane of many bands is the 1kB standard.
Fig. 3: Example of a successful PRC, with products run on a 2% agarose gel. There are clear
bands at about 700bp for most samples, including the positive control, and an empty lane for the
negative control. The 1kB standard is in the first lane.
Fig. 4: Parsimony tree produced with all 68 sequences. Horizontal branch lengths represent
numbers of nucleotide changes. Included are three major clades used in analysis.
Fig. 5: Minimum spanning tree with identical sequences grouped in circles and divergent
sequences radiating outwards on branches proportional to the number of nucleotide changes.
Fig. 6: Plot of pairwise Fsr versus distance (km), including negative values of FsT.
Fig. 7: Plot of pairwise Fsr versus distance (km), with negative values of Fsr changed to zero.
Fig. 1
Strawberry <span style='background-color:#cfc' title='Hille : Exact'>Hille</span>
Bodega Bay
Half Moon Baye
HMS
Soberanes
San Luis Obispoy
Lompoc


Blazyk 22
Blazyk 23
Blazyk 24
0
Fig. 4
B817
HMBIO
8512
ORI3
HEE
BB19

ORI
s
ORI
SBI8
— S820
SLO9
SP9
0.5 changes



Tegula funebralis CO
— HMBS
BB13
BB14
B16
HEG
— EMBS
OR4
SB14
HE9
2
5105
ORII
BB18
Hr3
— HMBS
— HP10
S10
910
-8p11



Clade 1
Clade II
Clade III
Blazyk 25
SLOA¬
Fig. 5
SPI1
S105
BBI4
NO S0
HEIO HMB7 BBI? HPS
ORi4 SB13 SB159819 HE3
)SPI SP2 SPS SP10
BBI3

SP6 SP8
o
HMB8
Tegulh furebraks c0
— HP7
EIORBBIS
9917590
S109
HMB6
HE8
()
SLO1 HP10
HEA
HMB4
Blazyk 26
-HP3
AMESBBI8
sio2
Fig. 6
y- 1E-04x- 0.0604
Pairwise Fsr v. Distance
8=0.5035
0.08
0.06

0.04
0.02
-0.02 0
—200
—600800 • 10001200
-0.04

-0.06
0
-0.08
-0.1 -
-0.12
Distance between locations (km)
Blazyk 27
Fig. 7
Pairwise Fsr versus Distance
y = 5E-O5X - 0.0083
(without negative values)
R/=0.5894
0.08
0.06
0.04

0.02
oee
400 600 800 1000 1200
200
-0.02
Distance between locations (km)
Blazyk 28
</pre></body></html>