<html><body><pre style='margin:2rem auto;width:fit-content'>Abstract
Übiquitin is a small protein-binding protein that is found in all
eukaryotes. It displays a high degree of conservation among organisms. It is
involved in many cellular functions including DNA repair, cell cycle control,
and the stress response. One of its primary mechanisms of function is to mark
proteins for selective degradation. There are several classes of ubiquitin¬
protein ligating enzymes that are involved in the ubiquitin system, (El
ubiquitin activating enzyme, (E2) ubiquitin-conjugating enzyme, and (E3
ubiquitin-protein ligase. E2's, the ubiquitin-conjugating enzymes (UBC's), confer
specificity to the ubiquitin by selecting which proteins to tag with ubiquitin. In
the yeast, <span style='background-color:#cfc' title='Saccharomyces : Exact'>Saccharomyces.</span> cerevisiae, two UBC's have been shown to be
induced by heat shock and are inviolved in the stress response. <span style='background-color:#cfc' title='Tigriopus californicus : Exact'>Tigriopus
californicus,</span> a copepod abundant in the high splash pools of the California
Coast, displays an ability to tolerate wide environmental fluctuations in
temperature and salinity. Its response to thermal and osmotic stress makes it a
logical choice as an organism with which to study UBC's and their
relationship to the stress response. DNA and RNA were successfully isolated
from <span style='background-color:#fcc' title=' : NoMatch'>T. californicus.</span> Three genes coding for ubiquitin were found. One
ubiquitin-conjugating enzyme gene was found and sequenced (ÜBCTI). The
homology between the UBCTI gene and yeast UBC4 and UBC5 is extremely
high, indicating that UBCTI expression may also be induced during heat shock
and play a role in the stress response. The groundwork for future study of UBC's
and their relationship to stress in T. californicus has been set.
Ubiquitin is a 76 residue protein binding protein that is present in all
eukaryotes. It shows an extremely high degree of conservation among
organisms with its amino acid sequence being identical between animals and
differing by only three amino acids when compared to the yeast,
<span style='background-color:#cfc' title='Saccharomyces cerevisiae : Exact'>Saccharomyces cerevisiae</span> . A primary function of ubiquitin is marking
proteins for selective degradation by ubiquitin-conjugate
proteases (proteasome). Ubiquitin is involved in many cellular events
including DNA repair, cell cycle control, and the stress response (Finley and
Chau, 1991).
The ubiquitin system is very complex and consists of over forty gene
products. Among these are the enzymes of the ubiquitin-protein ligase
system. These enzymes act to bind ubiquitin to specific proteins. The El
enzymes are ubiquitin-activating enzymes that activate the C-terminus on
ubiquitin which eventually binds to the protein to be tagged. The E2's,
ubiquitin-conjugating enzymes (ÜBC's), confer specificity to the ubiquitin
system by leading ubiquitin to the protein to be targeted. The E3's, ubiquitin¬
protein ligases, act to ligate ubiquitin to selected proteins and are also
involved in conjugating a chain of ubiquitin to a monoubiquinated protein
(V. Kampen, J. and M. Wettern, 1992).
This paper is concerned with the stress function of the ubiquitin system
and is focused on the ubiquitin-conjugating enzyme. In the yeast <span style='background-color:#fcc' title=' : NoMatch'>S. cerevisiae
ten</span> UBC genes have been sequenced and studied. The expression of two of
these genes, UBC4 and UBC5, is induced by heat shock and is involved in the
stress response. Yeast ubcaubc5 double mutants show an inability to survive
high temperatures or in the presence of an amino acid analog that causes
abnormal protein synthesis. In addition, loss of UBC4 and UBC5 function
induces the stress response in yeast (Jentsch and Seufert, 1990)
The copepod Tigriopus californicus is an ideal organism for studying
stress as its life in the high splash pools of the western coastline subject it
wide temperature and salinity fluctuations. The survival of this species
depends on its tolerance of conditions in which predators cannot survive. It
can withstand salinities of up to six times that of normal sea water and
temperatures of up to 40° C. In addition to its ability to cope with a range of
conditions, it is a hardy animal that is easily maintained in the laboratory and
is found in abundance along the California Coastline (Haderlie et al., 1980).
The expression of UBC4 and UBC5 in yeast has been shown to be
induced by stress, and the synthesis of a family of heat shock proteins (HSP's),
the HSP 60's, in T. <span style='background-color:#cfc' title='Californicus : Exact'>Californicus</span> is inducible by temperature and salinity stress
(G. Ha and T. McEvoy, personal communication). The results presented here
lay the foundation for study of UBC's in T. Californicus by reporting on the
cloning and sequencing of the first UBC discovered in this marine organism,
UBCTI.
Materials and Methods
Collecting T. californicus. T. californicus were collected from a high
splash pool behind the Jacques Loeb building at Hopkins Marine Station in
Pacific Grove, CA. The specimens were captured by using a tea strainer and
held at room temperature in 18" x 11" x 6" Tupperware containers filled with
filtered sea water. The animals were fed three times a week with ground
mussels provided by Dr. George Matsumoto (Hopkins Marine Station, Pacific
Grove, CA). New specimens were collected every two weeks.
Isolation of DNA. Specimens were concentrated by in a 15 mL
polypropylene Falcon tube until they filled about 5 mL. After concentration,
they were anesthetized with 90% ethanol and isolation of DNA was
performed following a protocol used in study of the ubiquitin system in the
green algaeChlamydomonas reinhardtii (Dr. Jan von Kampen, personal
communication).
Isolation of RNA. Isolation of T. californicus RNA was performed
following a protocol used for isolation of sodium channel mRNA from rat
skeletal muscle (Cooperman, S. S. et al., 1987).
Southern Blot. Isolated genomic T. californicus DNA was restricted by
Pstl, EcoRl, and EcoRV restriction enzymes (Promega, Madison, WI). The
restricted DNA was run on a one percent agarose with a Lambda DNA ladder
(Promega). The DNA was transferred onto a sheet of Hybond H+ Nylon
Membrane (Amersham Arlington Heights, IL) by the Innovative Southern
Blot method (Dr. Jan von Kampen, personal communication). Once the
DNA was denatured the transfer was performed by sandwiching the gel with
Whatman paper 3mm and Hybond N+ Nylon membrane (Amersham) on
one side and four sheets of Whatman paper 3mm on the other. The stack
was then placed on a gel dryer with the nylon membrane below the gel and
dried at 80° C until the gel was flat. The DNA was fixed by baking at 80° C for
two hours. The DNA was probed with a P-32 labeled ubiquitin gene(ÜBM)
from <span style='background-color:#fcc' title=' : NoMatch'>C. reinhardtii,</span> kindly provided by Dr. Jan von Kampen. The nylon
membrane was then exposed on X-ray film (Kodak XAR-5 Rochester, NY) for
18 hours and then developed.
Gene cloning and analysis. The polymerase chain reaction (PCR) was
performed using isolated T. californicus genomic DNA and two degenerate
oligodeoxynucleotide primer:
OCP9 15' GAGGATCCTA(T,C)GA(A,G)GG(T,A)GG(T,A)GT(TOTT(TCTT 3"
Dj2 I5' CTGTCGACAC(A,G,C,T)GC(G,A)TA(CTTT(CTTT(A,GCTGTCCA
(C.T)TC 3’] (Oligo's Etc., Gilford, CT)
The primers were developed from the conserved regions of UBC4 and UBC5
in yeast and ÜbcDl in <span style='background-color:#cfc' title='Drosophila : Exact'>Drosophila</span> (Zhen et al., 1993). The PCR was performed
in 100 ul reaction mixtures containing 400 ng of genomic DNA, 500 pmol of
each primer, 10 ul of Taq polymerase 10X Buffer (Promega), 1 nmol of each
deoxynucleoside triphosphate, 10 units of Taq polymerase (Promega), and 0.25
umoles of MgCl2. The amplification consisted of 35 cycles of 94° C for 30
seconds, 55° C for 90 seconds, and 72° C for 90 seconds followed by a 10
minute incubation at 72° C. The product was precipitated overnight with 0.1
volume ammonium acetate and 3 volumes of ethanol and resuspended in 20
ul of sterile water. A second PCR was performed in a total volume of 50 ul
containing 15 ul of resuspended DNA, 50 pmol of each primer, 5 nmol of each
deoxynucleosides triphosphates, 25 nmol of MgCl2, 5 ul of Taq polymeraseloX
Buffer, and 1 unit of Taq polymerase. The products were precipitated
overnight with 0.1 volume sodium acetate and 3 volumes of ethanol and
resuspended in 20 ul of sterile water. A 5 ul sample was run on a two percent
agarose gel to confirm amplification of a fragment of the expected size,
roughly 309 base pairs. The product was then run on a two percent low-
melting agarose gel and bands at 309 base pairs were cut and eluted using the
Magic PCR Mini-Prep Kit (Promega).
The fragment was then inserted into pBluescript SK+ vectors
(Stratagene, La Jolla, CA), which had been restricted with EcoRV, by blunt end
ligation. Transformation into the XLI-Blue strain of <span style='background-color:#fcc' title=' : NoMatch'>E. coli</span> was performed
according to standard methods (Maniatis, 1982). After proliferation, the
vector with the inserted fragment was isolated from XLI-Blue using the Magic
Maxi-Prep Kit (Promega). A sample of the vector was restricted with BamHI
and Sall (Promega) and run on a one percent agarose gel to confirm that
amplification was successful.
The inserted fragment was sequenced using Sequenase 2.0 from US
Biochemicals (Cleveland, ÖH) and T3, T7(Promega), and -40 Universal (US
Biochemicals) primers. Sequenase 2.0 products were run on a 6%
polyacrylamide gel and bands were visualized by autoradiography (Kodak
XAR-5 film). Analysis of sequence was performed using the Genalign, Sec
and Gened programs from Intelligenetics (Mountain View, CA). The
sequence was first translated into an amino acid sequence and then compared
to the UBC4 and UBC5 amino acid sequences of the yeast <span style='background-color:#fcc' title=' : NoMatch'>S. cerevisiae.</span>
Results
Electrophoretic separation of T. californicus DNA on a one percent
agarose gel revealed a single distinct band at about 20 kilobases. The same
separation performed on RNA gave three distinct bands.
The T. californicus genomic DNA digested with three restriction
enzymes EcoRV, EcoRl, and Pstl resulted in different degrees of digestion.
EcoRV digested more completely revealing a smeared DNA band where as
Pstl and EcoRl were less effective with their corresponding bands being more
distinct. The sharpness of the bands indicate that the DNA was in relatively
larger fragments (figure 3).
Following hybridization of the P-32 labeled UBM probe to the digested
genomic T. californicus DNA and exposure to X-ray film for 18 hrs, at least
three distinct bands were seen on the resulting autoradiograph. A
reproduction of the autoradiograph is not included due to its low resolution.
A fragment of about 310 base pairs was observed when PCR products
and the restricted vector with fragment insert were separated on a two percent
agarose gel by electrophoresis. (figure 4 and 5). The sequencing of this
fragment resulted in the sequence given in figure 6.
The obtained DNA sequence was translated into an amino acid
sequence and is shown in figure 7 with the yeast UBC4 and ÜBC5 amino acid
sequence. Table I gives the percent of amino acid identity between the
sequenced fragment from T. Californicus and the yeast UBC4 and UBC5
amino acid sequences.
Discussion
T. californicus is an ideal organism in which to study the effects of
stress on the expression of ubiquitin-conjugating enzyme. This is true for
several reasons. First, both high molecular weight DNA and intact RNA are
efficiently isolated with good yield and purity. In figure 1 the uppermost
band of the molecular weight marker labeled M is 21.7 kilobases. The three
juxtaposed bands to the right are T. Californicus DNA and are estimated to be
roughly 19 kilobases. In addition to being relatively long, the DNA also
appears as sharp, distinct bands indicating a high degree of purity. Figure 2
shows that the RNA has similar characteristics. The ribosomal bands are
fairly distinct indicating the RNA is both pure and intact. Efficient isolation
of RNA that has not been degraded is a critical step in Northern blot analysis
which can be used to characterize the nature of ÜBCTI expression.
Southern blot analysis confirmed that ubiquitin was present in T.
Californicus. The autoradiograph of T. californicus DNA in the EcORV
restriction that was hybridized with the UBM probe from C. reinhardtii
showed at least three bands, thus providing strong evidence that at least three
genes coding for ubiquitin exist in T. californicus. This conclusion assumes
that the probe hybridized with only intact genes or that the restriction enzyme
did not cut within an ubiquitin coding region. Restricting the genomic DNA
with more restriction enzymes will reveal the plausibility of these
assumptions. The discovery of ubiquitin genes in T. californicus is not
surprising, however, considering that all eukaryotes studied thus far possess
the ubiquitin system. The poor resolution of the autoradiograph prevented
its reproduction for this paper.
Having established the existence of ubiquitin in T. californicus, the
presence of at least one gene coding for ubiquitin-conjugated enzyme was
very likely. The primers OCP9 and Dj2 prime for a conserved ubiquitin¬
conjugating enzyme coding region in yeast UBC4 and ÜBC5 and ÜbcDl in
Drosophila that is roughly 309 base pairs long. Figure 4 shows the products of
the PCR separated by two percent agarose gel electrophoresis. The expected
fragment can be seen to be roughly 309 base pairs. Following cloning,
restriction of the vector revealed a fragment again at 309 base pairs, indicating
successful cloning (figure 5 - lane 5 from the right.).
Final confirmation that the T. californicus homologue of yeast UBC4
and UBC5 had been cloned was established by elucidating the DNA sequence.
Each end of the cloned fragment was identical to the sequence of the primer
used for PCR indicating that the isolation and cloning was successful in
amplifying UBCTI (see figure 6.). Comparison of the amino acid sequences
shows a remarkable level of amino acid identity. Of particular interest is the
region circled in figure 7. The cystine residue in this portion of the sequence is
crucial for the ubiquitin-UBC thioester bond and the catalytic core of E2's
(Finley and Chau, 1991). The complete amino acid identity in this region is
further indication that the gene sequenced is a homologue of the yeast UBC4
and UBC5 genes.
The high degree of conservation between yeast UBC4 and UBC5 and
UBCTI suggest that ÜBCTI expression can also be induced by heat shock and
other stresses. Further examination of this question by Northern blot analysis
of RNA isolated from T. californicus subjected to various stresses such as
salinity and temperature is needed to confirm this hypothesis. This study has
established T. californicus as a marine organism that is amenable to further
investigation of ubiquitin-conjugating enzymes and the ubiquitin system and
their relationship to heat shock and stress response.
Acknowledgments
I am indebted to Dr. Jan von Kampen and Julie Golden for their patience and
guidance. I thank Greg Ha for the outdoor adventures and for being a partnei
in "harshing" on our lab mates. I also thank Christine Shin and Theresa
McEvoy for their comic relief and taking the "harshing" without retaliation. 1
am eternally grateful to Dr. R. Paul Levine for providing me the opportunity
to study under his tutelage and for assembling one of the finest labs at
Hopkins Marine Station.
Literature Cited
Cooperman, S. S., S. A. Grubman, R. L. Barchi, R. H. Goodman, and G.
Mandel. 1987. Modulation of sodium channel mRNA levels in rat skeletal
muscle. Proc. Natl. Acad. Sci. USA. 84: 8721-8725.
Finley, D. and V. Chau. 1991. Ubiquitination. Ann. Rev. Cell Biology. 7: 25-
Haderlie, E., D. Abbott, and R. Caldwell. "Three Öther Crustaceans: A
Copepod, a Leptostracan, and a Stomatopod." Intertidal Invertebrates of
California. Stanford University Press, Stanford, California 1980. p. 631-632.
Maniatis, T., E. F. Fritsch, and J. Sambrook. 1982. Molecular Cloning. A
Laboratory Manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor,
New York.
Seufert, W., S. Jentsch. 1990. Übiquitin-conjugating enzymes UBC4 and UBC5
mediate selective degradation of short-lived and abnormal proteins. 9: 543-
550.
V. Kampen, J. and M. Wettern. 1992. Übiquitin-abhangige. Degradation und
Modifikation von Proteinen. Naturwissenschaften. 79: 163-167.
Zhen, M., Heinlein, R., Jones, D., Jentsch, S., and E. Candido. 1993. The ubc-2
Gene of <span style='background-color:grey' title='Caenorhabditis elegans : Fuzzy'>Caenorhabditis elagans</span> Encodes a Übiquitin-Conjugating Enzyme
Involved in Selective Protein Degradation. 13: 1371-1377.
10
Figure Legends
Figure 1. One percent agarose gel of isolated DNA from T. Californicus.
Upper band of M is 2
1.7 kb and next band is 5..15 kb. Lanes 1, 2, 3 are DNA in
increasing concentration.
Figure 2. One percent agarose gel of isolated RNA from T. Californicus. On
original distinct bands can be visualized.
Figure 3. One percent agarose gel of restricted DNA. The uppermost band of
Mis 21.7 kb. The EcoRV restricted the best showing the most diffuse band of
the three restriction enzymes.
Figure 4. Two percent agarose gel of PCR products. The lower four bands of
the 1 kb standard are 506, 516 bases, 396 bases, 344 bases, and 295 bases. The
two circled fragments are UBCTI.
Figure 5. Two percent agarose gel of vector restricted by BamHI and Sall. The
lower four bands of the 1 kb marker are 506, 516 bases, 396 bases, 344 bases, and
295 bases. The circled fragment in lane five is UBCTI. The upper bands are
the vector in super-coiled, nicked, and linear form.
Figure 6. Sequence of cloned fragment. The ends of the sequence correspond
to the base pairs of the primers indicating that the primed fragment was
sequenced.
Figure 7. The sequence of the cloned fragment that has been translated into
an amino acid sequence. T. Californicus sequence is on top followed by UBC4
and UBC5. Lines indicate homology between sequences.
Table I. The percent amino acid identity between the the yeast UBC4 and
UBC5 sequence and the T. californicus UBCTI sequence.
Figure 1
T.DNA
Figure 2

Figure 4
Peut
Figure 5
Plasmid
Figure 6
6' tacgagggaggagtttttttettgaatattcacttercagttgactatecttteaagccaccaaaggttt
ttogaccagaatttacaraaatattaattacaggttecatetgeettgacatetaaagga
acagtagagtecagctetraccatctecagggttetectatccatcagttecttgttgacegacccaaad
tgatgatetttaatecagagategetacatetataagactgacattaataagtacagagctaseg
agagaataaaaaaatagag
Figure 7
. Califon
eVFFLnIHFFVDY

1
UECAVERE
MSSEEEIOKELSDLEDEFtSSAGFVSDLLYHWCASIMFDSFYAGVFFLSIHFFTDY




SSSERIARELSLLGRDFFASLSAGFVHDDLYHWGSSIMGFSDEFYAGGVFFLSIHFFTDY
UBCEYEAST
ronsensus
msssriakelsdl-rapp-esagpygadlyhugasimgp-depYadVFFLSIHFFtDY
I. Califorr
17 FFKFFKVSFaTT IYHFNINSHGSILLDILKECWSFALTiSFVLLSISSLETDENFDDFLVF
111


CECVE
E FFEFFKISFTTKIYHFNIMARGNICLDILRDUWSFALTLSKVLLSILSLETDANFDDFLVF

E PFEPFKVRFTTKIYHFNINSSGNIULDILRDUWSFALTLSKVLLSICSELTDANFDEFLUF
UBCEYERST
FFEPFKVERtTKIYHFMINS-GRILLDILKGGWSFSLTISEVLLSICSLETDANFLDFLVF
ronsensus
I.Californ
EIGHIYRTDIRRYTATHEEWTFYE

12 EIAHIYKTDTPHYEATHREMTRRYSV
UBC+YEAST

UECS YEAST 1S EIHIYRTLRARYEHTHAENIEETHV
nID—e
RSenUE
UBCTI - UBC4
UBCTI - UBCS
UBCA- UBCS
% AA Identit
84
84
92
Table 1
</pre></body></html>