<html><body><pre style='margin:2rem auto;width:fit-content'>ABSTRACT
Many marine organisms posses a multitoxin, or multixenobiotic, defense
mechanism that enables their cells to resist the effects of a variety of environmental
pollutants. A membrane transport protein similar to the multidrug resistance protein
(MDR) in human tumor cells pumps moderately hydrophobic compounds out of the cell
membrane, thus conferring a multixenobiotic resistance (MXR) to cells. One particular
strain of marine bacteria isolated from the digestive gland of the mussel <span style='background-color:#cfc' title='Mytilus californianus : Exact'>Mytilus
californianus</span> produces or contains compounds that are potential substrates of the MXR
protein in mussel gill tissue. Substances produced by this strain increased the level of
rhodamine accumulation in gill tissue and reduced the rate of rhodamine efflux when
compared with a marine broth control. This strain was isolated separately from two
different mussels collected more than seven weeks apart, indicating that it is endogenous
to the mussel gut, either as a regularly ingested dietary bacteria or as a digestive
symbiont. The presence of the strain in the mussel digestive gland and its high affinity
for MXR suggests a possible evolutionary function for the MXR transporter may be to
protect against potential toxins that the mussel encounters during the digestive process.
keywords: multixenobiotic (MXR) transport, multidrug resistance (MDR), digestive
gland, mussel
INTRODUCTION
The ability of many marine organisms to survive in habitats containing an
abundance of both natural and man-made toxins has raised research interest concerning
the mechanisms by which these organisms resist or tolerate environmental contaminants.
Recent studies have indicated that several marine invertebrates achieve this resilience
through what has been termed a multitoxin, or multixenobiotic, defense mechanism that
enables cells to resist the affects of a variety of environmental pollutants (Cornwall, et. al.
1995; Epel 1998; Eufemia and Epel 1998; Toomey and Epel 1993). A membrane
transport protein uses the energy of ATP to pump moderately hydrophobic compounds
out of the cell, thus conferring a multixenobiotic resistance (MXR) to cells. These
transport proteins were initially characterized in human tumor cells, where a 170 Kda
protein, P-glycoprotein, confers a multidrug resistance (MDR) to a variety of anti-cancer
drugs, relating study of the MXR system in marine invertebrates to medical oncological
research on cancer treatment (Toomey, et. al. 1996).
Research has indicated that, for many marine invertebrates, the MXR protein may
function as a first line of defense against environmental contaminants. The MXR protein
has been previously characterized in mussel gill tissue because of its ability to defend the
organism against man-made toxins such as many moderately hydrophobic pesticides
(Cornwall, et. al. 1995). This is unlikely, however, to be the original function of the
protein in a natural environment. Related research on human cell lines reveals specific
localization of the MDR protein to the apical surface of epithelial cells lining many
human digestive organs, including the liver, pancreas, kidney, jejunum, and colon
(Thiebaut, et. al. 1987). These studies suggest that the protein has a role in the normal
digestive secretion of waste metabolites into bile and urine for excretion. The presence of
the MXR protein in the digestive tract of the filter feeding mussel Mytilus californianus
indicates that the protein may serve an analogous function in mussels by protecting the
organism from toxic products encountered during its natural digestive process.
Mussels acquire nutrients by using mucociliary mechanisms on the ctendia and
labial palps to filter and ingest an array of suspended particles including phytoplankton,
detritus, and bacteria (Gosling 1992). While phytoplankton and other detrital organics
generally comprise the main sources of nutrition, free floating and bound bacteria are
estimated to collectively comprise up to 73% of the mussel’s metabolic requirements for
nitrogen and up to 30% of the metabolic requirements for carbon, making bacteria a
significant and essential component of the <span style='background-color:#cfc' title='Mytilus : Exact'>Mytilus</span> diet (Gosling 1992). The mussels may
also harbor symbiotic bacteria in their digestive tract that, in competition for space and
nutrients, engage in a sort of chemical warfare with each other through secretion of toxic
or noxious substances. By undergoing regular dietary exposure to bacteria and bacterial
products, <span style='background-color:#ffcc00' title='Mytilus : PartialExact'>Mytilus may</span> be subject to these potentially toxic chemical byproducts.
Resistance to accumulation of such dietary or symbiotic bacterial toxins is hypothesized
to be one of the primary functions of the MXR protein.
This research project investigates the evolutionary role of the MXR transport
protein in the digestive tract of mussels. I hypothesize that symbiotic or ingested bacteria
in the digestive tracts of the mussel Mytilus californianus produce or contain secondary
metabolites that may be toxic and that these products are also substrates of the MXR
transporter.
METHODS
Isolation of Digestive Bacteria
The studies were conducted using bacterial samples isolated from the digestive
tract of the mussel, Mytilus californianus. The mussels were collected from the rocky
intertidal at Hopkins Marine Station. A half inch portion of the digestive gland was
extracted from the dissected mussel and Dounce homogenized in one ml sterile seawater.
One, 10, 50, and 100 ul aliquots of this homogenate were plated onto marine agar plates
that were then incubated at 15°C for 5-6 days. The bacterial strains on each plate were
phenotypically compared and recorded. The four most common strains were selected and
streaked onto fresh marine agar plates to isolate individual colonies. These plates were
stored at 4°C for further use.
Extraction of Digestive Bacteria
Extracts were made both from the pellet (to isolate internal compounds that would
be released upon death of the bacteria), as well as from the supernatant (to isolate soluble
compounds excreted by the bacteria).
5 ml cultures of marine broth were each inoculated with a single bacterial colony
and incubated at 15°C for 2-5 days. The cultures were then removed from incubation and
centrifuged at 1400 x g for 15 minutes. The supernatants from each culture were
removed and transferred to sterile 15 ml test tubes. The pellets were each resuspended in
1 ml methanol, which were then also transferred to sterile 15 ml test tubes. Methanol was
added to the tubes to produce a 60:40 ratio of methanol: solute.] (In this case, this
amounted to 5 ml supernatant:7.5 ml methanol and 1 ml pellet:2.5 ml methanol) This
ratio was selected after repeated trials revealed that a 60:40 ratio of methanol:marine
broth resulted in the greatest amount of salt precipitation. These solutions were left
undisturbed for 30 minutes to allow for precipitation of salts. The supernatants were
collected and the methanol was evaporated from each solution under vaccuum. The
remaining compounds were extracted into 2 ml ethanol and stored in sterile Eppendorf
tubes at -25°C. An extract of 5 ml marine broth served as a control with each set of
bacterial extracts. Extracts were tested within 12 hours of production to minimize loss of
activity.
Rhodamine B Accumulation Assay
The effect of bacterial extracts on the activity of the MXR protein was measured
using a rhodamine B accumulation assay in which mussel gills are exposed to rhodamine
B, a fluorescent dye that diffuses readily across the cell membrane and is then effluxed by
MXR protein. A relative increase in the level of intracellular fluorescence due to
increased accumulation of rhodamine B in the presence of the bacterial extract, (as
compared to a marine broth control), indicates that a substance in the bacterial extract
affects the action of the protein by serving as either a substrate for or an inhibitor of
MXR.
Gill tissue was dissected from the mussel and incubated in filtered sea water
(FSW) on ice. Forceps were used to remove excess mucus from the gills. Small
(-25mm*) pieces were cut from the dorsal edges of each gill. To test various extracts for
MXR substrate/inhibitor activity, the following solutions were set up in 10 mm petri
plates (Falcon 1008):
• 1 uM rhodamine + 5 ml FSW (rhodamine control)
• 1 uM rhodamine + 20 uM verapamil + 5 ml FSW (verapamil control)
• 1 uM rhodamine + 10 ul marine broth extract + 5 ml FSW (marine broth control)
• 1 uM rhodamine + 10 ul extract + 5 ml FSW (separate plate for each extract tested)
Verapamil, a known inhibitor of the MXR protein, served as a positive control for the
experiment; the marine broth extract served as a negative control.
5-6 pieces of gill tissue were placed in each petri plate. The petri plates were then
placed on an orbital shaker to shake at 15°C for one hour. The tissue pieces were
removed from the dye solutions and washed by swirling tissues in 30 ml FSW for 30
seconds. The pieces were then placed onto microscope slides and viewed at 10x
magnification under the fluorescence microscope. Tissues were viewed and
photographed with the Image-1 analysis system within 5 minutes of their removal from
the dye solutions.
Rhodamine B Efflux Assay
While the rhodamine accumulation assay tests for a modification in MXR activity
resulting from the presence of bacterial extract, it cannot differentiate between a
substance that increases the entry of rhodamine B into the cell, increasing dye
accumulation, and one that serves as a competitive substrate for the MXR protein, thereby
decreasing dye efflux. To distinguish between these scenarios, a rhodamine efflux assay
was employed on those samples that yielded positive results on the accumulation assay.
This assay measures the increase in extracelluar rhodamine fluorescence with
respect to time. Rhodamine B is added in excess to the cells and then removed from the
seawater. The accumulated dye is then pumped out of the cell by MXR, causing a
gradual (exponential) increase in rhodamine fluorescence of the surrounding media with
respect to time. If the bacterial extract contains a compound that is damaging to the cell
membrane, rhodamine fluorescence will increase very rapidly. Alternatively, a bacterial
extract containing a substrate of the MXR transporter that binds competitively to the
MXR protein or competes with the protein will cause the extracellular level of rhodamine
fluorescence to increase at a much slower rate, closer to that exhibited by verapamil, a
known inhibitor of MXR.
All four gill tissues were removed from a mussel, cut in half, and placed on ice in
filtered sea water. The eight resulting pieces of tissue were each placed in a separate 10
mm petri plate filled with FSW. This process was repeated with as many mussels as
solutions being tested, (including controls), yielding separate groups of gill tissue, each
containing one gill piece from each organism.
A solution of luM rhodamine in FSW was prepared using the 1 mM stock
solution. This new solution was stored in the dark. The groups of gill tissue created
above were removed from the FSW and transferred to 15mm petri plates each containing
25 ml of the luM rhodamine solution. The plates were placed on the orbital shaker to
shake in the dark for 1 hour at 15°C.
After removal from the orbital shaker, the rhodamine solution was aspirated from
the plates, and each plate was rinsed with 30 ml FSW for 30 seconds. The FSW was then
removed by aspiration and replaced by one of the following solutions:
• 30 ml FSW (rhodamine control)
30 ml FSW + 20 uM verapamil (verapamil control)
30 ml FSW + 60 ul marine broth extract (marine broth control)
• 30 ml FSW + 60 ul extract (one solution for each extract to be tested)
The plates were replaced on the orbital shaker in the dark at 15°. At 0, 10, 20, 30, 40, 50,
and 60 minutes after adding the solutions, 500 ul of each solution was transferred from its
respective petri plate to a 0.5 ml cuvette. This procedure was repeated in triplicate for
each solution at each time point. The relative fluorescence of each solution was measured
using a Perkin-Elmer fluorescence spectrophotometer (EX = 550 nm, EM = 580 nm),
which had been previously zeroed with 500 ul of FSW.
Isolation of Genomic DNA
The genomic DNA (gDNA) was extracted from clonal colonies of the various
strains of bacteria isolated from the mussel digestive gland using the RapidPrep Micro
Genomic DNA Isolation Kit <span style='background-color:#cfc' title='Pharmacia : Exact'>(Pharmacia</span> Biotech). This kit employs the use of individual
ready-made resin-packed columns to purify the DNA from 2 x 10° microbial cells
(approximately 1 ml of overnight culture). The DNA was eluted in 400 ul of elution
buffer, precipitated from this suspension with addition of 320 ul of 80% isopropanol, and
finally collected by centrifugation. The gDNA pellet isolated from each sample was re-
suspended in 20 ul of double deionized H2O and stored at 4°C.
RAPD PCR Analysis
The genomic differences between the various strains of bacteria isolated from the
digestive gland was investigated using Random Amplified Polymorphic DNA analysis
(RAPD). This technique detects genomic polymorphisms by using short oligonucleotide
primers (7 to 15 bp) of random sequence to initiate the polymerase chain reaction,
generating a fingerprint of amplification products specific to the strain in question.
"Ready to Go" RAPD Analysis Beads (Pharmacia Biotech) were used to
implement the RAPD analysis. These premanufactured beads contain all the reagents
needed for the reaction, with the exception of the primers, which were purchased
separately (Pharmacia Biotech). To prepare the samples for RAPD analysis, one bead
was placed in each sterile PCR tube with 5 ul of the primer to be used. 2 ul of genomic
DNA was added to each tube and brought up to a total volume of 25 ul with addition of
sterile sea water. Each trial included a control without primer. The samples were run
using a Perkin Elmer thermal cycler. When the reaction was complete, 3 ul sample buffer
was added to each PCR tube, and 10 ul of each sample was loaded onto a 2% agarose gel.
Gels were stained with ethidium bromide, viewed under ultraviolet light, and the band
patterns compared using the ratio of common bands to the total number of bands.
RESULTS
Isolation of Digestive Bacteria
The four most predominant bacterial strains were isolated from the digestive
gland homogenate of a mussel dissected March 28, 1998. These strains were designated
HPl, HP2, HP4, and HPS. While HP4 and HP5 bore phenotypic resemblance to each
other, HPI and HP2 were phenotypically distinct, both from each other and from HP4 and
HPS.
Phenotypic Description
HPI - large, white, opaque
HP2 - small, white w/orange center
HP4/HP5 - small, translucent
Rhodamine Accumulation Assay
The rhodamine B accumulation assay was initially performed on the supernatant
and pellet extracts of all four bacterial strains to screen for potential substrates (Figure 1).
The level of fluorescence in gill tissues exposed to each extract was compared with the
fluorescence in an extract of marine broth, which served as a negative control for the
experiment. Verapamil, a known inhibitor of the MXR protein, served as a positive
control. Gill tissues incubated with verapamil showed an average increase in rhodamine
fluorescence of 75% above the control marine broth. Neither the HPI supernatant nor the
pellet showed an increase in fluorescence when compared with the control; the
supernatant showed an average fluorescence 22% below the control, and the pellet 18%
below. The HP4 and HPS supernatant extracts also did not display substrate activity,
producing an average fluorescence 3% and 12% below the control, respectively. These
strains were not subject to further investigation. The HP4 and HPS pellets showed
approximately equal increases in fluorescence; both extracts produced average increases
of 10-15% above the control. The HP2 pellet demonstrated the most dramatic increase,
showing an average fluorescence 28% greater than the control, accompanied by a small
5% increase from the HP2 supernatant. Although the HP2, HP4, HPS pellet extracts and
the HP2 supernatant extract all demonstrated relative increases in rhodamine
accumulation, only the HP2 and HPS pellet extracts demonstrated statistically significant
increases in fluorescence at the p = 0.05 level.
Second Isolation of Digestive Bacteria
The four most predominant bacterial colonies were isolated from the digestive
tract homogenate of a second mussel dissected May 5, 1998 and labeled DNI, DN2,
DN4, and DN5.
Phenotypic Description and Relative Abundance
(f out of a total 183 colonies present on 50 ul plate)
DNI - large, white w/orange center (48 colonies)
DN2 - small, white w/orange center (96 colonies)
DN4 - small, white rimmed, clear in middle (12 colonies)
DN5- v. small bright orange (8 colonies)
RAPD Analysis
A RAPD PCR analysis was carried out on genomic DNA isolates from HP1, HP2,
HP4, HPS and the results visualized on a 2% agarose gel (Figure 2). Visual comparison
of the banding patterns produced by each primer revealed no similarity between any of
the four strains.
A second set of genomic DNA isolates from HP2, HP4, DN2, and DN4 were also
analyzed using RAPD PCR and visualized on an agarose gel (Figure 3). Comparing the
banding patterns created by primer 2 (the most accurate primer as recommended by the
manufacturer) revealed a strong similarity between lanes 9 and 10, which contained DNA
from HP2 and DN2, respectively.
The gel was repeated using primer 2 alone with only HP2 and DN2 samples
(Figure 4). HP2 and DN2 exhibit identical banding patterns, as illustrated by lanes 4 and
5, respectively. Ratio of similarity between lanes 4 and 5 = 3 bands in common/3 total
bands =1.
Efflux Assays
Efflux assays were initially conducted on both pellet and supernatant extracts of
the HP2, DN2, HP4, and DN4 bacterial strains (Figures 5, 6, 7, 8). When the rates of
efflux of these extracts were compared to those exhibited by rhodamine and verapamil,
only the extracts of the DN2 and HP2 pellets consistently showed efflux rates below that
of rhodamine. These two extracts were subject to further individual testing, upon which
they continued to exhibit efflux rates between those of rhodamine and verapamil.
(Figures 9, 10) In all cases, the marine broth control exhibited an efflux rate
approximately equal to or higher than that of rhodamine (Data not shown).
DISCUSSION
The original aim of this study was to identify potential bacterial substrates for the
MXR protein in Mytilus californianus. To achieve this goal, several extracts of bacterial
strains isolated from the mussel digestive gland were screened for their level of
interaction with the MXR protein using both the rhodamine B accumulation and efflux
assays. The HP2 and DN2 pellet extracts were the only extracts to show potential
substrate activity, both in the accumulation assay (for HP2) and the efflux assay (for
both), indicating that these two bacteria contain products that serve either as substrates or
inhibitors of the MXR protein. The close similarity in the banding pattern between HP2
and DN2 observed on the agarose gel (Figure 4), as well as the similar morphology
between the plated colonies, indicate that HP2 and DN2 are in fact identical bacterial
strains. This conclusion is further supported by the similarities in their respective rates of
efflux with respect to rhodamine and verapamil controls (Figures 9, 10), a result that
would be expected from identical strains.
The six other tested strains did not show significant substrate activity, indicating
that the compounds in HP2 and DN2 that are substrates of MXR are not general products
of bacterial metabolism. The results with these strains then serve as controls for the
assays used in the study. For example, the HPI strain provided a negative bacterial
control; subject to the same extraction procedure as the other strains in question, HP1
consistently showed no substrate activity. The HP4 pellet extract provided a control for
assay specificity; while the HP4 pellet showed consistent activity when tested with the
accumulation assay, it repeatedly yielded negative results when tested with the efflux
assay. This result indicates that the HP4 pellet contains a substance that increases
rhodamine accumulation into tissues but does not decrease its rate of efflux into the
surrounding media. A possible explanation is that the substance in question interacts
with the gill cell wall without specifically binding MXR, thereby facilitating rhodamine
diffusion into the cell without affecting its efflux.
The prevalence of substrate activity in the HP2 and DN2 pellets despite the
absence of activity in the supernatants raises interesting questions as to the nature of the
substance causing the activity. There are a number of possibilities - the substance in
question may be a component of the bacterial cell wall and may thus end up solely in the
pellet fraction, or the substance could be one that is secreted by the bacteria and is soluble
in both the pellet and supernatant fractions, but simply more concentrated in the pellet.
Another, more probable explanation is that the bacteria may sequester the substance in
the cell. While the bacteria may secrete trace amounts of the substance into the
supernatant, the majority of the substance may remain concentrated inside the bacteria
itself. When the bacteria are lysed upon homogenization, centrifugation or exposure to
methanol, the lysed bacteria as well as their concentrated internal products would end up
in the pellet fraction. The extraction processes may be analogous to the digestive process
in vivo, for mussels synthesize a variety of digestive carbohydrases, lipases, esterases and
phosphatases that have the ability to lyse and assimilate ingested bacteria (Gosling 1992).
Mussels are therefore likely to encounter even those toxic products that are entirely
contained within an ingested bacterium.
The fact that the HP2 and DN2 bacteria were isolated from two distinct mussels
collected seven weeks apart indicates that the strain is persistent in the mussel digestive
gland, either as a regularly ingested component of the diet or as a permanent symbiont.
Related studies have isolated and identified a variety of potentially pathogenic symbiotic
bacteria from the digestive gland, gills, primary ducts, kidney, and gill ciliates of the
Spanish mussel <span style='background-color:#cfc' title='Mytilus galloprovincialis : Exact'>Mytilus galloprovincialis</span> (Villabla, et. al. 1997). Regardless of the
source of the bacteria, however, the production of potential MXR substrates by a
persistent bacterial strain indicates that a natural function of MXR may be to enable the
mussel to reap the nutritive benefits from bacteria without being harmed by their
potentially toxic byproducts. Studies on the function of MXR in the sediment-dwelling
worm <span style='background-color:#cfc' title='Urechis caupo : Exact'>Urechis caupo</span> have reached similar conclusions; marine bacteria isolated from the
worm’s intestine were found to produce moderately hydrophobic compounds that were
substrates of the P-glycoprotein (Toomey, et. al. 1996). Analogously, the presence of the
HP2/DN2 bacteria in the digestive gland of <span style='background-color:#ffcc00' title='Mytilus californianus : PartialExact'>Mytilus californianus may</span> be part of the
reason for the concentration of MXR protein that is found there.
FUTURE DIRECTIONS
This study will continue during the next three months. One possible future
research direction would entail a chemical characterization of the identified bacterial
substrate for MXR transport protein. Öther possibilities include investigating whether
exposure to increased concentrations of the substrate induce greater levels of MXR
expression in mussels, and whether similar substrates can be isolated from the human
digestive tract.
ACKNOWLEDGMENTS
This work would have never materialized had it not been for the insight and guidance of
<span style='background-color:#cfc' title='Melissa : Exact'>Melissa</span> Kaufman (for the help with bugs), Nancy Eufemia (for the help with mussels and
life), Chris Patton (for the technical help) and David Epel (for taking me into his lab).
Thank you all.
FIGURE LEGENDS
Fig. 1. Rhodamine Accumulation Data: Fluorescence of gill tissues exposed to pellet
and supernatant extracts of HPl, HP2, HP4, and HPS bacterial strains as compared with a
marine broth control extract. Figure includes data pooled from three separate rhodamine
accumulation assays. All fluorescence levels are expressed as a percentage of the 100%
marine broth control. Error bars represent the standard error between the three
experiments. (**) denotes statistical significance at the p-0.05 level.
Fig. 2. Agarose Gel: RAPD PCR analysis of genomic DNA isolated from HP1, HP2,
HP4, and HPS bacterial strains. No two lanes produced significantly similar banding
patterns.
Fig. 3. Agarose Gel: RAPD PCR analysis of genomic DNA isolated from HP2, HP4,
DN2, and DN4 bacterial strains. Note similarity between lanes 5 and 6, which contain
strains HP2 and DN2, respectively.
Fig. 4. Agarose Gel: RAPD PCR analysis of genomic DNA isolated from DN2 and HP2
bacterial strains. Note similarity between lanes 4 and 5, which contain strains HP2 and
DN2, respectively.
Fig. 5. Rhodamine Efflux Assay: HP2 pellet and supernatant extracts. Fluorescence
measured on a Perkin-Elmer fluorometer zeroed with 500 ul FSW. Included are
rhodamine efflux curves from mussel gills incubated with (rhodamine only),
(+verapamil), (-HP2 pellet), and (-HP2 supernatant). Each time point was measured in
triplicate. Error bars represent standard errors from the three fluorometer readings at each
time point. HP2 pellet exhibited a slower rate of efflux than the rhodamine control; HP2
supernatant did not.
Fig. 6. Rhodamine Efflux Assay: HP4 pellet and supernatant extracts. Fluorescence
measured on a Perkin-Elmer fluorometer zeroed with 500 ul FSW. Included are
rhodamine efflux curves from mussel gills incubated with (rhodamine only),
(+verapamil), (-HP4 pellet), and (-HP4 supernatant). Each time point was measured in
triplicate. Error bars represent standard errors from the three fluorometer readings at each
time point. Neither HP4 pellet nor HP4 supernatant exhibited a slower rate of efflux than
the rhodamine control.
Fig. 7. Rhodamine Efflux Assay: DN2 pellet and supernatant extracts. Fluorescence
measured on a Perkin-Elmer fluorometer zeroed with 500 ul FSW. Included are
rhodamine efflux curves from mussel gills incubated with (rhodamine only),
(+verapamil), (-DN2 pellet), and (-DN2 supernatant). Each time point was measured in
triplicate. Error bars represent standard errors from the three fluorometer readings at each
time point. DN2 pellet exhibited a slower rate of efflux than the rhodamine control;
DN2 supernatant did not.
Fig. 8. Rhodamine Efflux Assay: DN4 pellet and supernatant extracts. Fluorescence
measured on a Perkin-Elmer fluorometer zeroed with 500 ul FSW. Included are
rhodamine efflux curves from mussel gills incubated with (rhodamine only),
(+verapamil), (-DN4 pellet), and (+DN4 supernatant). Each time point was measured in
triplicate. Error bars represent standard errors from the three fluorometer readings at each
time point. Neither DN4 pellet nor DN4 supernatant exhibited a slower rate of efflux
than the rhodamine control.
Fig. 9. Rhodamine Efflux Assay: DN2 pellet. Fluorescence measured on a Perkin-
Elmer fluorometer zeroed with 500 ul FSW. Included are rhodamine efflux curves from
mussel gills incubated with (rhodamine only), (+verapamil), and (+DN2 pellet). Each
time point was measured in triplicate. Error bars represent standard errors from the three
readings at each time point. DN2 pellet exhibited a slower rate of efflux than the
rhodamine control
Fig. 10. Rhodamine Efflux Assay: HP2 pellet. Fluorescence measured on a Perkin-
Elmer fluorometer zeroed with 500 ul FSW. Included are rhodamine efflux curves from
mussel gills incubated with (rhodamine only), (+verapamil), and (+HP2 pellet). Each
time point was measured in triplicate. Error bars represent standard errors from the three
fluorometer readings at each time point. HP2 pellet exhibited a slower rate of efflux than
the rhodamine control.
LITERATURE CITED
1. Cornwall, R., Toomey, B.H., Bard, S., Bacon, C., Jarman, W.M., and Epel, D. 1995.
Characterization of multixenobiotic/multidrug transport in the gills of the mussel
Mytilus californianus and identification of environmental substrates. Aquatic
Toxicology. 31:277-296.
2. Epel, David. 1998. Use of multidrug transporters as first lines of defense against
toxins in aquatic organisms. Comparative Biochemistry and Physiology. Part A. (in
press).
3. Endicott, J.A. and V. Ling. 1989. The biochemistry of p-glycoprotein-mediated
multidrug resistance. Annual Review of Biochemistry. 58:137-171.
4. Eufemia, Nancy A., and Epel, David. 1998. The multixenobiotic defense mechanism
in mussels is induced by substrates and non-substrates: Implications for a general
stress response. Marine Environmental Research. (in press).
5. Gosling, Elizabeth, ed. 1992. Physiology of Production. pp. 172-174 in The Mussel
Mytilus: Ecology, Physiology, Genetics and Culture. New York.
6. Thiebaut, F., Tsuruo, T., Hamada, H., Gottesman, M.M., Pastan, I., and Willingham,
M. 1987. Cellular localization of the multidrug-resistance gene product P-
glycoprotein in normal human tissues. Proceedings of the National Academy of
Sciences. 84:7735-7738.
7. Toomey, Barbara H., and Epel, David. 1993. Multixenobiotic Resistance in Urechis
caupo embryos: Protection from Environmental Toxins. Biological Bulletin.
185:355-364.
Toomey, B.H., Kaufman, M. and Epel, D. 1996. Marine bacteria produce compounds
that modulate multixenobiotic transport activity in Urechis caupo embryos. Marine.
Environmental Research. 42:393-397.
9. Villalba, A., Mourelle, S.G., Carballal, M.J., Lopez, C. 1997. Symbionts and
diseases of farmed mussels Mytilus galloprovincialis throughout the culture process
in the Rias of Galicia (NW Spain). Diseases Of Aquatic Organisms. 31(2):127-139.
L
8
tatakataka-
2
uselon jonuo%
99
9
Figure 2
234 5 67
Lane 1: HPI (primer 1
Lane 2: HP2 (primer
Lane 3: HPA (primer 1
Lane 4: HP5 (primer
Lane 5: HPI (primer 2'
Lane 6: HP2 (primer 2)
8 9 10 11 12
Lane 7: HPA (primer 2
Lane 8: HP5 (primer 2)
Lane 9: HPI (primer 3)
Lane 10: HP2 (primer 3)
Lane 11: HPA (primer 3
Lane 12: HP5 (primer 3
Figure 3

6
7 8 9 10 11 12 13 14 15 16
2
Lane 1: HP2 (primer 1
Lane 9: HP2 (primer 3)
Lane 2: DN2 (primer
Lane 10: DN2 (primer 3
Lane 3: HPA (primer 1
Lane 11: HP4 (primer 3'
Lane 4: DNA (primer 1
Lane 12: DNA (primer 3
Lane 13: HP2 (primer 4
Lane 5: HP2 (primer 2
Lane 14: DN2 (primer 4
Lane 6: DN2 (primer 2
Lone 15: HPA (primer 4)
Lane 7: HP4 (primer 21
Lane 8: DNA (primer 2
Lane 16: HPA (primer 4
Figure 4
4
Lane 1: XHIII MW marker
Lane 2: HP2 (no primer
Lane 3: DN2 (no primer)
Lane 4: HP2 (primer 2
Lane 5: DN2 (primer 2)
60
50
40
5 30
8 20
10 9
60
50
40
30 -
6 20 -
10
10
10
20
20
Figure 51
30
time (min)
Figure 61
30
time (min)
40

- HP2 sup
- rhodamine
— verapamil
HP2 pellet
﻿50

- rhodamine
verapamil
- HP4 pellet
- HP4 sup
40
﻿50
﻿60
﻿60
﻿50
40
30
6 20
10
60
﻿50
40
3 30
20
10
10
10
20
IFigure 71
A
—
-- rhodamine
verapamil
- DN2 pellet
DN2 sup
﻿50
40
time (min)
Figure 81
kt-

rhodamine
verapamil
- DN4 pellet
- DN4 sup
40
30
﻿50
time (min)
﻿60
60
50
40
30
20
﻿60
﻿50
8 40
5 30
20
10
10
10
20
20
Figure 91

30
time (min)
[Figure 101
30
time (min)
40
40
rhodamine
verapamil
DN2 pellet
﻿50

rhodamine
verapamil
HP2 pellet
﻿50
﻿60
﻿60
</pre></body></html>