<html><body><pre style='margin:2rem auto;width:fit-content'>Expression of GFLNI Sodium Channel mRNA at Embryonic Stages
of the Squid <span style='background-color:#cfc' title='Loligo opalescens : Exact'>Loligo opalescens</span>
ABSTRACT
Messenger RNA encoding GFLNI, the sodium channel proposed to exist in the
giant axon of the squid, was detected in embryos of Loligo opalescens. PCR analysis
and RNAse Protection Assay both confirm the initiation of GFLNI mRNA
expression at stage 25 of embryonic development. The level of GFLNI mRNA
expression was found to increase rapidly at stage 27, which immediately precedes
giant axon function. Expression levels progressively increase during late
developmental stages (28-30) which coincide with increased speed of the escape
response.
INTRODUCTION
The squid giant axon has been a popular model for studying the role of
sodium channels in electrical conduction. Extensive studies of the giant axon have
elucidated biophysical properties of the sodium channel, but molecular research
lags far behind. Two types of sodium channels in the squid have only recently been
sequenced and cloned. One type of sodium channel has been identified in the optic
nerve (Sato & Matsumoto, 1992), while another, GFLNI, has been found in the
stellate ganglion (Rosenthal & Gilly, 1993).
GFLNI represents a putative sodium channel that is expressed in the
neurons which form the giant axon. Evidence from RNAse Protection Assay and
Northern blot hybridization confirm that giant fiber lobe neurons express mRNA
which corresponds to GFLNI. Because voltage clamp experiments reveal that one
major sodium channel type exists in the giant axon, GFLNI has been implicated as
the sodium channel type which is responsible for electrical conduction in the squid
giant axon. However, GFLNI may also be present in other neurons in the stellate
ganglion as well (Rosenthal, 1993).
Much work has been done with adult squids regarding sodium channel
characterization. However, in squid embryos, interest in the giant axon system has
only recently been spurred. Previous embryonic work includes the anatomical
development of the stellate ganglion and the ontogeny of the escape response (Gilly
et al, 1991). Also, structural organization of the first order giant axon system in
squid embryos has been documented (Martin, 1965). This paper focuses on the
expression of GFLNI sodium channel mRNA during embryonic development of
Loligo opalescens using Polymerase Chain Reaction and RNAse Protection Assay
MATERIALS AND METHODS
Collection and Staging of Loligo opalescens embryos
Squid egg cases were collected from Monterey Bay and stored under running
sea water at 10°C. Two to three egg cases were dissected every day until hatching.
During dissection, the outer membrane of the egg case was sliced open and
separated from the egg mass. Dissecting forceps were used to remove the embryos
from their inner egg membranes and periviteline fluid. The external yolk sac was
detached from each embryo under a dissecting microscope and discarded. Loligo
opalescens embryos from the following stages were collected: 23, 25, 26, 27, 28, 29,
29+, 29++, and 30 (hatchlings). The embryo was staged according to the
characteristics described by Segawa et al (1988) and Gilly et al (1991). Embryos were
then washed with filtered sea water, frozen in liquid nitrogen, and stored at -70°C
until homogenization.
RNA Isolation
For each stage of development, approximately 75-100 embryos were
homogenized in a Dounce Homogenizer containing 2 ml guanidine thiocyanate
(50 g guanidine thyocyanate, 0.5 g n-lauryl sarcosine, 2.5 ml of 1M sodium citrate;
pH 7.0) and 0.014 ml beta-mercaptoethanol (Cooperman et al , 1987). Adult stellate
ganglion and adult gill tissue were homogenized and subjected to the same
treatment as the embryos. 1.5 ml of the homogenate was gently layered over 550 ul
cesium chloride (5.7 M) and centrifuged in a Beckmann RP 55-S Swinging Bucket
Rotor for 3 hours at 55,000 rpm and 22'C. The RNA pellet was carefully separated
and resuspended in 100 ul TE-SDS solution containing 0.1% SDS. The RNA was
subjected to phenol/chloroform extraction and ethanol precipitation. RNA density
was quantified by spectrophotometry and the extent of RNA degradation was
determined by running 2 ug of RNA on a 1% agarose gel containing ethidium
bromide.
Detection of GFLNI by Polymerase Chain Reaction
A sample of RNA (3 ug) from each developmental stage was used as a
template for cDNA synthesis using ÜSB Reverse Transcriptase. The standard
conditions for USB RT were slightly modified. Äfter an ethanol wash, the RNA
was resuspended in 10.75 ul sterile water and set in a 65°C water bath for 3 minutes.
Each reaction component was added in a scaled-down volume (0.4X) from the ÜSB
protocol: 5.0 ul RT buffer, 5.0 ml dNTP (10 mM total nucleotide pool), 2.5 ul oligo
dT primer (10 uM), 1.0 ul RNAsin, 1.25 ul reverse transcriptase (MMLV). The
reaction was allowed to incubate for 1 hour at 37°C. The temperature was then
raised to 95°C for 5 minutes. The CDNA was subjected to phenol/chloroform
extraction and ethanol precipitation.
Polymerase Chain Reaction detected the presence of GFLNI among the pool
of various CDNA. The primers SNC2 and SNC3 (Rosenthal & Gilly, 1993) were
used to amplify the segment of GFLNI from nucleotide 3720 to 4640. The predicted
size of this fragment is 921 base pairs. For each microliter of CDNA suspended in
NaAc/EtOH solution, the PCR reaction mixture consisted of 2.5 ul Taq buffer (10X),
2.5 ul dNTP (8mM total nucleotide pool), 1.25 ul SNC 2 (sodium channel primer)
1.25 ul SNC 3 (sodium channel primer), 17 ul sterile water, 0.03 ul Tag DNA
Polymerase, and 1 drop of mineral oil.
PGFLNI served as the positive control. To determine the sensitivity of PCR,
serial dilutions of this plasmid were subjected to PCR, and the amplified products
were separated on a 1% agarose gel.
A Model 480 Thermocycler (Perkin Elmer) was used to run the PCR. The
PCR process consisted of 35 cycles. In all cycles, denaturation occurred at 94°C and
elongation at 72°C. In the first five cycles, annealing was at 60'C and followed a 2
minute temperature ramp. In the remaining cycles, annealing occurred at 55°C
and the ramp was omitted. Ötherwise, all other target temperatures were attained
immediately. Each of these steps (denaturation, annealing, and elongation) in all
cycles continued for 1 minute. A 1% agarose gel was run with 8.0 ul of the
amplified CDNA from each stage of development.
Detection of GFLNI by RNAse Protection Assay
An RNAse Protection Assay was performed with radio-labelled sodium
channel probes. Probes were synthesized with the following components: 2 ul
buffer (5X), 1 ul DTT (0.1 M), 0.5 ul RNAsin, 2 ul rATP/TGTP/TCTP ribonucleoside
mix (7.5 mM total pool), 1.2 rUTP (100 uM), 2.5 ul P32-rUTP (25 uCi, 800 Ci/mMol),
0.5 ul linearized plasmid template (50 ng pNZ5- cut with Ndel; Rosenthal & Gilly,
1993), and 0.5 ul T7 RNA polymerase (30 units/ ul). The undigested probe
incorporates two regions: 237 bases from the 3' end of GFLNI from the Ndel site in
the coding region through the untranslated region, and 30 bases of the vector to the
T7 promoter site (Rosenthael & Gilly, 1993). The reaction mixture was incubated
for 1 hour at 37°C. Then 1 ul of RQ-1 DNAse (Promega) was added and incubation
continued for 30 more minutes at 37°C. Reaction products were subjected to
phenol/chloroform extraction and ethanol precipitation. Yeast tRNA (10 ug) was
used as carrier.
1005 cpm of probe were hybridized to 10 ug RNA from each embryonic stage.
The assay solution included 3 ul hybridization buffer (4 M NaCl, 400 mM PIPES,
and 10 mM EDTA) and 24 ul deionized formamide. The hybridization mixtures
were heated to 85’C for 5 minutes and then incubated at 48°C overnight.
Digestion of single-stranded RNA was performed by adding 300 ul of RNAse
digestion solution to each sample and incubating for one hour at room
temperature. The RNAse digestion solution consisted of 300 mM NaCl, 5mM
EDTA, 10 mM Tris, 10 ug/ml RNAse A, and 0.4 ug/ml RNAse TI. To terminate
the digestion, 20 ul of 10% SDS and 2 ul proteinase K (25 mg/ml) were added to the
samples, which were then incubated further for 15 minutes at 37°C. The reaction
products were phenol extracted and ethanol precipitated with 25 ug tRNA as
carrier. Samples representing each stage of development were run on a 6%
acrylamide sequencing gel. Also, 1000 cpm of undigested probe for GFLNI mRNA
was run in the gel after loading. Kodak XAR-5 film with one intensifying screen
was exposed to the gel for three days at -70°C.
RESULTS
Analysis of Sensitivity of PCR for Amplifying GFLNI CDNA
The PCR primers, SNC2 and SNC3, bind to very specific regions of GFLNI.
SNC2 corrresponds to nucleotides 3720 to 3754 of GFLNI, whereas SNC3 recognizes
nucleotides 4624 to 4640. In Figure 1, the level of sensitivity of the PCR protocol
used in this experiment is assessed by the detection of pGFLNI in serial dilutions.
The most dilute sample which still displays a definite band corresponds to lane 9,
and this sample was calculated to contain 1110 copies of Na channel cDNA. Thus,
the same dilution of plasmid as that in lane 9 served as the positive control for
testing the presence of GFLNI mRNA at different embryonic stages.
Detection of GFLNI mRNA in Loligo opalescens embryos by PCR
PCR is specific enough to detect the targeted segment of GFLNI from a pool
of various cDNA coding for different proteins. In Figure 2, lane 1 contains the
amplified region of pGFLNI and provides a positive control. Lane 2, the negative
control, did not contain cDNA template in the PCR reaction, and no band is
present. This negative control is important because of the risk of contamination
and the sensitivity of PCR as proven above. Lane 3 represents amplified cDNA
made from RNA of adult squid stellate ganglion. The calculated size of the
amplified segment of GFLNI is 921 base pairs, which corresponds to the size of the
bands in lanes 1 and 3 measured by a 1 kilobase DNA ladder. Lane 4 contains
CDNA amplified from adult gill tissue and shows a very faint band at the position
of the gel expected for the GFLNI segment. Lane 5 contains no detectable band,
suggesting that GFLNI mRNA is not present at stage 23 of the squid embryo.
The presence/absence of GFLNI mRNA in each embryonic stage was
similarly tested, and the results are conveyed in Figure 3. Sodium channel mRNA
initially appears somewhere between embryonic stage 23 and 25, and expression
continues from this point until hatching at stage 30.
Detection of GFLNI mRNA in Loligo opalescens embryos by RNAse Protection
Assay
Because PCR is very sensitive and amplification is exponential rather than
linear, PCR data is difficult to analyze quantitatively as a means of determing levels
of mRNA expression. RNAse Protection Assay relies on hybridization between the
probe and the mRNA. Because this molecular binding process occurs on a one-to¬
one basis, the intensities of the radio-labelled bands give more reliable data
regarding the level of Na channel expression.
Results from RNAse Protection Assay performed with the probe described
above and mRNA isolated at a series of developmental stages are presented in
Figure 4. As demonstrated by the presence of fully protected bands, GFLNI mRNA
is first expressed at stage 25, but at relatively low amounts. Expression steps up
sharply at stage 27 and then increases progressively, attaining a relatively high level
at stage 30. Levels of sodium channel mRNA in adult stellate ganglion tissue are
also very high, as previously demonstrated (Rosenthal & Gilly, 1993).
DISCUSSION
This study of the expression of GFLNI mRNA in squid embryos holds many
implications about the development of the giant axon system. Most specifically,
correlates between the levels of GFLNI mRNA expression and
anatomical/behavioral development can now be proposed.
At embryonic stage 25, RNAse Protection Assay detected a relatively low
level of GFLNI mRNA. The appearance of the sodium channel at this stage of
development coincides with the beginning of mantle contraction in response to
mantle contraction of the tentacles . This same study showed that morphologically
distinct giant axons do not appear until stage 26 (Gilly et al , 1991). These findings
suggest that GFLNI mRNA detected at stage 25 may be expressed by the non-giant
motoneurons , or that a lag time could exist when GFLNI mRNA is expressed in
the developing neurons destined to form the giant axon prior to the actual fusion
of these neurons. Fusion presumbly must occur before the giant axons can be
clearly identified morphologically in the stellar nerves.
Stage 27 embryos display a rapid increase in the level of sodium channel
mRNA. The increased production of GFLNI mRNA at stage 27 immediately
precedes giant axon function which is first detectable at stage 28 (Gilly et al, 1991).
Again, lag time between GFLNI mRNA expression and GFLNI Na channel protein
translation may be relevant to the one-stage mismatch between molecular and
anatomical /functional observations. As the embryo passes beyond stage 28 and
approaches the hatching, parallel increases occur in the level of GFLNI expression,
rapidity of the escape response, and size of the giant axons (Gilly et al, 1991).
Speculation must surround the correlations drawn between molecular
expression of GFLNI mRNA and anatomical/behavioral development relating to
the giant axon, primarily because the relationship between GFLNI mRNA levels
and the strength of the escape jet is undoubtedly complex. For example, because
GFLNI mRNA may be translated more than once, the level of sodium channel
mRNA detected by RNAse Protection Assay may not directly indicate the number
of functional sodium channels in the giant axon. Thus, reduced levels of GFLNI
mRNA expression, such as at stage 29, does not necessarily reflect lower numbers of
sodium channel proteins. Moreover, alternative gene splicing can lead to different
types of sodium channel mRNA that might cross-react with PCR primers and
RNAse Protection Assay probes.
Many of these problems can be addressed by further studies based on the
work described here. Dissection of embryonic stellage ganglion tissue for RNA
isolation can limit cross-reaction between the probe and other sodium channel
types found in neurons from the brain. The spatial distribution of GFLNI mRNA
expression can also be determined by in situ hybridization. Finally, analysis of
GFLNI protein itself, such as by western blotting, will clarify the temporal
relationships between GFLNI mRNA transcription and GFLNI protein translation.
These experiments represent just the initial studies that will elucidate the
development of one of the most popular systems, the squid giant axon system.
Acknowledgements
My undying thanks extend to William Gilly who has given me the support to
pursue my project and to focus my efforts very well. Also, I owe much gratitude to
Josh Rosenthal who mentored me everyday and sacrificed much of his time for my
project. Also, many thanks to Marie Perri and Simone Alin, who both gave me
companionship in lab, working space in lab, and good advice. I would like to thank
Alberte’s lab for use of the ultracentrifuge, Thompson’s lab for use of the camera¬
microscope, Weissman's lab for use of the PCR machine, and Power's lab for use of
the speed vac, PCR machine, and oven. I also thank Chris Patton for preparing my
figures. Lastly, I am grateful to Molly Cummings and George Matsumoto for
collecting my squid egg cases for me.
Literature Cited
Cooperman, S. S., S. A. Grubman, R. L. Barchi, R. H. Goodman, and G. Mandel.
1987. Modulation of sodium-channel mRNA levels in rat skeletal muscle.
Proc. Natl. Acad. Sci. USA. 84: 8721-8725.
Gilly, W. F., Bruce Hopkins, and G. O. Mackie. 1991. Development of Giant Motor
Axons and Neural Control of Escape Responses in Squid Embryos and
Hatchlings. Biol. Bull. 180: 209-220.
Martin, R. 1965. On the structure and embryonic development of the giant fibre
system of the squid <span style='background-color:#cfc' title='Loligo vulgaris : Exact'>Loligo vulgaris.</span> Z. Zellforsch . 67: 77-85.
Rosenthal, J. J. C. and W. F. Gilly. 1993. Amino acid sequence of a putative sodium
channel expressed in the giant axon of the squid, Loligo opalescens
Proceedings of the National Academy of Sciences . In press.
Sato, C. and G. Matsumoto. 1992. Primary structure of the squid sodium channel
deduced from the complementary DNA sequence. Bioch. Biophys. Res.
Comm. 186: 61-68.
Segawa, S., W. T. Yang, H. J. Marthy, and R. T. Hanlon. 1988. Illustrated
embryonic stages of the eastern atlantic squid <span style='background-color:#cfc' title='Loligo forbesi : Exact'>Loligo forbesi.</span> Veliger 30:
230-243.
Figure 1. Sensitivity of PCR for detecting pGFLNI. Each lane contains a serial
dilution of pGFLNI which was amplified by PCR. Lanes 1 and 8 each consist of a
one kilobase DNA ladder, while lanes 2-7 and 9-13 constitute the serial dilutions of
PGFLNI. Lane 2 was loaded with the most concentrated sample of pGFLNI and
corresponds to 1x102-2 ug or 1.11x109 molecules of pGFLNI. Each subsequent lane
contains a ten-fold dilution of pGFLNI. Lane 14 is the negative control. The band
measures 921 kb.
Figure 2. Specificity of PCR for detecting GFLNI CDNA. PCR was performed with
GFLNI CDNA made from RNA isolated from different squid tissues/stages. Lane 1,
the positive control, contains 1x100-8 ug of pGFLNI. The negative control was run
in lane 2 and contains no DNA template in the PCR mix. Lanes 3-5 each represent
3 ug of RNA extracted from adult stellate ganglion (SG), adult gill tissue, and stage
23 embryos (23), respectively. The molecular size of the amplified GFLNI segment
is 921 kb.
Figure 3. Detection of GFLNI CDNA by PCR from staged squid embryos. The
numbered lanes indicate the embryonic stage from which RNA was extracted. Each
lane represents 3 ug of RNA that was made into cDNA and amplified in PCR.
RNA from adult stellate ganglion (SG) and pGFLNI (PL) served as positive controls
for detection of GFLNI. The molecular weight of the amplified region of GFLNI is
921 kb.
Figure 4. RNAse Protection Assay with GFLNI-specific probe. Hybridization
occurred between 10 ug RNA per embryonic stage and 3.33x10°4 cpm of probe.
Undigested probe (P) was loaded at 1000 cpm and measured 267 bp. The protected
sodium channel sequence measured 237 bp in length.

921
igure 2
60-0
zoO
921
Figure 3
ooobo
NONNNNQOOOZO.
267
237-
Figure
P 23 26 26 27 28 29 29t 29630 s0

â..
Oueu
ere de à Ces  my pr
roe tat tey de Soth
Aeet. Te to opy K ogen
U
ochue o De lst 1 poes
te boto opy Vos plotgoded
ge put da f d
G
Goun
de
1
De Ordivol pictues
aea
Dorkvolt
vouge orpopers
Se Suso
nd
Hew
g
</pre></body></html>