<html><body><pre style='margin:2rem auto;width:fit-content'>FCP Gene Family of <span style='background-color:#cfc' title='Macrocystis pyrifera : Exact'>Macrocystis
pyrifera</span> (Giant Kelp):
Development of Gene Specific
Probes
Marina L. Tostado
Stephanie K. Clendennen
Powers Lab
June 5, 1992
ABSTRACT
Eight genes in Macrocystis pyrifera code for the FCP
protein. Each gene has similar coding regions but very different
noncoding regions (dissimilar sequences and sizes). From this
gene family three genes, 2A, 4B, and 4C, were studied. Gene
specific primers were made and challenged with positive results.
These primers were used in PCR reactions to make gene specific
probes. The gene specificities of the probes were tested using
Southern Blot Analysis concluding with supporting results. A
light regulation experiment using RNA obtained from incubated
kelp blades which were analyzed on a Northern Blot using the
radioactive gene specific probes was conducted but its results
were ambiguous.
INTRODUCTION
Macrocystis pyrifera contains about eight distinct genes
which code for the FCP protein, similar to the CAB protein of
higher plants and green algae for both are part of the light
harvesting center connected to Photosystem II complex of photo¬
synthesis (Grossman et al, 1990). FCP proteins function as a
satellite dish which focuses the suns light to reaction centers.
These eight different genes make up the FCP gene family of giant
kelp.
In this gene family, every gene is made up of a) about
an 858 homologus coding region of about 700 bp and b) a distinct
noncoding region of different sizes and of different sequences.
Of the family 3 specific genes were studied: 2A, 4B, and 40.
2A has a noncoding region length of about 1 kb while 4B and
4C have about a 500 bp noncoding region.
Why does <span style='background-color:grey' title='Macrocystis pyrifera : Fuzzy'>Macrosystis pyrifera</span> contain many different genes
to code for the same protein? In the <span style='background-color:#cfc' title='Drosophila : Exact'>Drosophila</span> actin gene family
(which like the FCP gene family has similar coding regions with
noncoding regions of different sizes and sequences), the actin
genes were found to be developmentally regulated and tissue
specific (Fyrberg et al, 1983). In tomatoes, the CAB genes were
found to be organ/tissue specific and light regulated (Kellmann
et al, 1990). To answer this basic question gene specific probes
were made. By creating such probes the expression of these
different genes could be followed through different developmental
stages and stress responses by measuring the amount of its mRNA,
thus elucidating the presence of the various genes.
In order to make gene specific probes, gene specific primers
were designed and made. Using Polymerase Chain Reactions (PCR),
we amplified the noncoding regions to use as probes on Southern
Blot Analysis to test the probe's specificity.
A light regulation experiment in which giant kelp blades
were incubated at low level light, so as to induce the production
of FCP proteins, was then set up and samples were obtained
periodically. Form these samples RNA was exptracted and a
Northern Blot was performed using radioactive labeled probes.
Our results were encouraging.
MATERIALS AND METHODS
Designing Primers:
Using the known sequence of the noncoding region of 2A,
4B, and 40, a 20bp sequence for each gene containing a unique
sequence (with little or no repeats) and one rich in GC pairs
was found. The GC pairs are preferred over AT pairs because
of their stronger bonds (three instead of two) which are more
stable and thus permit for higher temperatures in PCR reactions.
A T74 primer was also made for the plasmid vector (see fig.
1).
PCR Reactions:
For the Polymerase Chain Reactions the doing region, 2A,
4B, and 40 gDNA were used as templates. dNTPs, TAQ enzyme and
buffer, T74 primer, along with the respective primers were added.
The reactions were carried out at 90°C for denaturing, 55°
to anneal, and 72°C to extend for 25-30 cycles.
Agarose Gel:
63 agarose gels were used for the DNA. 23 EtBr was added
to the agarose to visualize the DNA when irradiated. Ikb ladders
were used as references.
Southern Blots:
DNA Extraction:
liquid Nitrogen was used to grind the tissues.
The DNA was then chloroform extracted. The
polysaccherides were precipitated in 203 EtOH.
The DNA was precipitated in EtOH, resuspended
in water and quantitated in the spectrometer
DNA Digest:
Each gene (coding, 2A, 4B, and 40) was digested
with Sac I, Pst I, and Hind III overnight and
visualized on an agarose gel (see fig. 2).
The DNA/RNA was transferred onto a nylon
Transfer:
membrane (see fig. 3) and the membrane was then
UV irradiated to permanently bind the DNA to
the membrane.
Prehybridization: Blotted the membranes with Salmon Sperm DNA.
Added water, Klenow Buffer, GAT mix, random
primers, radioactive CTP and Klenow Enzyme.
Hybridization: 108 SSPE, 28 SDS, and water.
Exposure:
24-48 hrs.
RNA Extraction: The tissue was ground in liquid Nitrogen. RNase
free reagents used. Extracted with Chloroform. Precipitated
polysaccherides with EtOH. The RNA was precipitated with Licl
in BME. Extracted with Acid Phenol. Precipitated with NaOAc
and EtOH. Quantitated on spectrometer.
RNA Gel: 1.28 agarose, 6.68 formaldehyde gel composition. RNase
free reagents were used. Running buffer: 0.2 M <span style='background-color:#cfc' title='Mops : Exact'>MOPS,</span> 10mM EDTA,
1OmM NaOAc, and 378 formaldehyde.
Light Regulation Experiment: filtered sea water; incubation
temperature: 15°C; light: 70 micro E/m's which is approximately
half the saturating illumination (Gerard et al, 1986).
RESULTS
The primers indeed were gene specific. Each primer was
challenged by introducing various templates of different sizes
to the PCR reaction and then analyzing it on a gel. Only one
product of the correct size was obtained (see fig. 1).
After a 24 hr. exposure, the Southern revealed promising
results. In the blot labeled with the coding region the molecular
weight marker was easily distinguished while many bands in each
digest lane were visible. On the blot labeled with the 2A
noncoding region, again, the molecular weight marker could easily
be spotted. Using the marker as a reference point, one band
in each digest lane could be distinguished, each of which
corresponded to one band on the blot labeled with the coding
region. These results highly suggest that the probes were gene
specific because only one band per lane was seen on the blot
labeled with the noncoding region.
Unfortunately, due to time constraints, results from the
Northern Blot Analysis were ambiguous; clear results could not
be obtained.
DISCUSSION
The evidence obtained strongly suggests the presence of
gene specific probes. When the genes are probed with the coding
region, eight different products are found in a southern.
However, when the 2A noncoding region was used as the probe
it hybridized to only one of the eight products, strongly
suggesting gene specificity.
More experiments, however, are needed. The development
and use of gene specific probes can be extremely helpful in
future experiments. Once these probes are made they can easily
be amplified by PCR reactions. These probes could monitor the
expression of genes through the measure of the mRNA. For example,
at different depths kelp blades are exposed to different
quantities of light and so future experiments could focus on
light regulation of the FCP genes. Also, experiments focusing
on tissue specificity and developmental stages (where FCP
production in juveniles and full grown kelp with canopies are
compared) or nutrient regulation (i.e. during upwellings) can
provide invaluable to the comprehension of gene families and
their regulation. In all, gene specific probes are but the first,
but nonetheless valuable, step in elucidating the need of various
different genes to produce one protein.
GENE EAMILY MAP
S
2A
.K8

48


4
1.1KB

2A PRIMER, SEQUENCE: GCA TCG CTA CCG ATT GTC
GG
4B PRIMER, SEQUENCE: TCG TCT CGT GAC CTC TCG
E
TG
AC PRIMER, SEQUENCE: GCA CGT GTC GGC ACG CGT

GG
T7+ PRIMER, SEQUENCE: GCG TAA TAC GAC TCA
CTA TAG G (CODNG REGON)
Fig. 1
1.7KB
Hig
Nylon
oxt butter

59
MA
DNA Daests
2
6 La weiget
+Gas
Vlatt
Jaget Tanl Sack
—3 Euer gger

e
Wick


ugpo
Vaneter oP DNA
0 Nlon Menrne
Mer
16
Fiwe
Prodet

Bier
Mallen
+
*
**
4
Vo
r
+


Zrttern
lyg
Eot
REFERENCES
Fyrberg,
E.A., Mahaffey, J.w.,
Bond,
B.J., and Davidson,
(1983). Transcripts of the Six Drosophila Actin Genes Accumulate
in a Stage-and Tissue-Specific Manner. Cell. Vol. 33, 115-123.
Gerard, V.A., (1986). Photosynthetic Characteristics of Giant
Kelp (Macrocystis pyrifera) Determined in situ. Marine Biology.
90, 473-482.
Snyder
Grossman,
Manodori,
A.
(1990).
A.
and
D.
Proteins of Diatoms: Their Relationship to
Light-harvesting
the Chlorophyll
a/b Binding Proteins of Higher Plants and Their
Mode of Transport into Plastids. Molecular General Genetics.
1-10.
Kellmann, J.w.
Pichersky, E., and Piechulla, B. (1990). Analysis
of the Diurnal
Expression Patterns of the Tomato Chlorophyll
a/b Binding
Protein
Genes.
Influence
and
Light
Characterization of
the Gene Family. Photochemistry
and
Photobiology, Vol. 52. 35-41.
Kishore
R.,
and Schwartzbach,
S.D.,
(1992).
Photo and
Nutritional Regulation of the Light-Harvesting Chlorophyll a/b
Binding Protein of Photosystem II mRNA Levels in Euglena. Plant
Physiology, Vol. 98. 808-812.
</pre></body></html>