<html><body><pre style='margin:2rem auto;width:fit-content'>Abstract
Mitochondrial DNA from the livers of adult rockfishes were
examined for differences in restriction fragment lengths. Äfter the
mitochondrial DNA from kelp rockfish, blue rockfish, black rockfish,
and other rockfishes was isolated, it was digested with seven dif¬
ferent six-base restriction endonucleases, run on agarose gels,
transferred to nylon membranes, probed with radioactive mitochon¬
drial DNA, and the resulting autoradiographs examined for polymor¬
phisms. We found that all the enzymes showed differences among
the three principal species, and that only the kelp rockfish displayed
polymorphisms within the species. This method might be used to
identify larval rockfish. The results also have interesting implica
tions for the population genetics of the kelp rockfish, <span style='background-color:#fcc' title=' : NoMatch'>S. atrovirens.</span>
Introduction
Purpose of the Study
There is a good deal of interest in managing rockfish populations
because rockfish are commercially important. Part of successful
management of a population is an understanding of its recruitment
patterns. Recruitment is the state of bringing new individuals from
one life stage to another (Powers, et. al., unpubl.). In this study we
are interested in ways to study larval recruitment of rockfish.
In order to study recruitment, it is important to be able to iden¬
tify larval fish, and rockfish larvae are difficult to identify. Ways
to identify larvae accurately and consistently need to be developed.
Studies of larvae are quite difficult because a fertile mother needs
to be kept until she bears her live young, and to investigate variation
within a species requires raising several mother fish until they
release their larval offspring. A graduate student from Moss Landing
who defended his master's thesis in May had some success in looking
at pigment differences of different species' larvae in a study he
recently finished, but there are still many technical problems with
this approach.
Another way to tackle the problem is through the use of isozyme
studies. However, in rockfish there is little variation within spe¬
cies and not enough between species so that recruitment studies
using isozymes are impractical (Seeb, 1986). Besides, it is not
clear if the isozymes that a larva uses are the same ones he uses as
an adult. DNA studies of rockfish have not been investigated until
recently. Seeb has begun to investigate mitochondrial DNA (mtDNA)
patterns in a group of rockfish to better understand their phylo¬
genetic relationships. MtDNA is particularly useful because of its
pattern of inheritance. It is possible that a species has one or a few
distinct types of mtDNA that no other species shares while it is
harder to find a unique version of an enzyme that sets that species
apart in isozyme studies.
We chose to study mitochondrial DNA over nuclear DNA for a few
reasons. First, there are a lot more copies of an individual fish's
mtDNA than its nuclear DNA because there are more mitochondria
per cell than nuclei, and each mitochondrion has several copies of
its genome (Alberts, et. al., 1985). Second, mtDNA has a pattern of
inheritance which helps in examining branching patterns on phylo¬
genetic trees. Third, its possibly higher mutation rate might show
more differences than would nuclear DNA.
Ihe Genus <span style='background-color:#cfc' title='Sebastes : Exact'>Sebastes</span>
The genus Sebastes is made up of nearly 100 species of rock-
fishes worldwide, and more than sixty of these are found in Cali¬
fornia waters (Burgess and Axelrod, 1984; Miller, 1972). It is the
most speciose fish in the eastern North Pacific. As a group, rockfish
make up the biggest catch for the fishing industry in California
(Amidei, 1986). Both commercial fisheries and sportfishers fish for
rockfish.
Some rockfish live near the shore on rocky bottoms or in kelp
beds while other species live offshore as far out as 300 miles and
as deep as 450 m (Phillips, 1957). Sebastes is in the only subfam¬
ily of <span style='background-color:#cfc' title='Scorpaenidae : Exact'>Scorpaenidae</span> that gives birth to live young (Seeb, 1986). That
is, rockfish have internal fertilization and viviparity. Adults range
in length from 15 cm to 90 cm and longevity varies from less than
ten years to as long as 100 years (Amidei, 1986). They are charac¬
terized by spines found on their dorsal and anal fins and also on
their heads. It can be very difficult to identify certain species if
not because there are just so many species to tell apart but because
many of the species look very similar. Color and pigmentation are
important in identifying species as are morphometric and meristic
characters (Phillips, 1957). Some researchers are turning to other
means of identification like gel electrophoresis.
The three species that were examined in this study were S.
atrovirens, <span style='background-color:#fcc' title=' : NoMatch'>S. melanops,</span> and <span style='background-color:#fcc' title=' : NoMatch'>S. mystinus</span> which are commonly
referred to as the kelp rockfish, black rockfish, and blue rockfish
respectively. All three are nearshore varieties with maximum
lengths ranging from 40 to 60 cm. All three have mottled coloring
with a predominant color that corresponds with that species name
(the kelp being a brownish-green color) (Phillips, 1957). The kelp
belongs to the subgenus, <span style='background-color:#cfc' title='Pteropodus : Exact'>Pteropodus</span> (seeb, unpubl.). The black and
blue varieties are more closely related to each other than they are
to the Pteropodus subgenus Seeb, 1986).
Mitochondria and Their Special Genome
Mitochondria are found in almost every cell of every eukaryotic
organism, so it is not surprising that they provide a unique and
essential function. It is generally believed that many millions of
years ago, they were independent energy-producing bacteria, and
they evolved into endosymbiotic organelles important for cell res
piration. The mechanism for the electron transport chain resides in
the mitochondria. Mitochondria divide independently of the cell and
are passed on to daughter cells through the cytoplasm. A cell has
from a few to a few thousand mitochondria depending on its energy
requirements (Alberts, et. al., 1985).
Although not always, mitochondria are usually inherited from the
mother (Park, pers. comm.). This is because the mitochondria in the
sperm (from the father) either do not survive or never enter the egg.
and the mother contributes thousands of mitochondria from her egg
As they used to be independent organisms, mitochondria use a
genetic code which is different from the nuclear genomes of their
"hosts“ and even from bacteria, which they are related to (Brown,
1985). Mitochondrial DNA is a circular duplex with a range of 16.3
to 19.2 kilobases (kb) in vertebrates (Brown, 1985). Beckwitt and
Petruska (1985) found the mtDNA of rockfishes to vary between 16.9
and 17.4 kb. The genome is haploid, but there are multiple copies of
it in each mitochondrion. mtDNA encodes 37 genes in animals: 22
are for tRNA's, two for TRNA's and 13 encode proteins. There is also
a control region, which is important in the regulation of mtDNA rep¬
lication and gene expression. Much of the sequence and length varia¬
tion occurs in this control region (Brown, 1985). There is some
debate concerning the evolutionary rate of mtDNA compared to that
of nuclear DNA. In mammals for instance, mtDNA seems to have five
to ten times more variation. The data for fish are not available, but
even the data that do exist for other organisms are difficult to
interpret (Park, pers. comm.).
What makes mtDNA particularly useful in population genetics is
its means of inheritance alluded to earlier. It is maternally inher¬
ited and haploid so non-deleterious mutations in the mtDNA of one
mother are inherited in all of her descendants without mixing of
genes. That means it is possible to look at phylogenies among
groups of organisms. New branches on a phylogenetic tree are crea-
ted with new mutations, but branches on the tree die when the last
of a female individual's female descendants fail to reproduce.
Sophisticated math proves that with an initial population size of n,
all the descendants of that population will be able to trace their
mtDNA to one female ancestor in 4n generations (Avise, 1986).
BELP Analysis
To analyze DNA, it is either cut into pieces using sequence-spe¬
cific restriction endonucleases or the nucleotide bases are se¬
quenced. Restriction endonucleases are enzymes which recognize a
certain sequence of nucleotide bases and cut the DNA molecule at
that sequence. When these fragments of DNA are exposed to an elec
tric potential in a gel, they travel at different rates according to
their length. Differences in these fragment lengths between indi¬
viduals are called polymorphisms. A restriction fragment length
polymorphism (RFLP) analysis is useful for many things from
explaining phylogenies to doing recruitment studies.
It is possible to get mtDNA from a rockfish larva but only a lim¬
ited amount (Park, pers. comm.). How to use this DNA with economy
presents a problem. Different restriction endonucleases recognize
different sequences, so choosing which ones to use in a study is
essential for economy of samples. Mitochondrial DNA happens to be
particularly rich in adenine and thymine (Brown, 1985), so maybe the
investigator wants an enzyme that recognizes a sequence with A's
and T's so the mtDNA gets cut more. What he really wants is an en¬
zyme that can point out a lot of differences in RFLP patterns be¬
tween individuals. The best way to find out which enzymes are best
for this is empirical testing of several different enzymes. I tested
how good certain enzymes were at producing polymorphisms, so that
these enzymes could later be used as diagnostic tools when dealing
with the limited amount of mtDNA in a larval rockfish.
Methods
Rockfish were collected by the California Department of Fish and
Game, and the livers from the fish were dissected. Liver was the
tissue of choice because it is so rich in mitochondria. As will be
described later, mitochondrial DNA was extracted according to the
Chapman and Powers (1984) method. The DNA samples were diges¬
ted with an assortment of restriction endonucleases, and the
products of these digestions were run on an agarose gel. A Southern
blot transferred the DNA to a nylon membrane which was probed
with mitochondrial DNA from <span style='background-color:#cfc' title='Fundulus heteroclitus : Exact'>Fundulus heteroclitus</span> in a lambda
phage. The radioactive membrane was exposed to X-ray film, and the
film was developed and interpreted.
Collection of Samples
The fish were collected by Dave VanTresca and others at the
Monterey office for the California Department of Fish and Game.
Using SCUBA, the collectors speared fish and tried to keep them cool
until they could be dissected. We had asked them to look for spe¬
cific species on their dives, and they identified the species.
Either they would dissect the livers for us or we dissected the
livers and the hearts also in some cases. The tissues were placed
labelled bags on ice. The people from Fish and Game kept track of
measurements of the fish, their sex, and other data for their own
records. (These data may prove to be useful for further population
studies of these samples.)
There was no more than a two hour delay between collection and
dissection. The first collection was off of Del Monte Beach in Mon¬
terery. In addition to several kelp rockfish, blues, and blacks, a
copper <span style='background-color:#fcc' title=' : NoMatch'>(S. caurinus)</span> and a gopher rockfish <span style='background-color:#fcc' title=' : NoMatch'>(S. carnatus)</span> were col¬
lected at this site. The second also came from Del Monte Beach. A
spearfishing contest at Carmel River State Beach was the third col¬
lection site. These fish were not actually caught by the Department
of Fish and Game, but they were the sponsors. We took kelps, blacks,
and blues from the event. Linda and 1 accompanied Dave VanTresca
and Peter Nelson, a graduate student from UC Santa Cruz, on the
final collection trip just north of the boardwalk in Santa Cruz. They
speared kelps, blacks, and blues and an olive rockfish <span style='background-color:#fcc' title=' : NoMatch'>(S. serranoi¬</span>
des). All of this is summarized in Table 1.
Isolation of mtDNA
Although these methods were not all carried out throughout the
study, the following is the most refined version of the methods that
were eventually performed.
On a piece of glass on ice, a small (- 500 ul) slice of liver was
cut, and saved at -70°C. About three grams of the remaining liver
were minced with a razor to a liquid consistency. If the liver was
big enough and parts of it could be spared, then pieces of liver not
containing the tough connective tissues were chosen.
The minced liver tissue was placed in a glass tube with 15 ml of
TEK (.O5M Tris, pH 7.5, .OIM EDTA, and 1% KCl.) The tube was placed
in an ice bucket and the sample was ground with a teflon pestle.
Grinding continued until it was possible to make two complete
passes with the pestle.
The liver homogenate was then placed in a centrifuge tube and
layered with a 15% sucrose/TEK solution that was passed through
pasteur pipet under the homogenate. This was spun at 1000 G for
ten minutes. The supernatant was drawn off from the lower layer of
debris, but the thin layer of fat above the supernatant was not saved.
This step of layering with sucrose/TEK, spinning, and drawing off
the supernatant was repeated once.
The supernatant from the second spin was spun at 18,000 G for
thirty minutes, which leaves a pellet of mitochondria in glycogen.
The supernatant was discarded, and the mitochondrial portion of the
pellet was resuspended in one milliliter of TEK. At this point, each
sample was divided equally into two eppendorf tubes.
The mitochondria were lysed by adding ten microliters of 20%
SDS. To increase the effectiveness of the SDS, the samples were
placed in a 37°C water bath for about five minutes. To purify the
DNA, 500 ul of phenol was added next, the mixture shaken vigor-
ously, and then spun at full speed in the microfuge for ten minutes.
The top layer was drawn off from the lower phenol layer and the
unwanted white material at the interface. The phenol extraction
was repeated once. To clean the phenol out of the sample, the same
extraction procedure was performed with chloroform. However, the
spin for the chloroform extraction was only five minutes.
To cause the DNA to precipitate out of the TEK solution, 25 ul of
5M Nacl and one milliliter of -20°C ethanol were added to the solu¬
tion. After several hours in the freezer, the tubes were spun at full
speed in the microfuge for fifteen minutes. The supernatant was
discarded, and the pellet of DNA resuspended in 100 ul of 1X TE.
Finally, the two tubes for each sample were combined in one tube.
Enzyme Digestions of DNA Samples
Seven different restriction endonuclease enzymes were used to
examine differences in cutting patterns between individual samples.
Six-base cutters were chosen because they generally have from zero
to five or more restriction sites on mitochondrial DNA. The enzymes
were BamHl, Clal, ECORI, ECORV, Hindill, Pstl, and Xhol.
Enzymes were placed in distilled water and the appropriate salt
for that enzyme so that the digestion would be in a 10% salt buffer
solution (i.e. 2.5 ul salt for a 25.0 ul reaction.) For each reaction,
0.3 ul of enzyme in 15.0 ul was used to digest 10.0 ul of DNA sus¬
pended in TE. The labelled reaction tubes were finger-vortexed and
placed in the 37°C water bath for several hours as the enzymes
acted.
To ensure that the digestion was complete, another 0.3 ul of the
same enzyme in an extra 5.0 ul of buffered solution was added to
each reaction tube. Äfter several hours in the warm water bath.
some of the liquid in the tubes had condensed on the side of the
tubes, so the tubes were microfuged for about fifteen seconds so
that its contents would all be interacting together at the bottom of
the tube. Once again, the tubes were finger-vortexed and placed in
the 37°C water bath for several hours.
If a double digestion was being performed, then 0.3 ul of the
second enzyme would be added instead of the extra 0.3 ul of the
same enzyme. Otherwise double digestions were done the same way.
Bunning Gels
The agarose gels were made with 0.8% agar in 1X TAE. The agar
solution was heated in the microwave for several minutes until it
boiled. While the agar boiled, an 11 X 14 cm gel tray was prepared
by taping its ends tightly and placing a fourteen-toothed (thin tooth
variety) comb at one end. The very hot agar was poured, making sure
there were no bubbles and no leaks from around the tape. This
cooled for about thirty minutes, at which time the tape and comb
were removed.
The fourteen lanes were each filled with the full contents (30u))
of a reaction tube using a predetermined order. One lane in each ge¬
was reserved for the standard, lambda phage cut with Hindlll in a
solution containing a blue dye. The dye has two components, one
which travels at the same pace as a 100 base pair fragment and one
at 5000 base pairs. This standard dye is used to estimate how far
the DNA has migrated in a gel. Ten microliters (.25 to .50 ug) of the
standard were loaded in the standard lane.
The gel trays were placed in the gel rigs and the rigs filled with
about one liter of 1X TAE buffer. The rig was filled with TAE to just
under the top of the gel. The power was turned on to 100 volts for
about five minutes, and the DNA entered the gel. Then, the gel was
covered with more TAE and run at a lower voltage, to achieve good
separation of the fragment bands. Voltages used ranged from
twenty-five to seventy-five volts, and the gels ran for several hours
and often overnight.
When the leading dye front had gone about three-fourths of the
way across the gel, the power was turned off, and the gel was
prepared for visualizing the DNA. The gel was placed in 500 ml of
the buffer it just ran in with 100 microliters of ethidium bromide
solution (10 mg/ml). The gel soaked in the dilute ethidium solution
for at least ten minutes. Then, the gel was rinsed twice with tap
water.
The gel was placed on the ultraviolet light source in the dark
room, and a transparent ruler was placed over the standard lane to
serve as a reference. At an f-stop of 11, the shutter was released
for three seconds, and a polaroid picture was developed immediately.
Southern Blots
The gel was covered with denaturing solution and gently rocked
on a shaker for ten minutes. The denaturing solution was replaced
with fresh denaturing solution, and the gel was once again shaken
for ten minutes. Äfter a rinse with neutralizing solution, the gel
was covered with neutralizing solution and gently rocked for fifteen
minutes. The neutralizing solution was replaced, and the gel was
shaken for another fifteen minutes.
A blot was made by layering several materials around a gel. Two
pieces of Whatman's paper cut to the size of the gel had soaked in
20X SSC and were then placed on a sheet of plastic wrap on a flat
surface. Air bubbles between the paper and plastic were removed by
stroking them out on the sides. The gel was placed on the paper and
air bubbles removed again. A sheet of nylon membrane which was
cut to size and wetted in water was soaked in 20X SSC, and it was
placed on the gel. Air bubbles were carefully removed before ano¬
ther piece of 20X SSC-soaked Whatman's paper was layered next. A
stack of paper towels was placed on the gel layer, and a glass plate
placed on the whole stack. Finally, an object with some degree of
mass such as a full flask, was placed on the plate.
The blot sat on the bench for at least eight hours before the nylon
membrane was removed and saved in a sealable plastic bag. This bag
was placed on the ultraviolet light source with the DNA side of the
membranes facing the light and then exposed to UV light for two
minutes. This covalently bonds the DNA to the nylon.
Hybridization
The filters were prepared for hybridization with a pre-hybridiza¬
tion solution. This solution is three parts 20X SSC and one part each
of SDS and nonfat milk in fifteen parts water. It was poured in the
bag and the air bubbles removed. Then the bag was sealed and placed
in a 65°C water bath for two hours, at which time it was ready for
hybridization.
did not grow the probe, so I cannot precisely explain the me¬
thods of growing it. However, I do know that the probe we used is a
lambda phage with a <span style='background-color:#cfc' title='Fundulus : Exact'>Fundulus</span> mtDNA insert, and it was grown in <span style='background-color:#fcc' title=' : NoMatch'>E.
coli.</span>
The probe was labelled with cytosine nucleotides containing the
phosphorus radioisotope 32P. To label the probe, 12 ul (.25 ug) of
the probe DNA and 1.0 ul of random DNA primers were boiled in a
screw-top tube for five minutes. While the DNA was boiling, a mix¬
ture of 2.0 ul of Klenow, 2.0 ul of adenine, thymine, and guanine, and
2.5 ul of Klenow buffer was made. The hot DNA was cooled quickly
on ice before the Klenow enzyme mixture was added. This safeguard
prevents denaturing of the enzyme when the two solutions are
mixed. Äfter the Klenow was added to the DNA, 5.0 ul of the radio¬
active dCTP was added, and the reaction occurred for the next two
to four hours in a lead container.
Eighty microliters of 1X TE were added to the radioactive probe
to stop the reaction. To remove the unincorporated dCTP's, the probe
mixture was run through a sephadex column and microfuged for about
five minutes. The filtered solution was boiled in a screw-top tube
for ten minutes to denature the DNA strands. Then, the bag with the
nylon membranes was cut open, the radioactive probe injected into
the bag, the bag resealed, the contents mixed around inside the bag,
and the bag placed into a 65°C water bath for twelve hours.
Next, the membranes had to be washed. The liquid contents were
carefully disposed of into the radioactive waste, and the membranes
were rinsed in a wash solution of 0.2 % SDS and 2X SSC. This rinse
requires only enough wash solution to rinse off the membranes and
still be a reasonable amount to throw away in the radioactive waste,
For the next wash, the membranes soaked in more of the same wash
solution for ten minutes. For the last wash, the wash solution was
replaced with fresh wash solution which had been diluted by one¬
half, and the membranes soaked for fifteen minutes. The membranes
were rinsed with distilled water, and they were ready for exposure
to film.
Developina
The membranes were patted dry with paper towels and wrapped
in one layer of plastic wrap. The wrapped membranes were taped to
a developing screen. Then, X-ray film was placed between the mem¬
branes and another developing screen. This was closed tightly into a
developing cartridge and placed in a -70°C freezer. The film was
exposed to the radioactive bands in the membranes for three days at
which time the film was run through the developing machine, produ¬
cing the final product, an autoradiograph.
Results
The first result to report is the success of the probe we used. It
appears that a mtDNA probe from Fundulus is sufficient for hybri¬
dizing with rockfish mtDNA. At least earlier in the study, the auto¬
radiographs were clear, and the probe seemed to be working fine.
Later, unexpected problems arose with the autoradiographs. It was
difficult to get the probe radioactively labelled, and this may have
been the problem. However, the problem could have been in any of a
number of steps from denaturing the gels to UV-fixing the mem¬
branes to hybridizing and washing the membranes. Nevertheless,
was able to get enough data from the autoradiographs that did deve¬
lop and from the polaroid pictures of the ethidium-stained gels, that
could quantify the data in some ways.
was able to see fragment patterns of blacks, blues, and kelps
for all the enzymes considered. A maximum of around ten to fifteen
individuals in each species were informative for any given enzyme,
although sometimes fewer than five individuals in one species could
be clearly analyzed for an enzyme. The four individuals from other
species did not always show up clearly for every enzyme they were
digested with.
No polymorphisms appeared in black rockfish with any of the
enzymes tested. Only one polymorphism was detected in the blue
rockfish, and that was with Xhol; two of the blue rockfish had a
restriction site that was not shared by the other blues.
Kelp rockfish usually had two polymorphisms with each enzyme
tested: only with BamHI and ECORV were no polymorphisms detected.
Hindlll had two sites that were variable within the species, and the
other four enzymes each had one variable site. In eight kelp rockfish
individuals that were collected together, two groups of four segre¬
gated together in four of the six variable sites. Only one individual
kelp differed from the other kelps at the fifth site, and the sixth
site that is variable within kelps is the same one that seemed to be
variable in the blue rockfish (a cut that creates a 3 kb fragment in
Xhol.) Double digests were used to more closely examine the poly¬
morphisms within the kelp species. Clal and BamHl were used to¬
gether (see figure 1), and Pstl and Clal were tested together also.
The polymorphisms we had suspected were confirmed by the double
digests.
In addition to restriction fragment length polymorphisms in the
kelps, length variation was also observed. One of the individuals is
consistently smaller than the others, and there are possibly other
length differences that are harder to detect.
All of the enzymes showed some differences between the species
(see figures 2-8). It is not possible to make accurate maps of the
restriction sites because there are no sequence data available for
these samples and because not enough double digests were per¬
formed. Therefore, the following figures are only proposed restric¬
tion maps for each of the enzymes. They are schematic representa¬
tions of my best interpretations of the bands I was able to detect on
either the autoradiographs or the polaroid pictures.
For example, in figure , we see that BamHI, had two polymorph¬
isms. The blue, kelp, gopher, and olive rockfishes had a pattern con¬
sisting of two bands, a pattern which would be produced by two re¬
striction sites. The black rockfish's DNA digest shows only one
band which means there was one restriction site. These sites are
labelled in the diagrams at the bottom of the figure. According to
the most parsimonious interpretation of the data, site a is shared by
all the species, and site b has been lost in black rockfish. The top
line in the schematic of the bands represents the well where the
lane started. The numbers next to the other lines give the approx¬
imate lengths of those fragments in kilobases. (O.C. is an abbrevia¬
12
tion for open circle, a distinctive band which travels about half the
distance from the well to the band where linear fragments greater
than 20 kb cluster.
think the figures are fairly clear upon exam¬
ination, and I will discuss them in greater detail in the discussion
section.
From the variability of the sites, genetic distances were calcu¬
lated and a cladogram was also constructed. Only the data from the
kelp, blue, and black rockfish were used because no other species
had complete data for all the enzymes. As it is, these results are
merely speculative. Not nearly enough individuals were examined to
give significant results; however, it is still interesting to see what
phylogenetic information these data give.
Genetic distance, d, can be calculated according to Nei's genetic
distance
d= Ein(S))/r
where r is the number of nucleotide bases per restriction site and
(S) is the probability that a pair of sequences X and Y share a given
restriction site according to
(S) = 2nxy/(nx + ny).
In the last equation, the ny and ny terms refer to the number of DNA
fragments in all the digests for species X and Y while nyy is the
number of shared fragments (Hartl and Clark, 1990). Table 2 gives
calculated values for all the groups of individuals that had the same
polymorphisms for every enzyme, or in other words, all the groups of
individuals that segregated together.
The cladogram was constructed with McClade on a Maclntosh SE
(see figure 9). Cladistics attempts to build phylogenetic trees based
on informative restriction sites. For all the sites which show dif¬
ferences between groups, presence or absence of the site is recor¬
ded. It two or more groups share the presence or absence of a site,
then that site is phylogenetically informative. Based on the number
of steps (site changes) it would take for the groups to arrange
themselves in a particular order, a best tree is selected which
requires the least number of steps. This tree is presented in fig. 9.
13
Discussion
We looked at mitochondrial DNA in rockfishes in the first place
because we wanted to see if it would be a useful tool for identifying
larvae in recruitment studies. With the limited scope of these
findings, the answer appears to be yes because mitochondrial DNA is
cut into different patterns of fragment sizes by certain enzymes
according to species. All the enzymes we used showed differences
between species.
It is important to note that much of the raw data from the auto
radiographs and polaroids was quite difficult to interpret due to
experimental problems and the poor resolution we could see. was
able to see the band patterns for about ten individuals for each spe¬
cies with each enzyme; however, I did not always see that many
individuals. Nevertheless, the results I reported seem consistent
for all the enzymes with the exception of Xhol. In the case of Xhol.
could detect two patterns: one which appeared to be a single frag¬
ment which had one restriction site resulting in a 17 kb fragment
and one which resulted in two bands of lengths 14 and 3 kb, presu¬
mably due to one additional site. It is often difficult to see the
bands of small DNA fragments because they bind less ethidium, and
the radioactive probe has less to bond to on a small piece of DNA.
Larger fragment sizes are difficult to estimate because they tend to
cluster together due to the logarithmic nature of their migration
rate through the gel. Thus, it is easy to see how I could not have
seen a small band in some lanes and how 1 could have mistaken a 14
kb fragment for a 17 kb fragment in the Xhol digests. For this and
other reasons which will be brought up in this section of the paper,
would propose that the data 1 reported from the Xhol digests be con¬
sidered invalid.
The data from the other enzyme digests show that RFLP analysis
using mtDNA is useful for differentiating between at least kelp,
blue, and black rockfishes. EcoRl has different banding patterns for
each of these species, so it could conceivably be used as the diag¬
nostic enzyme for them. However, the enzymes we used did not
show differences between the individuals from some of the other
species (copper, gopher, and black and yellow rockfishes) and some
of the kelp rockfish. However, this is a very interesting result
which I can discuss only after describing the variation within the
kelp rockfish species.
14
The kelp species divides into two main groups of RFLP patterns,
which I call kelp A and kelp B. Eight kelp individuals that were col¬
lected together could always be analyzed in each of the digests, and
four were A type while four were B type. In the six reliable enzymes
we tested, there are five restriction sites which show polymorph¬
isms in S. atrovirens, the kelp rockfish. One of these polymorph¬
isms was a single site difference in Pstl in one individual (of the A
type) which set it apart from the other kelp individuals. With the
other four site differences, the individuals of the A type always
segregate together as do all the individuals of the B type. The black
and yellow rockfish <span style='background-color:#fcc' title=' : NoMatch'>(S. chrysomelas)</span> segregates with the kelp B
type in digests with Clal, EcoRI, and Hindlll, and it segregates with
both kelp types together in the ECORV digest. The copper rockfish
(S. caurinus) segregates with the kelp B type in the Hindlll digest at
both sites, and with both kelp types together in the ECORV digest.
The gopher rockfish (S. carnatus) segregates with kelp B type in the
Clal digest, and with both kelp types together in the BamHI, ECORV
and Pstl digests. It is not surprising that these three individuals
from other species segregate closely with the kelp rockfish when
one considers that all four belong to the same subgenus, Pteropo¬
dus. If the five groups of fish (including the two groups within S.
atrovirens) are all in the same subgenus, then why is it that kelp A
is so different from all the other groups of fish? The variation
within the kelp species is greater than the overall variation of the
subgenus, an obviously significant result.
There are at least four explanations for the result that in terms
of mtDNA, the kelp A group is so different from the other fish in the
subgenus. The first is simply that the kelp rockfish individuals
were not identified correctly in the first place. The species most
easily confused with the kelp rockfish is the grass rockfish <span style='background-color:#fcc' title=' : NoMatch'>(S.
rastrelliger)</span> which is also in the same subgenus with the kelp
rockfish. One would expect that the grass rockfish's mtDNA would
look similar to the mtDNA of the other species in the subgenus,
which all look very similar in RFLP analysis. If it is not actually
kelp rockfish and not grass rockfish either, it might be a more dis¬
tantly related fish in the genus.
The other three explanations of the strange result try to explain
the result assuming the species were identified correctly. They also
try to account for the fact that it is unusual for a species to support
two different strains of mtDNA because branches seen with mtDNA
on the tree of an organism are expected to die out with time as dis¬
cussed earlier. The second explanation is that there is actually a
cryptic species that has never been recognized before. This seems
unlikely, but some support comes from the kelp rockfish's depth dis¬
tribution. Often, two species of rockfish which occupy a similar
niche will live in mutually exclusive depth ranges. For instance,
Hallacher (1977) found that the black and yellow rockfish occupies
the shallower water, and the gopher rockfish lives in the deeper
water in the Monterey Bay area. In the same study, he found the kelp
rockfish distributed evenly through the different depths. Perhaps,
two species could be recognized, one would find that the spatial
distribution would get divided as it did in the above example. Using
allozyme studies, it should be simple to test the hypothesis that
there is a cryptic species. Comparing the kelp A mtDNA type to the
kelp B type, if one were to find that an allozyme is fixed in one type,
and a different allozyme is fixed in the other type, then that would
be proof of the cryptic species. The third and fourth explanations
are that the present kelp species comes from two different popula¬
tions that were not mixing genes but are now, and their mtDNA types
both persist. The two populations could have been different species
which hybridized, resulting in the present species of kelp rockfish.
Hybridizations are not at all uncommon in Sebastes (Seeb, 1986),
and the possibility exists that a hybrid was once fertile. The fourth
explanation is that a population of kelp rockfish was once isolated
from another population of kelp rockfish long enough for a different
mtDNA type to develop, and now those two populations are mixing.
Support for this would come from allozyme studies which have un¬
fortunately not been conducted in depth for the kelp species. A stu¬
dent from last year's spring class found little allozymic variation
within the kelp species (Bachmann, 1989), a finding which supports
the last hypothesis. Another one of her findings was a significant
difference in allozymes between the kelp rockfish and the black and
yellow rockfish, which contradicts the earlier hypothesis that there
exists a cryptic species. If there were a cryptic species, one would
expect that there would be more allozymic variation within the kelp
species than Bachmann found. Öther evidence which suggests that
isolation may have once occurred is the current evidence for some
geographical separation. It is known that there is a geographic
boundary between central and southern California populations of S.
﻿16
atrovirens which is manifested in differing numbers of tympanic
spines (Love and Larson, 1978). Whatever the case may be, this sub¬
ject warrants further investigation.
As far as the other species which has not been discussed vet.
there is not enough information to make a conclusion concerning the
olive rockfish <span style='background-color:#fcc' title=' : NoMatch'>(S. serranoides)</span> and its mtDNA. With allozyme
studies, it appears to be a sister species with S. mystinus, the blue
rockfish (Seeb, 1986) . In the only digests for which I have included
olive rockfish data, it seems to segregate with the blue rockfish.
However, inconclusive data from other digests seemed to show
differences between the two species. If this is true, then it would
demonstrate how much more effective mtDNA studies are in showing
differences between species as compared to isozyme studies.
More samples need to be analyzed, and more enzymes need to be
tested to get a more accurate assessment of mtDNA variation in the
genus Sebastes. To further resolve the question of the kelp rock¬
fish species, allozyme studies should be conducted. In addition.
direct sequencing may give the higher degree of resolution that is
needed to make the best conclusions concerning the interesting
results we found here.
Acknowledgments
I will never forget my experience at the Hopkins Marine Station,
and I have several people to thank for it. Without Linda Park, my
project would not have existed, and I appreciate the great deal of
time, work, patience, and care she gave to me this quarter. I would
also like to thank my advisor, Dennis Powers for his help and
instruction and for letting me work in his lab. Rob Rowan, Lynna
Hereford, Lani West, Doug Crawford, and all the others in the lab
really helped me get through my project. Dave VanTresca and the
other collectors did excellent work. Sam Wang and Jenifer Levitt
put great effort into making the class a success. I thank all the
other faculty and staff at the Marine Station for keeping things
going and making the class feel so welcome. Without the
encouragement of my peers who took the class in previous years, my
advisor Bob <span style='background-color:#cfc' title='Simoni : Exact'>Simoni,</span> and other professors back at Stanford 1 may not
have come, so 1 thank them. Finally, I would like to thank my father
and my parents for their support and encouragement during the
quarter.
Table 1
Rockfish Collections
Site
Date
Species
4-25-90
Monterey
Kelp
Del Monte Beach
Black
Blue
Copper
Gopher
5-1-90
Black and Yellow
Black
Blue
Carmel River State 5-17-90
Kelp
Beach
Black and Yellow
Black
Blue
5-22-90
Santa Cruz
Kelp
Boardwalk
Blue
Olive
Black and Yellow
Black
Number
Table 2
Nei's Genetic Distances
Kelp B
Kelp A 1 ind.
Kelp A 3 ind.
Black Blue
Kelp B 18
﻿0274
0209
0293
.0460
Kelp A1 14
.0055
﻿.0402
﻿0278
15
Kelp A3 15
16
.0341
﻿.0460
Black 13
13
.0304
Blue 11
Values on the diagonal are how many fragments that group has in total for all six reliable
enzymes. Values below the diagonal are shared fragment numbers. Values above the
diagonal are Nei's genetic distance (see text.)
Literature Cited
Alberts, Bruce, et. al. Molecular Biology of the Cell. Chapter 9:
Energy Conversion: Mitochondria and Chloroplasts. Garland
Publishing: New York. pp 483-548, 1983.
Amidei, Rosemary. Editor. Rockfish: A Focus for Research?
Proceedings of a California Sea Grant Workshop April 4, 1986.
Avise, J.C. Mitochondrial DNA and the evolutionary genetics of higher
animals. Phil. Trans. R. Soc. Lond. B 312, 325-342, 1986.
Bachmann, Anja. Electrophoretic Analysis of <span style='background-color:#cfc' title='Sebastes atrovirens : Exact'>Sebastes atrovirens</span>
and <span style='background-color:#cfc' title='Sebastes chrysomelas : Exact'>Sebastes chrysomelas</span> in the Monterey Bay Area. Unpublished
MS on file at Hopkins Marine Station.
Beckwitt, Richard and John Petruska. Variation in Mitochondrial DNA
Size Among Fishes of the Family Scorpaenidae. Copeia. Vol. 4,
Pp 1056-1058, 1985.
Brown, Wesley M. The Mitochondrial Genome of Animals. In
Molecular Evolutionary Genetics. (R.J. Maclntyre, ed.) Plenum;
New York. pp 95-130, 1985.
Burgess, Warren E. and Herbert R. Axelrod. Fishes of California and
Western Mexico. Pacific Marine Fishes Book 8. T.F.H.: Neptune
City, N J. pp 2118-2140, 1984.
Chapman, Robert W. and Dennis A. Powers. A Method for the Rapid
Isolation of Mitochondrial DNA from Fishes. Maryland Sea Grant
Program Technical Report, 1984.
Hallacher, Leon Ernest. Patterns of Space and Food Use by Inshore
Rockfishes (Scorpaenidae: Sebastes) of Carmel Bay, California.
Doctoral Dissertation from UC Berkeley, 1977.
Hartl, Daniel L. and Andrew G. Clark. Principles of Population
Genetics., Second Edition. Sinauer Associates: Sunderland, MA.
Pp 331-335, 1989.
Love, Milton S. and Ralph J. Larson. Geographic Variation in the
Occurrence of Tympanic Spines and Possible Genetic
Differentiation in the Kelp Rockfish (Sebastes atrovirens).
Copeia. Vol. 1, pp 53-59, 1978.
Miller, Daniel J. and Robert N. Lea. Guide to the Coastal Marine
Fishes of California. California Fish Bulletin No. 157. pp 1-249.
1972.
Phillips, Julius B. A Review of the Rockfishes of California.
California Fish Bulletin No. 104, pp 1-158, 1957.
Powers, Dennis A. et. al. Application of Molecular Techniques to the
Study of Marine Recruitment Problems. Unpublished,
Seeb, Lisa Wishard. Biochemical Systematics and Evolution of the
Scorpaenid Genus Sebastes. Doctoral Dissertation from
University of Washington, 1986.
Seeb, Lisa W. Untitled Project Description for Analyzing
Mitochondrial DNA of the Sebastes subgenus Pteropodus.
figure 1. Two autoradiographs. Above is the double digest of three kelp individuals with
BamHl and Clal. The upper of the three bands for the individual is the BamHl digest, the middle
lane is the double digest, and the lower lane is the Clal digest The middle individual (of the
Kelp A varirty) seems to have a restriction site in Clal, but the 17 kb fragment is DNA that
has been physically nicked open. This is true because the double digest shows the BamHl
fragments not getting cut again. In the lower gel, several individuals are digested by Hindll.
From the top, a copper, a black and yellow, a black, and seven blues. The black and blue
rockfishes have the same RFLP, and the other two share an RFLP. The middle band of the blues
and black has been cut once more in the other two lanes.
figures 2 - 8. Schematics of the enzyme digestions. Proposed band lengths are labelled.
O.C. stands for open circle configuration. Below the bands are the proposed maps. They are
not necessarily accurate, just a visual aid to understanding the fragment patterns.
figure 9. Cladogram of the kelp groups, black, and blue rockfishes from the McClade
program.
Figure 1
Ban

8/


.

2a
Nif
8/
1B
84


2
5

4



Cano


6
0 1



15.
5




Ms-12
N
Blue
Kelp
Olive
Gopher

15
2 —
15
BamHI RFLP'S
Figure 2
Black
17 —
Blue
Kelp A
Olive
—
O.C. —
Clal RFLPS
Figure 3
Kelp B
Black
Gopher
Black and Yellow
17 —
o
s -
+
+

—
S
Blue
Black
14—
3 —
14
2
Kelp
Gopher
Black and Yellow
Copper
—
—



ECORV RFLP'S
Figure 5
HindlII RFLP'S
Kelp B
Blue
Black & Yellow Black
Copper
—
O —
10 -
5 —
4 —
2 —
2 —
—
10

Figure 6
6

Kelp A
6
—
5

4
2 -
a
4
Blue
Black
Kelp (1 indiv.)
10 —
4—
2.5 —
5 —
5.5
4.5
)
25c
PStI RFLP'S
Figure 7
Kelp (all others)
Gopher
5.5 —
4.5 —
4
2.5—
5 —
10
1.0
25/c
Blue (2 indiv.)
Black
Kelp (4 indiv.)
Gopher
14—
3 —

Xhol RFLP'S
17
Kelp (4 indiv.)
Blue (most indiv.)
Black and Yellow
Figure 8
5
e
.


ha
Ctar step-
reversible

Figure 9
</pre></body></html>