<html><body><pre style='margin:2rem auto;width:fit-content'>Abstract. The cryptic gastropod species <span style='background-color:#cfc' title='Nucella emarginata : Exact'>Nucella emarginata</span> and <span style='background-color:#cfc' title='Nucella ostrina : Exact'>Nucella ostrina</span> are distributed
along the northeastern Pacific coast from Alaska to Baja, California. <span style='background-color:#fcc' title=' : NoMatch'>N. emarginata</span> is found
primarily in the south and <span style='background-color:#fcc' title=' : NoMatch'>N. ostrina</span> is found primarily in the north, with their ranges
overlapping in central California. Snails (7- 34 individuals) from four locations in the overlap
zone were collected and their population structures were studied using an approximately 500 bp
fragment of the mitochondrial gene cytochrome c oxidase subunit I (COI). Contrary to what
might be expected, analysis showed snails from the southern locations of Soberanes Point and
San Luis Obispo are both N. ostrina (the northern species), while snails from the northern
locations at Hopkins Marine Station in Pacific Grove and Point Joe are both N. emarginata.
Although populations in each location are fairly homogenous, small amounts of variation do
exist within each population (i.e. some polymorphisms are present). Of particular note is the
variation that exists between snails collected from a wave exposed area at Hopkins Marine
Station and snails in a wave protected area on the other side of the same point. Despite this intra-
population variation, each location forms a genetically distinct group. Further sampling in
central California will give more insight into the distribution of <span style='background-color:#cfc' title='Nucella : Exact'>Nucella</span> over short geographical
distances and will hopefully elucidate more of these species’ evolutionary histories, as well as
possibly helping to determine what factors are important in Nucella distribution.
INTRODUCTION
Isolation between species leads to speciation. This isolation can be the result of
geographic barriers, as is the case with allopatric speciation. However, speciation can also occur
in the absence of obvious physical barriers, especially in the marine environment (Palumbi
1992). Here, isolation is the result of genetic differentiation, which in turn leads to reproductive
isolation. When species have large ranges, but low dispersing larvae, genetic differentiation
should be seen between distant populations since gene flow between them will be small. This
process is known as “isolation by distance“ and is generally intended to apply to continuously
distributed populations (Wright 1943). In a similar model, the "stepping stone" model (Kimura
and Weiss 1964), gene flow occurs between adjacent populations, but not between more distant
ones. Both of these models predict a decrease in genetic correlation over long distances.
Genetic differentiation over geographic distance has been observed along the Pacific coast of
North America in several species, including copepods (Burton et al. 1979, Burton and Feldman
1981) and corals (Hellberg 1994, Hellberg 1995). In the case of corals, Hellberg (1995) has
shown that the coral <span style='background-color:#cfc' title='Balanophyllia elegans : Exact'>Balanophyllia elegans</span> follows the stepping stone model over smaller
spacial scales, but not at the rangewide level. Both of these models would lead to parapatric
speciation, speciation in which some gene flow occurs (reviewed in Gavrilets et al. 2000). Taxa
with low larval dispersal make good systems for studying patterns of speciation because they
have higher rates of both speciation and extinction (Hellberg 1994). This study focuses on two
sister species from the gastropod genus Nucella, the dog-whelks, as another model system for
studying genetic differentiation over small geographic ranges. This system, however, is unique
because two closely related, and recently diverged, species can be studied simultaneously in a
range where their distributions overlap.
Members of Nucella are found along the eastern Pacific coast from Alaska to Baja,
California (Palmer et al. 1990, Marko 1998). Due to extreme morphological similarities between
these species, they have had a confused taxonomic history. Most recently, Palmer et al. (1990)
showed that the "species" N. emarginata was in fact two cryptic species. The southern
populations retained the name N. emarginata and the northern populations became N. ostrina.
Since then, considerable effort has been spent trying to describe reproductive, morphological and
genetic differences between these two species (Palmer et al. 1990, Marko 1998, Marko et al.
2003). N. emarginata and N. ostrina are fairly easy to distinguish by the shape of their egg
cases, with N. emarginata eggs being shorter and fatter than those of N. ostrina (Palmer et al.
1990). In contrast, using shell morphology to distinguish between species tends to be quite
difficult since several morphs exist within both species (Marko et al. 2003, Crothers 1984,
personal observation). Palmer et al. (1990) and Marko et al. (2003) suggest that N. emarginata
and N. ostrina can be differentiated by the number of body whorls, whether the spiral cords are
knobby or continuous, the size of the aperture, and the presence of an adapical tooth on the
columella. However, these traits are not always well defined and do not provide a reliable
method for identifying the species. Genetic methods are also available to distinguish between N.
emarginata and N. ostrina. Marko (1998) used both nuclear (allozyme) and mitochondrial DNA
to identify Nucella species along the northeastern Pacific coast. His analysis showed that N.
emarginata and N. ostrina are genetically distinct at nine allozyme loci. He additionally found
twenty-five distinct mitochondrial DNA haplotypes. Using these molecular techniques, Marko
found that N. ostrina is present from Alaska to central California, while N. emarginata is found
from central California to Baja, California. There is a zone of overlap between the two species
in central California. Marko further suggests that these two species diverged as the result of
reproductive isolation caused by a biogeographic boundary, possibly at Point Conception. The
current overlap of species in central California could then be the result of northward range
expansion by N. emarginata.
This study focuses on a closer examination of the distribution of Nucella in central
California, from Pacific Grove to San Luis Obispo (approximately 240 km away). We examined
the population structure by looking at cytochrome c oxidase subunit I (COl), a mitochondrial
gene involved in the electron transport chain during respiration. COl is highly conserved across
species, so variations within this locus can give insight into evolutionary relationships.
Polymorphisms within this gene additionally provide a reliable method for distinguishing
between N. emarginata and N. ostrina. The use of COI therefore allows the determination of the
distribution of species along the central California coast. The genetic structure of these
populations can be used to provide additional information about the evolutionary history of
Nucella, as well helping to explain what causes the observed distribution. It can be tested
whether sharp changes in genetic structure occur at biogeographic boundaries (see Avise 1992)
or if there is some other physical boundary responsible for the shift. It would also be possible to
determine if the distribution of Nucella in central California is dependent on other factors, such
as temperature, salinity or wave exposure gradients. In the future, if shifts were observed in the
range of either of these species, it could then be possible to relate these changes to climate
change or to changes in other environmental factors.
MATERIALS AND METHODS
Sample collection and DNA extraction. Nucella species were collected at four different
sites along the California coast between April 23 - May 24, 2003. At Hopkins Marine Station
(HMS) in Pacific Grove (36°37’N, 121°54’W), thirteen snails were collected in the wave
protected area near the pilings and twenty-one snails were collected in a wave exposed area. At
Point Joe (JOE) (36°36’N, 121°57W2), six snails were collected in a wave exposed area, five
snails were collected in an area with moderate wave exposure and two snails were collected in a
wave protected area. At Soberanes Point (SÖB), eight snails were collected from a wave-
exposed area near the USGS benchmark (36°22’30"N, 121°52’30“W). In San Luis Obispo
(SLO) (35°20’N, 120°43’W), seven snails were collected in a wave protected area in St. Luis
Harbor, near Pismo Beach. The shell morphology of snails was examined and shell coloration,
shell texture, the number of whorls, angular vs. rounded "shoulder", and the presence of an
adapical tooth were recorded. Snails were additionally photographed. Äfter the examination of
shell morphology, snails were placed in an isotonic solution of magnesium chloride (MgCl) for
30- 60 minutes in order to relax their muscles. A snail was considered to be relaxed when it no
longer withdrew into its shell when touched. A small piece of foot-tissue was then biopsied.
Tissue was digested and DNA was extracted using the Nucleospin Tissue Kit (Machery-
Nagel/BD Biosciences).
PCR and Sequencing. A 463 bp fragment from the mitochondrial gene cytochrome c
oxidase subunit I (COI) was amplified using the polymerase chain reaction (PCR). Several
previously published COI primers were tried, including the universal primers COla and COlf
(Palumbi), the universal primers 1490 and 2198 (Folmer 1994) and the <span style='background-color:grey' title='Nucella canaliculata : Fuzzy'>Nucella canliculata</span> COl
primers COIM-L and COIM-H (Sanford 2003). These primers, for the most part, did not result
in successful amplifications, so new primers were designed from published N. emarginata and N.
ostrina sequences: Nucella COIF (S’-CTCATATTGTGRGCCATTATTCAGC-3’) and Nucella
COIR (S’-CAACGACTATGAAGCGAAACACCC-32). DNA template (1 uL of a 1:100
dilution) was added to a reaction mix consisting of 16.4 uL ddH2O, 2.5 uL 1OX PCR buffer, 2.5
uL dNTPs (8 mM), 1.25 uL each primer (10 mM) and 0.3 uL AmpliTaq polymerase. The PCR
profile consisted of a 2 min incubation at 94°C, followed by 36 cycles of 94 °C for 30 sec, 60 °C
for 1 min, and 72 °C for 1.5 min. PCR products were visualized on 2% agarose gels stained with
ethidium bromide.
The products of successful PCR amplifications were then prepared for sequencing as
follows: PCR product (5 uL) was added to 2 uL Shrimp Alkaline Phosphatase (SAP), 1 uL
Exonuclease I (EXO) and 0.5 uL dilution buffer and then incubated at 37 °C for 30 min,
followed by 80 °C for 15 min. This reaction removes excess dNTPs and primer dimers. SAP
product (2 uL) was then added to 6 uL ddH2O, 1 uL Big Dye, 1.5 uL 5X buffer, and 0.5 uL
primer (10 mM). This reaction mixture was then cycle sequenced for 25 cycles of 96°C for 10
sec, 50 °C for 5 sec, and 60 °C for 4 min in order to add fluorescent dye to the nucleotide bases.
After cycle sequencing, DNA was precipitated with 40 uL of 75% isopropanol for 15 min and
then pelleted by spinning at approximately 3200 rpm for 1 hour. Excess isopropanol was
removed by inverting the opened tubes and spinning them at 700 rpm for 2 min. Samples were
resuspended in 20 uL of High Dye, heated at 96 °C and then sequenced using an ABI3100
Genetic Analyzer.
Analysis of sequences. Sequences were viewed using Sequencher"" 4.1. Both the
sequence of bases and their corresponding chromatograms were examined and sections of
sequence with unclear chromatograms (i.e. where obvious, individual peaks were not present)
were removed. The forward primer and reverse primer sequences for each snail were compared
and then combined into a single consensus sequence. These consensus sequences were aligned
by assembling them into a contig and then were examined for polymorphisms, both within and
-6-
between populations. All sequences were additionally compared to published sequences for N.
emarginata and N. ostrina (Genbank accession numbers AF076536-AF076560) to determine
which species were present within a geographic location. Following the “cleaning" of sequences
and the preliminary analysis using Sequencher"" 4.1, the consensus sequences were transferred
to PAUP“4.Ob10 (PPC) to create phylogenetic trees based on a parsimony analysis. Consensus
sequences were also analyzed using heap big Power Mac (program by S. Palumbi) to calculate
variation within and among geographic locations, as well as the corresponding Fsr value. The
Fsr value was calculated by dividing the variation within a population by the variation between
the populations and subtracting this answer from one. A bootstrap analysis was then performed
to calculate P-values.
RESULTS
Morphology. Shells from both N. emarginata and N. ostrina exhibited a wide range of
morphologies (Fig. 1). Shell colors ranged from light (white, various shades of gray, and tan) to
dark (black, brown, and maroon). Some shells additionally had stripes. Certain morphs of both
species had smooth shells, but only N. emarginata exhibited a highly knobby shell. Both species
had between four and six body whorls and could have either angular or rounded shoulders. The
presence of an adapical tooth was only observed in N. emarginata.
PCR and sequencing. Performing PCR with the universal primers 1490 and 2198
(Folmer) yielded no amplification products (Fig. 2). PCR with the universal primers COla and
COIf (Palumbi) either yielded no amplification products or products with multiple bands (Fig.
3). PCR with the primers COIM-H and COIM-L (Sanford) yielded good amplifications about
55% of the time (Fig. 4), but the corresponding sequences contained ambiguous bases at several
nucleotide sites. PCR with the new primers Nucella COIf and Nucella COIr yielded both clean
amplifications and clean sequences (Fig. 5).
Analysis of sequences. The 463 bp COI fragments from all four geographic locations
had 12 variable sites, which resulted in 10 unique haplotypes (Table 1, Fig. 6). Comparisons of
these haplotypes to published sequences for N. emarginata and N. ostrina show that the
populations at Hopkins Marine Station and Point Joe are N. emarginata and the populations at
Soberanes Point and San Luis Obispo are N. ostrina (Fig. 7). Four nucleotide base sites were
used to make the determination between species. At each of these sites, all of the N. ostrina
individuals had one base, while a majority of the N. emarginata individuals showed another
base. All of the individuals in this study were polymorphic at these sites, however, a few of the
previously published N. emarginata sequences (Marko 1998) did not share these polymorphisms
(see appendix). The presence of a certain nucleotide base at each of these four nucleotide sites
allowed for the positive identification of N. emarginata individuals, since these four bases are
only observed in N. emarginata. The presence of the alternate base implies that the individual is
an N. ostrina, however this assumption could possibly have led to misidentified individuals. To
help identify individuals with a greater level of certainty, unmarked shells from Point Joe and
San Luis Obispo were additionally sent to an expert for identification (Marko, pers. comm.). His
species designations agreed with the species designations made based on sequences. This
patchwork distribution of species is somewhat unexpected, since species often have continuous
ranges.
Not only is there variation between species, but variation within a population also exists.
Intrapopulation variation ranged from 0.056 - 0.194%, while variation between populations of
each species ranged from 0.218- 0.546%. These values give Fsr ratios between 0.303 and 0.852
(Table 2). Using these Fsr ratios to perform bootstrap analyses shows that all four geographic
locations have genetically distinct populations (P + 0.0001 in all cases). Although significant,
this variation is very small, with haplotypes of a species varying by no more than three
nucleotide bases. The significance arises because these haplotypes are very local, with only one
haplotype being observed in more than one geographic location (Table 1). Figures 8 and 9 show
two parsimony trees generated by doing a heuristic search with branch-swapping and random
stepwise addition. Figure 8 shows one of the one hundred trees generated. Figure 9 is the
majority rules consensus tree for these same one hundred trees, i.e. this tree shows only
branching that occurred in at least 50% of all trees.
Since Hopkins Marine Station had the largest sample size, its population structure was
examined more carefully. Three unique haplotypes are present at Hopkins Marine Station (Table
1). One haplotype clusters with N. ostrina on a phylogenetic tree, and differs from the other two
haplotypes by at least seven nucleotide bases. The other haplotypes cluster with N. emarginata
on a phylogenetic tree (Fig. 8) and differ from each other by only one nucleotide base. The
observed polymorphism is a cytosine to guanine transversion, which is somewhat uncommon. It
is possible that this transversion is due to sequencing error. However, the sequences showing the
transversion were sequenced on different days and with different sets of primers, so the
probability of this result being sequencing error is small. Looking only at the N. emarginata
sequences at Hopkins Marine Station, differences in population structure are apparent between
wave exposed and wave protected areas. The shift in haplotype frequencies between protected
and exposed sites is significantly different (Fsr = 0.089, P = 0.03. See Fig. 10). These two sites
are separated by approximately 100 m, including an approximately 10 m stretch of sand, which
could have prevented members of these populations from reaching each other to reproduce.
DISCUSSION
Both N. emarginata and N. ostrina were found in central California, as expected based on
previous work (Marko 1998, Marko 2003). Even though it is the southern of the two species, N.
emarginata was found at the two northern sites at Hopkins Marine Station and Point Joe, while
N. ostrina was found at the two southern sites at Soberanes Point and San Luis Obispo. A brief
glance at the phylogenetic trees suggests, and the results of bootstrapping confirm, that all four
of these locations are genetically distinct populations (P + 0.0001). This result is expected, since
Nucella populations are very isolated from each other. Although low larval dispersal does not
always lead to genetic differentiation over distance (McFadden and Aydin 1996), isolation
between populations is also observed in other low dispersal species, including some copepods
(Burton et al. 1979) and some corals (Hellberg 1994). Rather than releasing larvae into the
ocean where they can be carried hundreds of kilometers away as many marine invertebrates do,
Nucella lay egg cases in which the embryonic snails develop before emerging as juvenile snails.
Individual snails then do not move very far during their lifetime (see West 1986 for snail
movement over a 4 month period). Gene flow is thus very limited between separate populations
and the genetic structure of a population is determined almost entirely by local egg production.
Laboratory studies also show that little or no hybridization occurs between the two species
(Marko, pers. comm.), which would further reduce the possibility of genetic mixing of
populations.
Each geographic location additionally only has one of the two species. However, there
are two possible exceptions. First, one snail from Hopkins Marine Station (E7) clusters with the
snails from Soberanes Point. It is possible that this individual is actually an N. emarginata with
-10-
N. ostrina-like sequence, in which case there would still only be one species present at Hopkins
Marine Station. Or, if this individual is an N. ostrina, there could be a small population of N.
ostrina at Hopkins Marine Station, with more individuals not being found due to the relatively
small sample size (34 individuals). If this is the case, it would oppose the trend seen both in this
study and by Marko (1998, pers. comm.) for only one species to be present per site. Another
possibility is that a single <span style='background-color:#fcc' title=' : NoMatch'>N. ostrina made</span> it to Hopkins Marine Station, possibly stuck to the
side of a kayak, or brought accidentally by a researcher. Without other members of its species
present, this snail will most likely die without reproducing, thus having no effect on the genetic
structure of the Hopkins Marine Station population. The other possible exception involves San
Luis Obispo. This study found only N. ostrina at San Luis Obispo; however, Marko (1998)
found only <span style='background-color:#fcc' title=' : NoMatch'>N. emarginata there.</span> Most likely, these two sample populations were collected from
different areas, with each area having its own distinct population. If this is the case, the change
in genetic structure occurs over a very short distance. However, the location at which Marko
collected his snails is unavailable at this time, so it remains unknown as to whether these two
species inhabit the same rocky headland. Overall, the trend seems to be that only one species is
present per geographic location. Why would this be the case? One hypothesis is that this
phenomenon is due basically to chance. Whichever species happens to colonize a location first
(having gotten there by migration, or on floating debris or a boat, etc.) is able to dominate in
resource use and prevent members of the other species from establishing a population. Such a
founder event could also lead to a rapid shift in the genetic structure of a population, as it adapts
to a new environment (Palumbi 1994). If this founder effect occurred, it would further help to
explain the genetic distinctness observed at each geographic location. Another possibility is that
selection might be acting on these two species. Previous work has shown that clines in physical
-11-
factors can correspond to clines in the allozymes responsible for dealing with these factors.
Mitochondrial haplotypes often also follow this cline (reviewed in Palumbi 1994). If one species
of Nucella is better adapted to certain environmental condition(s) in a particular location, natural
selection could prevent the other species from being able to establish a population. It is then
possible that the patterns observed in COI could be reflecting this selection.
Genetic structure is evident over small geographic ranges in Nucella in central California.
Further collection of snails in the zone of overlap between the two species will hopefully shed
light on what is responsible for the distribution of each species - whether it is determined by an
environmental gradient, or if physical barriers are the major factor, or if distribution is the result
of random migration events in the past. Additional geographic locations would also allow the
determination of whether Nucella species follow either the isolation by distance or stepping
stone models. In order to examine these questions, more snails will be collected between
Soberanes Point and San Luis Obispo at various distances apart to continue to determine the
small-scale genetic structure of these two species.
ACKNOWLEDGEMENTS
I would like to thank Stephen Palumbi for his guidance and all of his time, as well as for
collecting snails. An additional thank you to the entire Palumbi lab for all of their help and
support, and especially for teaching me new lab techniques. And to Peter Marko, thank you for
the technical advice and the help with identifying snail shells.
-12-
Literature Cited
Avise, J. 1992. Molecular population structure and the biogeographic history of a regional fauna:
a case history with lessons for conservation biology. Oikos 63:62-76.
Burton, R., M. Feldman, and J. Curtsinger. Population genetics of <span style='background-color:#cfc' title='Tigriopus californicus : Exact'>Tigriopus californicus</span>
(Copepoda: <span style='background-color:#cfc' title='Harpacticoida : Exact'>Harpacticoida):</span> I. Population structure along the central California coast. Marine
Ecology Progress Series 1:29-39.
Burton, R., and M. Feldman. 1981. Population genetics of Tigriopus californicus: II.
Differentiation among neighboring populations. Evolution 35(6):1192-1205.
Crothers, J. 1984. Some observations on shell shape variation in Pacific Nucella. Biological
Journal of the Linnean Society 21(3):259-282.
Folmer, O., M. Black, W. Hoeh, R. Lutz, and R. Vrijenhoek. 1994. DNA primers for
amplification of mitochondrial cytochrome coxidase subunit I from diverse metazoan
invertebrates. Mol. Mar. Biol. Biotechnol. 3:294-299.
Gavrilets, S., H. Li, and M. Vose. 2000. Patterns of parapatric speciation. Evolution 54(4):1126-
1134.
Hellberg, M. 1994. Relationships between inferred levels of gene flow and geographic distance
in a philopatric coral, Balanophyllia elegans. Evolution 48(6):1829-1854.
Hellberg, M. 1995. Stepping-stone gene flow in the solitary coral Balanophyllia elegans:
equilibrium and nonequilibrium at different spatial scales. Marine Biology 123: 573-581.
Kimura, M., and G. Weiss. 1964. The stepping stone model of population structure and the
decrease of genetic correlation with distance. Genetics 49:561-576.
Marko, P. 1998. Historical allopatry and the biogeography of speciation in the prosobranch snail
genus Nucella. Evolution 52(3):757-774.
Marko, P., A.R. Palmer, and F. Vermeij. 2003. Resurrection of Nucella ostrina (Gould, 1852),
lectotype designation for N. emarginata (Deshayes, 1839), and molecular genetic evidence of
Pleistocene speciation. The Veliger 46(1):77-85.
McFadden, C., and K. Aydin. 1996. Spatial autocorrelation analysis of small-scale genetic
structure in a clonal soft coral with limited larval dispersal. Marine Biology 126:215-224.
Palmer, A.R., S. Gayron, and D. Woodruff. 1990. Reproductive, morphological, and genetic
evidence for two cryptic species of northeaster Pacific Nucella. The Veliger 33(4):325-338.
Palumbi, S. 1992. Marine speciation on a small planet. Trends in Ecology and Evolution 7:114-
118.
Palumbi, S. 1994. Genetic divergence, reproductive isolation, and marine speciation. Annual
Review of Ecological Systems 25:547-572.
Sanford, E., M. Roth, G. Johns, J. Wares, and G. Somero. 2003. Local selection and latitudinal
variation in a marine predator-prey interaction. Science 300:1135-1137.
West, Lani. Intertidal variation in prey selection by the snail Nucella (=Thais) emarginata.
Ecology 67(3):798-809.
Wright, S. 1943. Isolation by distance. Genetics 28:1 14-138.
-13-
Table 1. Number of individual snails sequenced from each location. Haplotypes correspond to
the numbers on the phylogenetic tree (Fig. 8). Numbers in parentheses are the number of
individuals with each haplotype at a location.
Haplotype.
Location
Number of Individuals
HMS Protected
1 (2), 2 (11)
13
HMS Exposed
1 (8), 2 (12), 6 (1)
JOE
13
1 (2), 3 (8), 4 (2), 5 (1)
SOB
7(7), 8 (1)
SLO
9 (5), 10 (2)
Table 2. Percentage variation within and between populations. Corresponding Fsr values are
shown in parentheses.
’ indicates the value was not calculated.
JOE
SLO
HMS
SOB
0.218%
HMS
0.194%
—
(0.303)


JOE
0.110%
0.546%
0.056%
SOB
(0.852)
0.105%
SLO
Fig. 1. Varying shell morphology in A) N. emarginata and B) N. ostrina. C) Rounded
"shoulder" morphotype. D) Angular "shoulder" morphotype. E) Adapical tooth. The tooth is
the small bump visible on the apical end of the columella.
Fig. 2. Unsuccessful PCR reaction with the primers 1490 and 2198. Lane 1 is a 1-kb ladder,
lanes 2-1 8 are individual snails and lanes 19 and 20 are the positive and negative controls,
respectively. Note only faint primer dimers are present.
Fig. 3. Unsuccessful PCR reaction with the primers COla and COIf. Lane 1 is a 1-kb ladder,
lanes 2-16 are individual snails, and lanes 17 and 18 are the positive and negative controls,
respectively. DNA templates either did not amplify (lanes 1, 2, and 6) or produced several bands
(lanes 3-5 and 7-16). Some product is also visible in the negative control, indicating this reaction
was contaminated.
Fig. 4. PCR reaction with the primers COIM-H and COIM-L. Lane 1 is a 1-kb ladder and lanes
2-20 are individual snails. Lanes marked with dots represent amplifications good enough for
sequencing.
Fig. 5. A) PCR reaction with the primers Nucella COIf and Nucella COIr. Lane 1 is a 1-kb
ladder, lanes 2-7 are individual snails, and lanes 8 and 9 are the positive and negative controls,
respectively. Lanes marked with dots represent amplifications good enough for sequencing.
B) Corresponding chromatogram. Note the peaks are all clearly defined and fairly uniform in
height.
Fig. 6. Haplotypes found at each location.
Fig. 7. Location of collection sites and the species found at each location.
Fig. 8. A parsimonious phylogenetic tree created from COI sequences. The numbers on the
right of the branches correspond to haplotype numbers.
Fig. 9. A phylogenetic tree showing branching that occurred in at least 50% of 100 parsimony
trees. Numbers indicate the percentage of trees displaying each branching pattern.
Fig. 10. Differences in COI haplotype frequencies between wave exposed and wave protected
areas at Hopkins Marine Station.

Fig. 2.
Fig. 3.
Fig. 4.
Fig. 5.
2   ..
B)
GC ITTATIGICIGAGC ICATCATATAT TTACAGIGGGAAL

L ALTNTRRENW
Fig. 6.

Monterey Bay
HMS -
JOE
SOB
2
SLO
Fig. 7.

Monterep be)
IIMS —
JOE—
SOB
• N. ostrina
• N. emarginata
SLO
Fig. 8.
— 0.1 changes
46510
47S10
48 S10
49 S10
51810
50stc
52 s10
P2HME
EAHME
-29HNE
-30 HNE
-31HME
E32 HNS
E33HNE
E3AHNE
E35 HME
E36 HNE
E37HNE
Ed3HNE
EA4HNE
P53 HME
P54HNE
P55HME
P56 HME
P5T HNE
P58 HNE
P59 HME
P6O HME
P6IHNE
P62 HME
E56 JOE
P75 JOE
E9 SOBF
EIISOBF
E12 SOBF
E13 SOB
EIASOBF
E15 SOBF
E16 SOBF
PIHNE
PSHME
ESHME
E6 HNEF
E3SHE
E40HNEF
EAIHNE
E42HNE
E45HNBE
E63 JOE
E64 JOE
E65 JOE
E68 JOE
E69 JOE
MT1 JOE
MT3 JOE
MTA JOE
— NTO
E 5
— E17 SOBF 8
MT72 JOE
1P76 JOE
-ETHME 6
Fig. 9.
53
53
53






—



100
— PIHNE
- P3HME
ESHME
- E6 HNEF
E38 HMEF
-E39HMEF
-E40 HMF
— E41HME
-E42HME
- E45 HMEF
-P2HNE
—EAHME
-E29HME
-E30HME
E3IHME
-E32 HME
-E33HME
-E3AHME
-E35HME
-E36HME
-E3THME
-Ed3HNE
-E44HME
P53HME
-P54 HME
- P55 HME
- P56 HME
P57 HME
— P58 HME
- P59 HME
P60 HME
P6IHME
-P62 HME
-E63 JOE
-E64 JOE
-E65 JOE
- E66 JOE
-E68 JOE
-E69 JOE
MTO JOE
- M2 JOE
- P76 JOE
-MT1 JOE
-MT3 JOE
-MTA JOE
- P75 JOE
-ETHME
-E9 SOBF
-EIISOBF
-E12 SOBF
-E13 SOBF
-EI4 SOBF
-E15 SOBF
-E16 SOB
-E17 SOBF
- 46S10
-47510
-48S10
-49S10
-50stc
-52S10
—51st0
Fig. 10.
Protected

Exposed
PPENDIX.
Sequences for unique haplotypes found at all four locations, plus previously published haplotypes for N. ostrina
and N. emarginata (Marko 1998).
40
20
CTCATATTGTGAGCCATTATTCAGCTAAAAAAGAAACATTTGGAACTT
[48
ostrina haplotype 25
emarginata haplotype 19
481
...........G....................................
[48
haplotype
...........G.o.
481
haplotype
........G..ccccccc.
481
haplotype
..........G..ccc......
haplotype
...........G..ccco
[481
haplotype 5
...........G....................................
[481
haplotype
[48
ooooooooooooooooooooooooo.
48
haplotype
cccceceoocooooooooooooooosoded.
[481
haplotype
oecooeooeeeeooeoeoeeeesosoo
haplotype9
..........G.c
481
haplotype10
2223333333.G....................................
1381
﻿70
80
90
﻿50
﻿60
Sstrina haplotype 35
TAGGTATAATTTATGCAATATTAGCTATTGGGGTTTTAGGTTTTATTG
196
emarginata haplotype 19
196
........................................C.......
........................................C.......
1961
haplotype
1961
............cc.ccC..
haplotype
haplotype
196
........................................C.......
196
haplotype
........................................0.......
haplotype
........................................C.......
[96
1961
........oooa
haplotype
haplotype
196
acoooooooeooooeeooooooooooooooooo
196
haplotype 8
.ooooooooooooooooooooooooooooo.
haplotype9
..........coooc.
1961
..........c.
1861
haplotype 10
ostrina haplotype 25
emarginata haplotype 19
haplotype
haplotype.
haplotype
haplotype 4
haplotype 5
haplotype
haplotype"
haplotype 8
haplotype 9
haplotype 10
ostrina haplotype 25
emarginata haplotype 19
haplotype 1
haplotype 2
haplotype
haplotype 4
haplotype 5
haplotype
aplotype
haplotype
haplotype9
haplotype 10
ostrina haplotype
emarginata haplotype 19
haplotype 1
haplotype 2
haplotype3
haplotype 4
haplotype 5
haplotype 6
haplotype
haplotype
haplotype
haplotype 10
﻿110
﻿120
﻿130
140
100
TCTGAGCTCATCATATATTTACAGTGGGAATGGATGTAGATACACGAG
144
eccceoooooooooooeooooo.
(144
144
eeooeeeooooeooooooooooooo..
eooooooooeooooooooooooooooooooooo.
1441
1441
oeeeeceeoooooooooooooooooooooooooo
(144
ccceeooooooooeoooooooooooooo.
(1441
eeoooooooeoooooooooooo
(1441
cccccccccccccoooooooooo.
...N....................N.......................
(1441
...N......C.............N.......................
1441
(1441
acooooooooooooooooooooo.
aeccccooeoooooooooooooooooooooooo.
(134
190
﻿150
﻿160
170
﻿180
CTTATTTTACAGCTGCCACAATAATTATTGCTGTACCTACAGGTATTA
(1921
192
ooooeooeoeeeooeoo..
(192
cecccccceceoooooooooooo.
192
aooooeoooeooooooooooooooo.
(1921
aoooooooooooooooooo
1921
aooooooooeoooooeeooooooooooooooo
(1921
aaooooooooeoooeeoeooeooeoeeoooo.
(1921
ooeeeooooooooooeeeooso..
(192
aoooeoeeeoeeeoeooseooeeoooeosoeoe.
1921
ccccc.....
(1921
acececeocooooooooooooooooooooooooo.
1821
........ccccccccG.cc....
230
200
﻿210
220
240
240
AAGTATTTAGATGATTAGCCACGATTCATGGTGCTAAAATTAAGTATG
240
.ceeoooooooooooooooooooooo.
240
.aaooooooooooooo.
240
...oooooooooooeooooooooooooo.
240
.aeoooooooooooooooooooooooooo.
240
aaaooooooooooooooooooooooooooooo.
240
acooooooosoooseoeoeoooo.
240
aeoeoeeeoooeeoooooosoeo.
.........N.......N..............................
240
(240
.........N.......N..............................
240
..ooooo.
230
.cooeeeoooooeoooooeooooooooo.
ostrina haplotype 25
emarginata haplotype 19
haplotype
haplotype 2
haplotype
haplotype 4
haplotype 5
haplotype 6
haplotype 7
haplotype 8
haplotype 9
haplotype 10
ostrina haplotype 25
emarginata haplotype 19
haplotypel
haplotype 2
haplotype 3
haplotype 4
haplotype 5
Saplotype 6
haplotype.
haplotype8
haplotype
haplotype 10
ostrina haplotype 25
emarginata haplotype 19
haplotypel
haplotype
haplotype
haplotype 4
haplotype 5
haplotype 6
haplotype
haplotype 8
haplotype9
haplotype 10
﻿270
﻿250
260
280
AAACACCTATATTATGGGCACTTGGATTTATTTTTTTATTTACAGTAG
coccoeoooooooo
eoooooo.
oooooooeeoooo..
aaoooeoeoeooo
ooooooooooooooooooooooooooooooooo
eeooeeeeeeeeeeeoeeseeeeeeeeeseeo..
ecaecccooeoeoeeoeooeeoooeoooooooo.
aoeoeeoooo.
oooooooooseoooo.
oooooooooo.
aaeocoooooeoeooooooeoooooooeeoooooo.
﻿290
300
330
﻿310
320
GAGGTTTAACAGGGATTGTGTTATCTAATTCTTCTTTAGATATTATGA
.....C..............C.........T.................
.....C..............C...........................
.....C..............C...........................
.....C..............C...........................
.....C..............C...........................
.....C..............C...........................
ooooooooooeoesoooooo.
ooooooeoooooooooooooooooooooo.
eaaaooooooooooooooooooseooooooo.
eoooooooeeoeeeeoeeeeoeeeoeeeo.
oooooooooooooooooo.
340
﻿370
﻿350
360
﻿380
TGCATGATACTTACTACGTAGTTGCTCATTTTCATTATGTTCTATCAA
...............................C................
eoooooooeooeoooeoooooooseooooo.
aooooeooooooeoeooeooeeooooo
eooooooooooooooooooo.
........................................C.......
.........ccccccccC......
.............T.................................G.
.c.oooooooo.
ccceoooo.
oooooooeooooooo
eeoeceeeeoeeeeeeeeeeoeeeoeseeoeo.
(288
288
(288
288
2881
2881
288
288
288
288
288
(2781
[336
(336
(336
[336
[336
(3361
(336
[336
3361
336
[3361
[326
384
[3841
(384
[3841
(384
[3841
(384
3841
3841
(3841
[384
(3741
ostrina haplotype 25
emarginata haplotype 19
haplotype 1
haplotype
haplotype
haplotype
haplotype 5
haplotype
haplotype 7
haplotype 8
haplotype 9
haplotype 10
ostrina haplotype 25
emarginata haplotype 19
haplotypel
haplotype
haplotype3
haplotype
haplotype
aplotype
haplotype
haplotype
haplotype
haplotype 10
﻿390
400
410
420
430
TAGGAGCAGTTTTTGCATTATTTGGAGCTTTTAATTATTGATTTCCTC
...G....ccccccccccccccccccccccecc
....G.....ccc..
....G....ccccc.........
....G....cc.....
....G....ccccc..
....G....ccccccccccccc
oooooooooosooooooo
eeeesoeeeeeceoseeseeeeeeeeeeeeeoeoosee..
eoooooeeooooeeooeeseeeesooo
eaaooooooooo
ooooooooo..
480
440
﻿450
460
﻿470
TTTTAAGGGGTGTTTCGCTTCATAGTCGTTGAACTAAAGCCCATTTTT
aoooooooooooooooooooooooooooooooo

. . . . . . . . . . . . . . . . . . ...............22
.... ...............C.............
...................C.............
........................... .....GAG.:::::::::::
.......T........
.. .....T.............
.......T............
.. . . . . . . . . . . . . . . . . . . . . ..........22
.................................
(4321
(4321
[4321
(4321
[4321
432
4321
4321
[4321
432
432
(4221
[480
480
[4511
4631
(4641
[463
468
[451
(4511
4511
463
[4531
</pre></body></html>